<!DOCTYPE html>
<html lang="en">
		<head>
		<meta http-equiv="Content-Type" charset="utf-8" name="viewport" content="width=device-width, initial-scale=1">
		<meta name="description" content="Generate a data management plan in minutes. Registration-free and FAIR."/>
		<meta name="keywords" content="FAIR, data, management, plan, DMP, free, open source">
		
		<!-- disable cache accross browsers for testing purpose -->
		
		<meta http-equiv="Expires" content="0" />
		<title>Data Management Plan Generator -- a user-centered FAIR tool</title>
		
		<!-- https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP -->
		<!-- loading css styleSheets -->
		<link href="css/bootstrap.min.css" rel="stylesheet" >
		<link href="css/custom.css" rel="stylesheet">
		<link href="css/bs5-intro-tour.css" rel="stylesheet" />
		
		
		<!-- loading javascript scripts -->
		<script src="js/bootstrap.bundle.min.js" type="text/javascript" charset="utf-8"> </script>
		<script src="js/FileSaver.js" type="text/javascript" charset="utf-8"></script>
		<script src="js/bs5-intro-tour.js" type="text/javascript" charset="utf-8"></script>
		<style type="text/css" data-style-name="dmp1" >
		ol{margin:0;padding:0}table td,table th{padding:0}
		.c1{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Arial";font-style:normal}
		.c3{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:13pt;font-family:"Arial";font-style:normal; color:#0088CC;}
		.c7{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:12pt;font-family:"Times New Roman";font-style:normal}
		.c22{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;height:12pt}
		.c4{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:17pt;font-family:"Arial";font-style:normal}
		.c14{padding-top:14pt;padding-bottom:4pt;line-height:1.15;orphans:2;widows:2;text-align:left}
		.c13{padding-top:24pt;padding-bottom:6pt;line-height:1.15;orphans:2;widows:2;text-align:left}
		.c10{padding-top:12pt;padding-bottom:3pt;line-height:1.15;orphans:2;widows:2;text-align:left}
		.c20{padding-top:12pt;padding-bottom:6pt;line-height:1.15;orphans:2;widows:2;text-align:left}
		.c9{padding-top:18pt;padding-bottom:4pt;line-height:1.15;orphans:2;widows:2;text-align:left}
		.c0{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;margin-top:0cm;
			margin-right:0cm;
			margin-bottom:10.0pt;
			margin-left:0cm;
			line-height:115%;
			text-autospace:ideograph-other; 
			}
		.c6{padding-top:12pt;padding-bottom:10pt;line-height:1.1500000000000001;orphans:2;widows:2;text-align:left}
		.c12{color:#000000;text-decoration:none;vertical-align:baseline;font-family:"Arial";font-style:normal}
		.c8{font-size:13pt;font-weight:700}
		.c2{font-size:23pt;font-weight:700}
		.c21{font-weight:400;font-size:20pt}
		.c19{max-width:468pt;padding:72pt 72pt 72pt 72pt}
		.c5{font-style:italic}.c17{font-size:12pt}
		.c15{font-size:7pt}.c18{font-size:17pt}
		.c16{background-color:#ffffff}.c11{font-weight:700}
		.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		li{color:#000000;font-size:12pt;font-family:"Arial"}
		p{margin:0;color:#000000;font-size:12pt;font-family:"Arial"}
		h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		h5{padding-top:12pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}
		h6{padding-top:12pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style>
	    

		<link href="favicon.png" rel="icon">
		
		<script>
		window.dataLayer = window.dataLayer || [];
		function gtag(){dataLayer.push(arguments);}
		gtag('js', new Date());

		gtag('config', 'G-2BBTS3ZDF7', {
				client_storage: 'none',
				client_id: 'G-2BBTS3ZDF7',
		});
		</script>
			</head>
			<script>
			// check if the browser is firefox. For firefox, an argument in window.find() function is different
			var is_firefox = navigator.userAgent.toLowerCase().indexOf('firefox') > -1;
			var doc_name = 'dmp1';
			var user_info = window.localStorage.getItem("user_info");
			var cached_a = window.localStorage.getItem("saved_a");
			var cached_template = window.localStorage.getItem("template");
			var initialed = false;
			
			//if (window.localStorage.getItem("fulltext_user")==null){
			//window.localStorage.setItem("fulltext_user", document.getElementById("dmp1").//innerHTML)
			//}
			//var scrollSpyContentEl = document.getElementById('scrollspy-element')
			//var scrollSpy = bootstrap.ScrollSpy.getInstance(scrollSpyContentEl)
			function nextField(current){
				for (i = 0; i < current.form.elements.length; i++){
					if (current.form.elements[i].tabIndex - current.tabIndex == 1){
						current.form.elements[i].focus();
						if (current.form.elements[i].type == "text"){
							current.form.elements[i].select();
						}
					}
				}
			}
			
			function enter2tab() {
				
				
				if (window.event && window.event.keyCode == 13) {
				
					window.event.keyCode = 9;
				}
			}
			//console.log("is firefox ?"+is_firefox);
			function load_dmp(callback=reload_answers, name=cached_template) {
				doc_name = name;
				window.localStorage.setItem("template", doc_name);
				let content = document.getElementById(name);
				let frame = document.getElementById("doc3");
				frame.innerHTML = content.innerHTML;
				callback();
				initialed = true;
				
				
			}
					// template answers
					let temp_a = {"replace":{"$_PROJECT":"Example Project","$_STUDYOBJECT":"Example Objective","$_PROJECTAIM":"Example Aim","$_dataofficer":"Example data officer name","$_UPDATEMONTH":"Example Month","$_PREVIOUSPROJECTS":"Previous Project Name","$_proprietary":"Proprietary Software","$_RAWDATA":"???","$_DERIVEDDATA":"???","$_VISUALIZATION":"visualization or not"},"checkbox":{"checkbox_1":{"checked":[],"unchecked":["check_protect","check_dataplant","check_update","check_previousprojects","check_industry","check_proprietary","check_partners","check_EU"]},"checkbox_2":{"checked":["check_image","check_RNAseq","check_genomic","check_metabolomic","check_proteomic","check_targeted","check_models","check_code","check_excel","check_cloned-DNA","check_phenotypic","check_genetic"],"unchecked":[]},"checkbox_3":{"checked":["check_MIAPPE", "check_MinSeqE", "check_dublin_core","check_marc_21", "check_early", "check_ipissue"  ],"unchecked":[]} }};
					if (cached_a !== null){
					var saved_a = JSON.parse(cached_a);
					}else {
					var saved_a = {"replace":{"$_PROJECT":"Example Project","$_STUDYOBJECT":"Example Objective","$_PROJECTAIM":"Example Aim","$_dataofficer":"Example data officer name","$_UPDATEMONTH":"Example Month","$_PREVIOUSPROJECTS":"Previous Project Name","$_proprietary":"Proprietary Software","$_RAWDATA":"???","$_DERIVEDDATA":"???","$_VISUALIZATION":"visualization or not"},"checkbox":{"checkbox_1":{"checked":[],"unchecked":["check_protect","check_dataplant","check_update","check_previousprojects","check_industry","check_proprietary","check_partners","check_EU"]},"checkbox_2":{"checked":["check_image","check_RNAseq","check_genomic","check_metabolomic","check_proteomic","check_targeted","check_models","check_code","check_excel","check_cloned-DNA","check_phenotypic","check_genetic"],"unchecked":[]},"checkbox_3":{"checked":["check_MIAPPE", "check_MinSeqE", "check_dublin_core","check_marc_21", "check_early", "check_ipissue"  ],"unchecked":[]} }};
					}
					// saved answers
					
					// initialization
					
					function consent() {
					window.localStorage.setItem("cookie_consent", "yes");
						if (user_info == null){
						setTimeout(function(){
							tour.show();
						}, 500);
						
						}
					}
					
					const prefix = '$_';
					
					function save_json() {
					
						aJSONString = JSON.stringify(saved_a);
						const blob = new Blob([aJSONString], {
							type: "application/json"
						});
						saveAs(blob);

					}
					/*============== tutorial ==============*/
					
					var steps = [
					{
							title: "Data Management Plan Generator (DMPG) ",
							content: '<p> Welcome! let&#39;s generate a DMP and related documents in 5 minutes<br><p>1. A DFG compatible detailed DMP is availabe <a id="myLink1" href="#" onclick=\'load_dmp(reload_answers,name="dfg-dmp")\' >here   </a><br>2. A European <a id="myLink2" href="#" onclick = \'load_dmp(reload_answers,name="dmp1")\' >DMP</a> in detail (11 pages);<br>3. A <a id="myLink3" href="#" onclick = \'load_dmp(reload_answers,name="proposal")\' >proposal DMP</a> (2 pages); <br>4. A <a id="myLink4" href="#" onclick = \'load_dmp(reload_answers,name="practical-guide")\' >practical guide</a> in detail throughout the project</p>'
							//
						}, {
							id: "interface",
							content: "<p>Questions are listed here.</p>"
						}, {
							id: "navbar-example2",
							content: "<p>There are five sections of questions.</p>"
						}, {
							id: "intro-tour-input",
							content: "<p>Each section has a few questions</p>"
						}, {
							id: "form_project",
							content: "<p>Please type your project name here. For example, you can type: \"your awesome project\" to the highlighted input box, then click anywhere else or use tab key to go to next question. The answers will be submitted automatically.</p>"
						}, {
							id: "doc",
							content: "<p>Now the document's content has been updated.</p>"
						}, {
							id: "save_button",
							content: "<p>You can save the answers to json or copy the document on the left side.</p>"
						}, {
							id: "upload_button",
							content: "<p> If you have saved the \"json\" file, you can also upload it any time, to resume your DMP generation process </p>"
						}, 
						{
							id: "change_template",
							content: '<p> One DFG compatible detailed DMP is availabe <a href="#" onlick=\'load_dmp(reload_answers,name="dfg-dmp")\' >here</a>; </p><br><p>beside the current document, a <a id="myLink" href="#" onclick = \'load_dmp(reload_answers,name="proposal")\' >short proposal </a> can also be downloaded. <br> you can also download a <a id="myLink" href="#" onclick = \'load_dmp(reload_answers,name="practical-guide")\' >practical guide </a>. You can follow this guide to make sure that your data is well managed.<br> <a href="https://github.com/nfdi4plants/plan-generator/wiki/Home--%F0%9F%A4%96" >If you have more questions, please check our wiki</a></p>' 
						}
					];
					
					
					
					var tour = new Tour(steps);
					/*============== tutorial initialization ends ==============*/
					
					// event listener for text input (no need to press submit button anymore)
					

					// compare updated answers and the original answers
					
					function compare_answers(old_a, new_a){
						let unreplaced_keys = [];
						let unsubmitted_checkbox = [];
						let total_keywords = 0, changed_keywords = 0, total_checkboxes = 0, changed_checkboxes = 0 ;
						for (const [key,value] of Object.entries(old_a["replace"]) ){
							total_keywords ++;
							if ((new_a["replace"][key] != value) && (new_a["replace"][key]!= "") ){
							changed_keywords ++;
							
							console.log(new_a["replace"][key]);
							}else {unreplaced_keys.push(key);}
						
						}
					for (const [key,value] of Object.entries(old_a["checkbox"]) ){
							total_checkboxes ++;
							if (JSON.stringify(new_a["checkbox"][key]) != JSON.stringify(value)){
							console.log(new_a["checkbox"][key]);

							changed_checkboxes ++;
							
							}else{unsubmitted_checkbox.push(key);}
						}
					const unchecked_options = new_a["checkbox"]["checkbox_1"]["unchecked"];
					
					const uncheck_update =  unchecked_options.includes("check_update") ;
					if (uncheck_update) { unreplaced_keys= unreplaced_keys.filter(item => item !== "$_UPDATEMONTH") ; new_a["replace"]["$_UPDATEMONTH"]="";  }
					const check_previousprojects = unchecked_options.includes("check_previousprojects") ;
					if (check_previousprojects) { unreplaced_keys= unreplaced_keys.filter(item => item !== "$_PREVIOUSPROJECTS") ;  new_a["replace"]["$_PREVIOUSPROJECTS"]="";}
					const check_proprietary = unchecked_options.includes("check_proprietary") ;
					if (check_proprietary) { unreplaced_keys= unreplaced_keys.filter(item => item !== "$_proprietary") ; new_a["replace"]["$_proprietary"]="" }
					let no_need_to_answer = uncheck_update + check_previousprojects + check_proprietary;
					total_keywords = total_keywords - no_need_to_answer;
					console.log( "total_keys: " + total_keywords + "; changed_keywords: " + changed_keywords +"\n ;  total_box  " + total_checkboxes + ", changed_checkbox: " + changed_checkboxes + ", unreplaced_keys " + unreplaced_keys)
					return [total_keywords,  total_checkboxes, changed_keywords, changed_checkboxes, unreplaced_keys, unsubmitted_checkbox];
					}
					
					
					// reload all answers			
					function reload_all(path) {
										
						

						
						const selection = window.getSelection();
						selection.removeAllRanges();
						
						reload_answers();
						
						
					}

					function replace(child) {

						const before = child.previousElementSibling.name;

						const after = child.previousElementSibling.value;
						console.log(before, after);
						//undo(child.nextElementSibling);
						saved_a["replace"][before] = after;
						let singleOption = {};
						singleOption[before] = after;
						load_dmp(reload_answers, name = doc_name);
						//find_replace(singleOption);
					}
					
					

					function undo(this_element) {
						
						const selection = window.getSelection();
						selection.removeAllRanges();
						const previous_element = this_element.previousElementSibling;
						console.log(previous_element.getAttribute('onclick'));
						if (previous_element.getAttribute('onclick') == "replace(this)") {
							try {
								delete saved_a["replace"][previous_element.previousElementSibling.getAttribute('name')];
							} catch (e) {
								console.log("no such content, already undo");
							}

						} else {
							try {
								delete saved_a["checkbox"][previous_element.classList[0]];
							} catch (e) {
								console.log("no such content, already undo");
							}
						}

						console.log("json is " + saved_a);
						
					}

					function reload_answers() {
						const selection = window.getSelection();
						selection.removeAllRanges();
						window.localStorage.setItem("saved_a", JSON.stringify(saved_a));
						
						for (const [key, value] of Object.entries(saved_a["replace"])){
							if (key !== "$_VISUALIZATION"){
								let element = document.getElementsByName(key)[0];
								element.value = value  ;
							}
						}
						for (const [key, value] of Object.entries(saved_a["checkbox"])){
							let checkboxes = value;
							try{
								for (const [key, value]of Object.entries(checkboxes["checked"])) {
									let element = document.getElementById(value);
									element.checked = true;  
									// the check box of dataplant is opposite to others. It will collapse when dataplant is checked. That is why here is an exception.
									if (value != "check_dataplant"){
										try{
											document.getElementById(element.getAttribute("aria-controls")).classList.add("show");
										
										}
										catch(e){
											console.log("error is"+value)
										}
									}else {
									
										try{
											document.getElementById(element.getAttribute("aria-controls")).classList.remove("show");
									
										}catch(e){
											console.log("error is"+value)
										}
									
									}
								
								};
							}catch(e ){
									console.log("checked is null");
									
									}

									try{
									for (const [key, value]of Object.entries(checkboxes["unchecked"])) {
									let element = document.getElementById(value);
								element.checked = false; 
								if (value == "check_dataplant"){
								try {document.getElementById(element.getAttribute("aria-controls")).classList.add("show");
								
								}catch(e){//console.log(e)
								}	
								
								} else {
									try {
									document.getElementById(element.getAttribute("aria-controls")).classList.remove("show");
									}catch(e){//console.log(e)
										}
								
								}
															
									};
									}catch(e ){
									console.log("unchecked is null" );
									
									}

								
								
								
								
								
								

							
						}
						console.log("answers is reloading");
						

						
						
						
						for (let answers in saved_a["replace"]) {
							let singleOption = {};
							singleOption[answers] = saved_a["replace"][answers]
								console.log(singleOption);
							find_replace(singleOption);
						}
						for (let answers in saved_a["checkbox"]) {
							let singleOption = {};
							singleOption[answers] = {
								"checked": saved_a["checkbox"][answers]["checked"],
								"unchecked": saved_a["checkbox"][answers]["unchecked"]
							}
							console.log(singleOption);
							checkbox(singleOption);
						}
						console.log("answer reload is finished");
						let unfinished = compare_answers(temp_a, saved_a);
						let p_bar = document.getElementById("p-bar");
						let percentage =  (unfinished[2]+ unfinished[3])/(unfinished[0] + unfinished[1]) *100;
						p_bar.style.setProperty("width", Math.ceil(percentage) + "%");
						p_bar.innerHTML =  Math.ceil(percentage) + "%";
						p_bar.setAttribute("aria-valuenow", Math.ceil(percentage));
						
						if (is_firefox){
						
						setTimeout(()=>{document.getElementById("doc3").children[0].scrollIntoView();
						//console.log("")
						;},0
						
						)
						}
						
						
						
					}

					function get_option(this_element) {
						undo(this_element.nextElementSibling);
						const class_name = this_element.dataset.checkbox;
						console.log("class name for question is " + class_name);
						const checked = document.querySelectorAll(`.${class_name}:checked`);
						const unchecked = document.querySelectorAll(`.${class_name}:not(:checked)`);
						//console.log([...checked]);
						const checked_list = [];
						checked.forEach(element => checked_list.push(element.id));
						const unchecked_list = [];
						unchecked.forEach(element => unchecked_list.push(element.id));
						console.log("uncheck and checked" + checked_list, unchecked_list)
						saved_a["checkbox"][class_name] = {
							"checked": checked_list,
							"unchecked": unchecked_list
						};
						let singleOption = {};
						singleOption[class_name] = {
							"checked": checked_list,
							"unchecked": unchecked_list
						};
						//try {
						console.log("+++++++++++++++++" + checked_list);

						//}catch (e)
						//{ alert("you have submitted, please use undo if you want to change your input");}
						load_dmp(reload_answers,name = doc_name);
						//checkbox(singleOption);

					}

					// go to the highlighted location
					function go_to(element1) {
						const warning = element1.value;
						console.log("the warning text tag is " + warning);
						const element = document.getElementById(warning);
						try {
							element.scrollIntoView({
							behavior: "smooth",
							block: "center",
							inline: "nearest"
						});} catch (error) {
						alert("warning issue is already resolved, please close this warning");
						console.error("element not found");
						};
						try{
						const caused_ele = element.parentElement.parentElement;
						caused_ele.setAttribute("class", "c1 border-highlight");
						console.log("highlited");
						setTimeout( () =>{
						caused_ele.setAttribute("class", "c1")}, 3000);
						console.log("back to normal");
						}catch(error){
						
						consloe.log(error);
						};
						
						

					}
					function remove_warning(element) {

						let warning_no = element.value;
						let warning = document.getElementById(warning_no);
						try {warning.remove();} catch (error) {
						
						console.error("element not found");
						}

					}
					
					function remove_all_warnings(){
							
							const warning_notification= document.getElementsByName("_warning_notification");

							while(warning_notification.length > 0){
							console.log(warning_notification[0] + " is removed");
							remove_warning(warning_notification[0].children[2]);
							warning_notification[0].remove();
							
						}

					}

					function issue_warning()
					{
						
					
						
						const warning_notification= document.getElementsByName("_warning_notification");
						const warning_text_location = document.getElementById("warningText");
						const selection = window.getSelection();
						selection.removeAllRanges();
						let text;
						if (window.find) {
							
							ns = window.find("#issuewarning", aWrapAround = 1);
							let warning_number = 0;
							
							if (ns) {
							//remove all elements by the name of warning notification
								
								while(warning_notification.length > 0){
								console.log(warning_notification[0] + " is removed");
								warning_notification[0].remove();
							}
							
							}
							
							while (ns) {
							// text range is the text between #issuewarning and #endissuewarning without the tag
								let text_range = window.getSelection().getRangeAt(0).cloneRange();
							// full range is the text between #issuewarning and #endissuewarning include the tag
								let full_range = window.getSelection().getRangeAt(0).cloneRange();
							// relate range is the text related to the warning.
								text_range.setStart(window.getSelection().focusNode, window.getSelection().focusOffset);
								window.find("#endissuewarning");
								let end_range = window.getSelection().getRangeAt(0).cloneRange();
								text_range.setEnd(window.getSelection().anchorNode, window.getSelection().anchorOffset);
								full_range.setEnd(window.getSelection().focusNode, window.getSelection().focusOffset);
								text = text_range.toString();
								console.log("full range text is" + full_range.toString());
								full_range.deleteContents();
								
								
								
								let newText = document.createElement("warning_number" + warning_number);
								
								newText.innerHTML = '<span id= "warning_number' + warning_number + '" class = "text-warning"><--pay attention to the sentence before because'+ text  +'</span>';
								full_range.insertNode(newText);
								
								const warning_no = "warning_number" + warning_number;
								
								let div = document.createElement('div');
								div.setAttribute("class", "alert alert-danger");
								div.setAttribute("role", "alert");
								div.setAttribute("name", "_warning_notification");
								let html1 = '<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="currentColor" class="bi bi-exclamation-triangle-fill flex-shrink-0 me-2" viewBox="0 0 16 16" role="img" aria-label="Warning:"><path d="M8.982 1.566a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767L8.982 1.566zM8 5c.535 0 .954.462.9.995l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995A.905.905 0 0 1 8 5zm.002 6a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"/></svg>' + text + '<button type="button" value = "' + warning_no + '" class="btn-primary"  onclick ="go_to(this)" >click to check what caused the warning</button><button type="button" value = "' + warning_no + '" class="btn-close" data-bs-dismiss="alert" aria-label="Close" onclick=remove_warning(this) ></button>';

								div.innerHTML = html1;
								warning_text_location.appendChild(div);
								ns = window.find("#issuewarning", aWrapAround = 1);
								warning_number = warning_number + 1;
							}

						}
						
						
						warning_text_location.scrollIntoView();
					}

					/**
					 * edit checked/unchecked placeholders in the file.
					 * @param {string} right_range - the extended keyword (includes #if and #endif) need to be replaced.
					 *
					 */

					function checkbox(options) {

						
						// remove ranges, very important. took me long time to find this function
						const selection = window.getSelection();
						selection.removeAllRanges();
						// all objects which first class name is the option name
						const class_name = Object.keys(options)[0];
						console.log(class_name);
						const checked = options[class_name]["checked"];
						const unchecked = options[class_name]["unchecked"];

						//console.log([...unchecked]);
						for (let i = 0; i < checked.length; i++) {
							let check_box_item = checked[i];
							//console.log(check_box );
							let name = check_box_item.split('_')[1];
							// here the ________ should not match anything, make sure not to use it as a real placeholder
							let match_name = "_________";
							try {
								match_name = prefix.concat(name.toUpperCase());
								console.log("check for loop, starts " + match_name + " " + checked.length);
							} catch (e) {}

							selection.removeAllRanges();
							if (window.find) //for chrome, firefox
							{
								selection.removeAllRanges();

								let ns = window.find(match_name, aCaseSensitive =0, aBackwards =0, aWrapAround =1,
					aWholeWord =0, aSearchInFrames =is_firefox, aShowDialog =0);
								while (ns) {
									// this is very tedious but necessary, because the find and replace of multiple character words in DOI is not trivial.  
									console.log("found");
									let left_range = window.getSelection().getRangeAt(0).cloneRange();
									let right_range = window.getSelection().getRangeAt(0).cloneRange();
									let original_range = window.getSelection().getRangeAt(0).cloneRange();
									let if_range = window.getSelection().getRangeAt(0).cloneRange();

									let extended_begin_node;
									let extended_begin_focus;
									let extended_end_node;
									let extended_end_focus;
									let begin_node;
									let begin_focus;

									if (window.getSelection().anchorOffset < 3) {
										begin_node = window.getSelection().anchorNode.previousSibling;
										begin_focus = window.getSelection().anchorNode.previousSibling.length + window.getSelection().anchorOffset - 3;
										console.log("begin extended through focus " + begin_node);
									} else {

										begin_node = window.getSelection().anchorNode;

										begin_focus = window.getSelection().anchorOffset - 3;
										console.log("begin extended through node " + begin_node.data);

									}

									if (window.getSelection().focusOffset == window.getSelection().focusNode.length) {
										extended_end_node = window.getSelection().focusNode.nextSibling;
										extended_end_focus = 1;
										console.log("end extended through next node ");

									} else {
										extended_end_node = window.getSelection().focusNode;
										extended_end_focus = window.getSelection().focusOffset + 1;
										console.log("end extended through focus " + extended_end_node.data);

									}

									right_range.setEnd(extended_end_node, extended_end_focus);

									if_range.setStart(begin_node, begin_focus);
									let if_text = if_range.toString();
									const if_cha = if_text.slice(-match_name.length - 1, -match_name.length);
									const right_text = right_range.toString();
									console.log("if_text: " + if_text);
									console.log("if_cha: " + if_cha);

									const right_cha = right_text.slice(-1);
									console.log("right_text: " + right_text);
									console.log("right_cha: " + right_cha);

									if (if_cha == "|") {
										if (window.getSelection().anchorOffset < 13 && window.getSelection().anchorNode.previousSibling !== undefined) {
											extended_begin_node = window.getSelection().anchorNode.previousSibling;
											extended_begin_focus = window.getSelection().anchorNode.previousSibling.length + window.getSelection().anchorOffset - 13;
											console.log("begin extended through focus " + extended_begin_node);

										} else {

											extended_begin_node = window.getSelection().anchorNode;

											extended_begin_focus = window.getSelection().anchorOffset - 13;
											console.log("begin extended through node " + extended_begin_node.data);

										}

										left_range.setStart(extended_begin_node, extended_begin_focus);
										const left_text = left_range.toString();

										console.log("left_text: " + left_text);
										place_holder = /\#if\$\_.{0,10}\$\_\w*\b/.exec(left_text);
										console.log("place holder: " + place_holder);
										selection.removeAllRanges();
										window.find(place_holder);
										if_range = window.getSelection().getRangeAt(0);
										if_text = if_range.toString();

										console.log(" selection updated because of \| : " + window.getSelection().focusNode.data + "________" + if_text);

										match_mod(if_range);
										selection.removeAllRanges();
										// to match different conditions
									} else if (if_cha == "f") {
										if (right_cha == "|") {
											window.find(" ");

											if_range.setEnd(window.getSelection().focusNode, window.getSelection().focusOffset);
											match_mod(if_range);
											if_range.detach();
										} else {
											match_mod(if_range);
											if_range.detach();
										}

									} else if (if_cha == "," || if_cha == " ") {
										console.log("!!!! replace match text");

										original_range.deleteContents();
										original_range.insertNode(document.createTextNode(name));
										original_range.detach();
									} else if (if_cha == "!") {
										if (window.getSelection().anchorOffset < 4 && window.getSelection().anchorNode.previousSibling !== undefined) {
											extended_begin_node = window.getSelection().anchorNode.previousSibling;
											extended_begin_focus = window.getSelection().anchorNode.previousSibling.length + window.getSelection().anchorOffset - 4;
											console.log("begin extended through focus " + extended_begin_node);

										} else {

											extended_begin_node = window.getSelection().anchorNode;

											extended_begin_focus = window.getSelection().anchorOffset - 4;
											console.log("begin extended through node " + extended_begin_node.data);

										}

										left_range.setStart(extended_begin_node, extended_begin_focus);
										const left_text = left_range.toString();

										console.log("left_text with <!>: " + left_text);
										match_remove(left_range);

									} else
										{

										console.log("_____other cases");
										console.log("last character is: ", right_cha);
										match_mod(if_range);
										if_range.detach();
									}

									//window.find("#endif");

									if_range.detach();
									ns = window.find(match_name, aCaseSensitive =0, aBackwards =0, aWrapAround =1,
					aWholeWord =0, aSearchInFrames =is_firefox, aShowDialog =0);
								}
							} else if (document.body.createTextRange) //for ie etc.
							{
								let range_if = document.body.createTextRange();
								while (range_if.findText(match_name)) {
									range_if.pasteHTML('atest');
								}
							}

							console.log("check for loop, ends " + match_name);
							
						}

						for (let i = 0; i < unchecked.length; i++) {
						// here starts the unchecked box, it is basically as same as the checked options. It should be warped in a more general function, so no copy-paste is used. but I am too lazy to warp it and it only shows twice, so I think it is OK.
							let check_box_item = unchecked[i];
							//console.log(check_box );
							let name = check_box_item.split('_')[1];
							let match_name = "_________";
							try {
								match_name = prefix.concat(name.toUpperCase());
								console.log("uncheck for loop, starts " + match_name);
							} catch (e) {
								match_name = "__________"
							}

							const selection = window.getSelection();

							selection.removeAllRanges();
							if (window.find) //for chrome, firefox
							{

								let ns = window.find(match_name, aCaseSensitive =0, aBackwards =0, aWrapAround =1,
					aWholeWord =0, aSearchInFrames =is_firefox, aShowDialog =0);
								while (ns) {

									console.log("unchecked found");
									let left_range = window.getSelection().getRangeAt(0).cloneRange();
									let right_range = window.getSelection().getRangeAt(0).cloneRange();
									let original_range = window.getSelection().getRangeAt(0).cloneRange();
									let if_range = window.getSelection().getRangeAt(0).cloneRange();
									let remove_range = window.getSelection().getRangeAt(0).cloneRange();

									let extended_begin_node;
									let extended_begin_focus;
									let extended_end_node;
									let extended_end_focus;

									if (window.getSelection().anchorOffset < 3) {
										extended_begin_node = window.getSelection().anchorNode.previousSibling;
										extended_begin_focus = window.getSelection().anchorNode.previousSibling.length + window.getSelection().anchorOffset - 3;
										console.log("begin extended through focus " + extended_begin_node);
									} else {

										extended_begin_node = window.getSelection().anchorNode;

										extended_begin_focus = window.getSelection().anchorOffset - 3;
										console.log("begin extended through node " + extended_begin_node.data);

									}

									if (window.getSelection().anchorOffset < 3) {
										begin_node = window.getSelection().anchorNode.previousSibling;
										begin_focus = window.getSelection().anchorNode.previousSibling.length + window.getSelection().anchorOffset - 3;
										console.log("begin extended through focus " + extended_begin_node);
									} else {

										begin_node = window.getSelection().anchorNode;

										begin_focus = window.getSelection().anchorOffset - 3;
										console.log("begin extended through node " + extended_begin_node.data);

									}

									if (window.getSelection().focusOffset == window.getSelection().focusNode.length) {
										extended_end_node = window.getSelection().focusNode.nextSibling;
										extended_end_focus = 1;
										console.log("end extended through next node ");

									} else {
										extended_end_node = window.getSelection().focusNode;
										extended_end_focus = window.getSelection().focusOffset + 1;
										console.log("end extended through focus " + extended_end_node.data);

									}

									right_range.setEnd(extended_end_node, extended_end_focus);
									left_range.setStart(extended_begin_node, extended_begin_focus);
									if_range.setStart(begin_node, begin_focus);
									let if_text = if_range.toString();
									const right_text = right_range.toString();
									console.log("if_text: " + if_text);

									const right_cha = right_text.slice(-1);
									console.log("right_text: " + right_text);
									console.log("right_cha: " + right_cha);

									const left_text = left_range.toString();
									const if_cha = left_text.slice(-match_name.length - 1, -match_name.length);
									console.log("left_text: " + left_text);
									console.log("if_cha: " + if_cha);

									place_holder = /\#if\$\_.{0,10}\$\_\w*\b/.exec(left_text);
									console.log("place holder: " + place_holder);

									if (if_cha == "|") {

										selection.removeAllRanges();
										window.find(place_holder);
										if_range = window.getSelection().getRangeAt(0);
										if_text = if_range.toString();

										console.log(" selection updated because of \| : " + window.getSelection().focusNode.data + "________" + if_text);

										match_remove(if_range);
										selection.removeAllRanges();
									} else if (if_cha == "f") {
										if (right_cha == "|") {
											window.find(" ");

											if_range.setEnd(window.getSelection().focusNode, window.getSelection().focusOffset);
											match_remove(if_range);
											if_range.detach();
										} else {
											match_remove(if_range);
											if_range.detach();
										}

									} else if (if_cha == "," || if_cha == " ") {
										console.log("!!!! remove the matched text");
										remove_range.setEnd(window.getSelection().focusNode ,window.getSelection().focusOffset +1);

										remove_range.deleteContents();

										remove_range.detach();
									} else if (if_cha == "!") {
									
										if (window.getSelection().anchorOffset < 4 && window.getSelection().anchorNode.previousSibling !== undefined) {
											extended_begin_node = window.getSelection().anchorNode.previousSibling;
											extended_begin_focus = window.getSelection().anchorNode.previousSibling.length + window.getSelection().anchorOffset - 4;
											console.log("begin extended through focus " + extended_begin_node);

										} else {

											extended_begin_node = window.getSelection().anchorNode;

											extended_begin_focus = window.getSelection().anchorOffset - 4;
											console.log("begin extended through node " + extended_begin_node.data);

										}

										left_range.setStart(extended_begin_node, extended_begin_focus);
										const left_text = left_range.toString();

										console.log("left_text with <!>: " + left_text);
										match_mod(left_range);

									} else {

										console.log("_____other cases");
										console.log("last character is: ", right_cha);
										match_remove(if_range);
										if_range.detach();
									}

									//window.find("#endif");

									if_range.detach();
									ns = window.find(match_name, aCaseSensitive =0, aBackwards =0, aWrapAround =1,
					aWholeWord =0, aSearchInFrames =is_firefox, aShowDialog =0);
								}
							} else if (document.body.createTextRange) //for ie etc.
							{
								let range_if = document.body.createTextRange();
								while (range_if.findText(match_name)) {
									range_if.pasteHTML('');
								}
							}

							console.log("uncheck for loop, ends " + match_name);
						}
				
					}

					/**
					 * match a place holder and then modify it.
					 * @param {string} right_range - the extended keyword (includes #if and #endif) need to be replaced.
					 *
					 */

					function match_mod(in_range) {
						
						const text_after_if = (in_range.toString().slice(3));
						console.log("mach_mod function: after_endif: " + text_after_if);
						in_range.deleteContents();

						window.find("#endif" + text_after_if);

						let end_if_range = window.getSelection().getRangeAt(0);
						end_if_range.deleteContents();
						end_if_range.detach();

					}
					/**
					 * match a place holder and then remove it.
					 * @param {string} right_range - the extended keyword (includes #if and #endif) need to be removed.
					 *
					 */
					function match_remove(right_range) {
						
						const text_after_if = (right_range.toString().slice(3));
						console.log("mach_remove function: after_endif: " + text_after_if);

						window.find("#endif" + text_after_if);

						right_range.setEnd(window.getSelection().focusNode, window.getSelection().focusOffset);
						right_range.deleteContents();
						right_range.detach();

					}
					
					
					function HtmlEncode(s)
						{
						  var el = document.createElement("div");
						  el.innerText = el.textContent = s;
						  s = el.innerHTML;
						  return s;
						}
					
					
					
					
					/**
					 * replace a keyword by its formal meaning.
					 * @param {string} before - the keyword need to be replaced.
					 * @param {string} after - the formal text represented by the keyword.
					 */
					function find_replace(option) {
						


						const selection = window.getSelection();
						selection.removeAllRanges();

						

						const before = Object.keys(option)[0];
						const after = option[before];
						console.log("before: " + before +", after: " + after);
						
						
						if (window.find) //for chrome, firefox
						{
						
						console.log("window.find is: " + window.find );

							while (window.find(before, aCaseSensitive =0, aBackwards =is_firefox, aWrapAround =is_firefox,
								aWholeWord =0, aSearchInFrames =0, aShowDialog =0 )) {
								
								let last_letter;
								
								let range = window.getSelection().getRangeAt(0);
								window.getSelection().focusNode;
								if (window.getSelection().focusNode.length == window.getSelection().focusOffset) {
								console.log("first if : " + window.getSelection().focusNode.length );
									last_letter = "";
									
								
								} else {
									range.setEnd(window.getSelection().focusNode, window.getSelection().focusOffset + 1);
									last_letter = range.toString().slice(-1);
								}

								if (window.getSelection().anchorOffset == 0) {
									const old_offset = window.getSelection().anchorOffset;
									console.log("no enough range");
									range.setStart(window.getSelection().anchorNode.previousSibling, window.getSelection().anchorNode.previousSibling.length - 1);
									first_letter = range.toString().slice(0, 1);
								} else {
									console.log("enough range" + window.getSelection().anchorOffset);
									range.setStart(window.getSelection().anchorNode, window.getSelection().anchorOffset - 1);
									first_letter = range.toString().slice(0, 1);
								}

								if (/[,]/.test(last_letter)) {
									console.log(range.toString(), "last letter is comma: " + last_letter);
									range.deleteContents();
									console.log(encodeURIComponent(after));
									let newText = document.createElement(before.split("_")[1] + "-to-");
									newText.innerHTML= newText.textContent = '<span class = "text-warning"> '  + first_letter + HtmlEncode(after) + last_letter + '</span>';
									range.insertNode(newText);

								}else if (/[a-zA-Z\|\$\!]/.test(last_letter)) {
									console.log(range.toString(), "last letter " + last_letter);

								} else if (/[a-zA-Z\|\$\!]/.test(first_letter)) {
									console.log(range.toString(), "first letter " + first_letter);
								} else {
									console.log(range.toString(), "deleted with last letter: " + last_letter);
									range.deleteContents();
									console.log(encodeURIComponent(after));
									let newText = document.createElement(before.split("_")[1] + "-to-");
									newText.innerHTML= newText.textContent = '<span class = "text-warning"> '  + first_letter + HtmlEncode(after) + last_letter + '</span>';
									range.insertNode(newText);

								}
							
							}
						} else if (document.createTextRange) //for ie etc.
						{
							var range = document.createTextRange();
							while (range.findText(before)) {
								range.pasteHTML(after);
							}
						}
						
					}
					
					//progress-bar
					var x = 0;
					
					
					window.addEventListener('beforeunload', function (e) {
					  // Cancel the event
					  e.preventDefault(); // If you prevent default behavior in Mozilla Firefox prompt will always be shown
					  // Chrome requires returnValue to be set
					  e.returnValue = '';
					});
					
					// initialization
					window.addEventListener("load", function () {
						if (cached_template==null  ){
						cached_template = doc_name;
						}else if (cached_template=="user-defined"){
						doc_name="user-defined";
						document.getElementById("user-defined").innerHTML=window.localStorage.getItem("fulltext_user");
						
								
						
						}
						if (window.localStorage.getItem("cookie_consent") !== "yes"){
						
						document.getElementById("modal-button").click();
						
						
						}
						
						window.localStorage.setItem("user_info", "visited" );
						load_dmp(reload_answers, name = cached_template);
						console.log("current url is: " + window.location.href);
						const urls = window.location.href.split("?");
						if (urls.length > 1) {

							fetch(urls[1])
							.then(response => response.json())
							.then(json => {
								saved_a = json;

								

							})
							


						}
						
						const uploadJson = document.getElementById("upload");
						uploadJson.addEventListener("change", reloadJson, false);

						function reloadJson() {

							uploadJson.files[0].text().then(text => (saved_a = JSON.parse(text))
							
							
							).then( saved_a => ( load_dmp(reload_answers, name = cached_template) ) );

							console.log(JSON.stringify(saved_a));
				
						}
					});

				function copy_doc(){
				
				
				
				const compared_list = compare_answers(temp_a ,saved_a);
				let warning_element_children =Array.from(Array(99).keys());
				
				if ((compared_list[2] == compared_list[0]) && (compared_list[3] == compared_list[1]) ) {
					issue_warning();
					 warning_element_children = document.getElementById("warningText").children;
						if (warning_element_children[0]== undefined){
						copy_doc1();
						}else{
						update_reminder(compared_list, warning_element_children);
						}
								 
				} else{
				
				update_reminder(compared_list, warning_element_children);
				}
				
				

				
				}
				
				function copy_doc1(){
				const text= document.getElementById("doc3");
				const selection = window.getSelection();
				const range = document.createRange();
				range.selectNodeContents(text);
				selection.removeAllRanges();
				selection.addRange(range);
				try {
					return document.execCommand("copy");  // Security exception may be thrown by some browsers.
					const selection = window.getSelection();
					selection.removeAllRanges();
					
				}
				catch (error) {
					console.warn("Copy to clipboard failed.", error);
					return false;
				}
				
				
				}
				
				function printDiv(){
					const compared_list = compare_answers(temp_a ,saved_a);
					let warning_element_children =Array.from(Array(99).keys());
					
					if ((compared_list[2] == compared_list[0]) && (compared_list[3] == compared_list[1]) ) {
						issue_warning();
						 warning_element_children = document.getElementById("warningText").children;
							if (warning_element_children[0]== undefined){
								try {
									return printDiv1();  // Security exception may be thrown by some browsers.
									const selection = window.getSelection();
									
								}
								catch (error) {
									console.warn("print failed.", error);
									return false;
								}
							}else{
							update_reminder(compared_list, warning_element_children);
							}
									 
					} else{
					
					update_reminder(compared_list, warning_element_children);
					}
					
				
				}
				
		function printDiv1() {
			var divContents = document.getElementById("doc3").innerHTML;
			var a = window.open('', '', 'height=800, width=600');
			a.document.write('<html>');
			a.document.write('<body > ');
			a.document.write(divContents);
			a.document.write('</body></html>');
			a.document.close();
			a.print();
		}

		function update_reminder(compared_list, warning_element_children) {
					const question_n = compared_list[1]+compared_list[0]-compared_list[2]-compared_list[3];
					const issue_n = warning_element_children.length;
					let question_reminder, issue_reminder;
					switch (question_n){
						case 0: question_reminder = "Congrats, all questions are answered.";
						break;
						case 1: question_reminder = "Only 1 question is left to be answered. <br> Please click the next button to finish answering the questions.";
						break;
						default: 
						question_reminder = "There are "+ String(question_n) +" questions need to be answered. <br> Please click the \"next\" button to finish answering the questions.";
							
					
					}
					switch (issue_n){
						case 0: issue_reminder = "Congrats, all warnings are resolved.";
						break;
						case 1: issue_reminder = "Only 1 issue left.";
						break;
						// 99 means that there are questions not answered.
						case 99: issue_reminder = " ";
						break;
						default: 
						issue_reminder = "There are "+ String(issue_n) +" issues need to be resolved." +' If you are aware of all the issues and you want to remove all the warnings, please click:  <button type="button" class="badge btn-secondary"  onclick="remove_all_warnings()" > remove all the warnings</button>';
							
					
					}
							let reminders = [
					{
							title: "Question reminders",
							content:  question_reminder +" <br> " + issue_reminder+ ' <br>  <br> If you need a <b>unchecked</b> or <b>unfinished</b> DMP, <br> you can also <a class="badge btn-secondary" href="#" onclick="copy_doc1()">copy</a> or <a class="badge btn-secondary" href="#" onclick="printDiv1()">print</a> it anyway.'
							
							
							//
						}
					];
					//console.log(compared_list[4].entries());
					for (const [index, element] of compared_list[4].entries()){
						const ele_id = "form_" + element.split("_")[1];
						
						//const ele = document.getElementById(ele_id);
						console.log("element id is " + ele_id); // ,
						
						const id_and_content= {id: String(ele_id.toLowerCase()), content : "this question was default value or empty, please revise it "  };
						reminders.push(id_and_content);
						console.log("id_is " + Object.values(id_and_content));
						
					
					
					}
					
					console.log(compared_list[5].entries());
					for (const [index, element] of compared_list[5].entries()){
						const ele_id = "form_" + element;
						
						//const ele = document.getElementById(ele_id);
						console.log("element id is " + ele_id); // ,
						
						const id_and_content= {id: String(ele_id.toLowerCase()), content : "this question was default value or empty, please revise it "  };
						reminders.push(id_and_content);
						console.log("id_is " + Object.values(id_and_content));
						
					
					
					}
					
					
					let reminder = new Tour(reminders);
				
				
					reminder.show();
		
		}
		</script>			
		<body  class="overflow-hidden" style=" height:100vh;width:100vw; padding-right: 0px "  >
		
		<div class="container-fluid" style=" height:100vh;width:100vw; padding-right: 0px ">

		<div class=" row g-3 " style= "  padding-left:12px; height:100vh;width:100vw; ">
			
			<div   class="col-8" id="doc"  style="   overflow-x: hidden;overflow-y: hidden; padding-right: 0px; ">
				<div  class="col-12" id="doc2" style="height:100vh; overflow-x: hidden;  padding-right: 0px;  ">					
			    <a href="#" data-toggle="tooltip" onclick="copy_doc()" title="click to copy the document" style="top:2vh;left:63vw;position:fixed; text-shadow: 2px 2px 4px 16px;border-radius: 2px;"  ><span  class="clip sticky-top float-end"><svg xmlns="http://www.w3.org/2000/svg" width="22" height="22" fill="currentColor" class="bi bi-clipboard-check" viewBox="0 0 16 16">
				<path fill-rule="evenodd" d="M10.854 7.146a.5.5 0 0 1 0 .708l-3 3a.5.5 0 0 1-.708 0l-1.5-1.5a.5.5 0 1 1 .708-.708L7.5 9.793l2.646-2.647a.5.5 0 0 1 .708 0z"/>
				<path d="M4 1.5H3a2 2 0 0 0-2 2V14a2 2 0 0 0 2 2h10a2 2 0 0 0 2-2V3.5a2 2 0 0 0-2-2h-1v1h1a1 1 0 0 1 1 1V14a1 1 0 0 1-1 1H3a1 1 0 0 1-1-1V3.5a1 1 0 0 1 1-1h1v-1z"/>
				<path d="M9.5 1a.5.5 0 0 1 .5.5v1a.5.5 0 0 1-.5.5h-3a.5.5 0 0 1-.5-.5v-1a.5.5 0 0 1 .5-.5h3zm-3-1A1.5 1.5 0 0 0 5 1.5v1A1.5 1.5 0 0 0 6.5 4h3A1.5 1.5 0 0 0 11 2.5v-1A1.5 1.5 0 0 0 9.5 0h-3z"/>
				</svg>
					</span></a>
					<div id = "doc3" class="row-auto overflow-auto" style="margin-right:0px">

					
					
					
					</div>
				</div>
			</div>
			<div   class="col-4 "  id="interface"  style="height:calc(100% - 80px); padding-right: 0px;  padding-left: 0px;overflow-x: hidden;overflow-y: hidden; ">

					
				
				

	
					
			<div class=" "   style=" overflow-x: hidden; padding-right: 0px; "	>

			<nav id="navbar-example2" class=" nav-fill navbar navbar-light bg-light " style="height:100px;" >
					  
					  <ul class="nav nav-fill nav-pills col-12" >
						<li class="nav-item">
						  <a class="nav-link text-center" style="font-size: calc( 1.1vw);padding:4px 8px 4px 8px" href="#scrollspyHeading1">basic <br> info</a>
						</li>
						<li class="nav-item">
						  <a class="nav-link text-center" style="font-size: calc(1.1vw); padding:4px 8px 4px 8px;" href="#scrollspyHeading2">data <br> type</a>
						</li>
						<li class="nav-item">
						  <a class="nav-link text-center" style="font-size: calc( 1.1vw);padding:4px 8px 4px 8px;" href="#scrollspyHeading3">data <br> volume</a>
						</li>
						<li class="nav-item">
						  <a class="nav-link text-center" style="font-size: calc( 1.1vw);padding:4px 8px 4px 8px;" href="#scrollspyHeading4">data <br> standard</a>
						</li>
						<li class="nav-item">
						  <a class="nav-link text-center" style="font-size: calc( 1.1vw);padding:4px 8px 4px 8px;" href="#scrollspyHeading5">data <br> analysis</a>
						</li>
					
					  </ul>
					  
					</nav>					
			
			
		<div id="scrollspy-element" data-bs-spy="scroll" data-bs-target="#navbar-example2" data-bs-offset="100" class="scrollspy-example" tabindex="0"   style="padding-left: 10px;"	>
		<div id="intro-tour-input" >
		<div  id= "warningText">
		</div>
		<a href="javascript:void(0);" style="font-size:small" onclick="tour.show()" ><span class="badge btn-tutorial text-dark sticky-top float-end">tutorial</span></a>
		<h2 id="scrollspyHeading1">1 Basic Information:</h2>
				<h3 >1.1 Give the project name or acronym to be used:</h3>
				
				<br>
			 
					<form class=""  style="margin-left:0px" id="form_project">
					<label for="input_project"></label>
					<input style="width: 26vw;" class="col-auto" type="text" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();" 
					   id="input_project"
					   name="$_PROJECT"
					   placeholder="the project name" 
					   onkeydown=' if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.nextElementSibling.nextElementSibling.children[1]);this.parentElement.nextElementSibling.nextElementSibling.children[1].focus();return false;}'
					   >
					<input  class="btn btn-primary btn-sm col-auto d-none"" type="button" id="input_project_submit"
					   value="submit"
					   onclick="replace(this)" >
					   <input class="btn btn-secondary btn-sm d-none" type="button"
					   value="undo" id="input_project_undo"
					   onclick="undo(this);" >
			  
					</form>
					<br>
					
					<form id="form_studyobject">
					<label for="input_studyobject"><h3 >1.2 Give the object 
					<a href="#" data-toggle="tooltip" title="click to see some example study objects" tabindex="-1">
					<span class=" text-success" tabindex="-1" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-obj" aria-controls="offcanvas-obj"> <svg xmlns="http://www.w3.org/2000/svg" tabindex="-1" width="16" height="16" fill="currentColor" class="bi bi-question-circle" viewBox="0 0 16 16">
					  <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/>
					  <path d="M5.255 5.786a.237.237 0 0 0 .241.247h.825c.138 0 .248-.113.266-.25.09-.656.54-1.134 1.342-1.134.686 0 1.314.343 1.314 1.168 0 .635-.374.927-.965 1.371-.673.489-1.206 1.06-1.168 1.987l.003.217a.25.25 0 0 0 .25.246h.811a.25.25 0 0 0 .25-.25v-.105c0-.718.273-.927 1.01-1.486.609-.463 1.244-.977 1.244-2.056 0-1.511-1.276-2.241-2.673-2.241-1.267 0-2.655.59-2.75 2.286zm1.557 5.763c0 .533.425.927 1.01.927.609 0 1.028-.394 1.028-.927 0-.552-.42-.94-1.029-.94-.584 0-1.009.388-1.009.94z"/>
					</svg> </span></a> to be studied:</h3> </label>

					
					<input style="width: 26vw;" class="col-auto" type="text" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);"  onfocusout="this.nextElementSibling.click();" 
					   id="input_studyobject"
					   name="$_STUDYOBJECT"
					   placeholder="the research object/target" 
					   onkeydown=' if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.nextElementSibling.children[1]);this.parentElement.nextElementSibling.children[1].focus();return false;}'>
					
					<input class="btn btn-primary btn-sm d-none" type="button"
						   value="submit" id="input_studyobject_submit"
						   onclick="replace(this)">
						   
					
						   <input class="btn btn-secondary btn-sm d-none" type="button"
						   value="undo"
						   onclick="undo(this);">
			  
					</form>
					
					<form id="form_projectaim"> 
					<label class= "" for="input_projectaim"> <h3 >and the project aim   <span class=" text-success" type="button" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-aim" aria-controls="offcanvas-aim">(what do you want to achieve?) <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-question-circle" viewBox="0 0 16 16">
					  <path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z"/>
					  <path d="M5.255 5.786a.237.237 0 0 0 .241.247h.825c.138 0 .248-.113.266-.25.09-.656.54-1.134 1.342-1.134.686 0 1.314.343 1.314 1.168 0 .635-.374.927-.965 1.371-.673.489-1.206 1.06-1.168 1.987l.003.217a.25.25 0 0 0 .25.246h.811a.25.25 0 0 0 .25-.25v-.105c0-.718.273-.927 1.01-1.486.609-.463 1.244-.977 1.244-2.056 0-1.511-1.276-2.241-2.673-2.241-1.267 0-2.655.59-2.75 2.286zm1.557 5.763c0 .533.425.927 1.01.927.609 0 1.028-.394 1.028-.927 0-.552-.42-.94-1.029-.94-.584 0-1.009.388-1.009.94z"/>
					</svg> </span></h3 >
					</label>
					<textarea type="text" style="width: 26vw;" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
					   id="input_projectaim"
					   name="$_PROJECTAIM"
					   placeholder="project aim" rows="3" style="width: 26vw;"
					   onkeydown=' if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.nextElementSibling.children[1]);this.parentElement.nextElementSibling.children[1].focus();return false;}'></textarea>
					<input class="btn btn-primary btn-sm d-none" type="button"
					   value="submit"
					   onclick="replace(this)">
					   <input class="btn btn-secondary btn-sm d-none" type="button"
					   value="undo"
					   onclick="undo(this);">
		  
				</form>
			
			
			
			
			<form class="" style="margin-left:0px" id="form_dataofficer">
				<label for="input_dataofficer"><h3 >1.3 Give the data officer name:</h3></label>
				<input style="width: 26vw;" class="col-auto" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
					   type="text"
					   id="input_dataofficer"
					   name="$_dataofficer"
					   placeholder="data officer name"
					   onkeydown=' if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.nextElementSibling.nextElementSibling.nextElementSibling.children[0]);this.parentElement.nextElementSibling.nextElementSibling.nextElementSibling.children[0].focus();return false;}'>
					<input class="btn btn-primary btn-sm col-auto d-none" type="button" id="input_dataofficer_submit"
						   value="submit"
						   onclick="replace(this)">
					<input class="btn btn-secondary btn-sm d-none" type="button"
						   value="undo"
						   onclick="undo(this);">
		  
			</form>
			
			 <br>
			 <h3 >1.4 Chose some additional information</h3>
				<div class="form-data_type form-check  " id = "form_checkbox_1"  onfocusout="this.nextElementSibling.click();">
				  <input class="form-check-input checkbox_1" type="checkbox" value="1" id="check_protect" >
				  <label class="form-check-label" for="check_protect">
					The data is protected.
				  </label>
				  <br>
					<input class="form-check-input checkbox_1" type="checkbox" value="2" id="check_dataplant"  data-bs-toggle="collapse" data-bs-target="#no_dataplant" aria-expanded="false" aria-controls="no_dataplant" role="button" checked>
					
				  <label class="form-check-label" for="check_dataplant">
					Yes! I will use <a href="https://nfdi4plants.github.io/" data-toggle="tooltip" title="An NFDI consortium of plant research." data-original-title="An NFDI consortium of plant research.">DataPLANT</a>.
				  </label>

				  <br>
				  <input class="form-check-input checkbox_1" type="checkbox" value="3" id="check_update" data-bs-toggle="collapse" data-bs-target="#q_updatemonth" aria-expanded="false" aria-controls="q_updatemonth" role="button"  >
				  <label class="form-check-label" for="check_update">
					This project will be updated.
				  </label>
				  <br>
				  

				  <input class="form-check-input checkbox_1" type="checkbox" value="4" id="check_previousprojects" 
				  data-bs-toggle="collapse" data-bs-target="#q_previousprojects" aria-expanded="false" aria-controls="q_previousprojects" role="button"  >
				  <label class="form-check-label" for="check_previousprojects">
					This project has previous projects.
				  </label>

				  <br>
					<input class="form-check-input checkbox_1" type="checkbox" value="5" id="check_industry" >
				  <label class="form-check-label" for="check_industry">
					This is an industry project.
				  </label>
					<br>
				  
				  <input class="form-check-input checkbox_1" type="checkbox" value="6" id="check_proprietary" 
				  data-bs-toggle="collapse" data-bs-target="#q_proprietary" aria-expanded="false" aria-controls="q_proprietary" role="button"  >
				  <label class="form-check-label" for="check_proprietary">
					This project use <a href="#" data-toggle="tooltip" title="software that legally remains the property of the organization, group, or individual who created it." data-original-title="">proprietary software</a>.
				  </label>
				<br>
				   
				  <input class="form-check-input checkbox_1" type="checkbox" value="7" id="check_partners">
				  <label class="form-check-label" for="check_partners">
					This project has partners.
				  </label> 
				  <br>
					<input class="form-check-input checkbox_1" type="checkbox" value="8" id="check_EU" >
				  <label class="form-check-label" for="check_EU">
					EU project.
				  </label> 
				  
				  
				</div> 

		
			<input class=" btn btn-primary form-check btn-sm d-none" type="button" id="form-check-input"
							data-checkbox = "checkbox_1"
						   value="submit"
						   onclick="get_option(this)" > 
						   <input class="btn btn-secondary btn-sm d-none" type="button"
						   value="undo"
						   onclick="undo(this);">

		


		  <div class="collapse" id="no_dataplant"  onfocusout="this.nextElementSibling.click();">

			<p>*** this question is a mock up (takes no effect on the document)
			Where will you submit your data as endpoints? </p>
			
			ENA, GEO, NCBI, genbank, pride, proteomexchange, zenodo, dataDyrad
			<div class="form-data_type form-check" >
		  <input class="form-check-input q_5th_b" type="checkbox" value="1" id="check_ena" >
		  <label class="form-check-label" for="check_ena">
			ENA
		  </label>
		</div> 
		<div class="form-data_type form-check">
		  <input class="form-check-input q_5th_b" type="checkbox" value="2" id="check_geo" checked>
		  <label class="form-check-label" for="check_geo">
			GEO
		  </label>
		</div> 
		<div class="form-data_type form-check">
		  <input class="form-check-input q_5th_b" type="checkbox" value="3" id="check_NCBI" checked>
		  <label class="form-check-label" for="check_NCBI">
			NCBI
		  </label>
		</div>  
		  
		<div class="form-data_type  form-check">
		  <input class="form-check-input q_5th_b" type="checkbox" value="4" id="check_genbank" checked>
		  <label class="form-check-label" for="check_genbank">
			genbank
		  </label>
		</div> 
			  </div>
			<input class=" btn btn-primary form-check btn-sm d-none" type="button" id="small-form-check-input"
							data-checkbox = "checkbox_5"
						   value="submit"
						   onclick="get_option(this)" > 



					<div class="collapse" id="q_updatemonth">
					<br>
					<form class="" id="form_updatemonth">

					    <label for="intput_updatemonth">At which month will your project updated?</label>
						<input style="width: 26vw;" type="text" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
						    id="intput_updatemonth"
						    name="$_UPDATEMONTH"
						    placeholder="month of update"
						    onkeydown='if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.parentElement.nextElementSibling.children[1].children[1]);this.parentElement.parentElement.nextElementSibling.children[1].children[1].focus();return false;}'>
						   <input class="btn btn-primary btn-sm d-none" type="button"
						    value="submit" id="intput_updatemonth_submit"
						    onclick="replace(this)" data-bs-toggle="tooltip" data-bs-placement="top" title="submit the checkbox first">
						   <input class="btn btn-secondary btn-sm d-none" type="button"
						    value="undo"
						    onclick="undo(this);">
					</form>
									
					</div>


					  <div class="collapse" id="q_previousprojects">
					  <br>
					<form class="" id="form_previousprojects">
							<label for="intput_previousprojects">The name of your previous project is:</label>
							<input style="width: 26vw;" type="text" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
								   id="intput_previousprojects"
								   name="$_PREVIOUSPROJECTS"
								   placeholder="previous project here"
								   onkeydown='if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.parentElement.nextElementSibling.children[1].children[1]);this.parentElement.parentElement.nextElementSibling.children[1].children[1].focus();return false;}'>
								<input class="btn btn-primary btn-sm d-none" type="button"
									   value="submit" id="intput_previousprojects_submit"
									   onclick="replace(this)" data-bs-toggle="tooltip" data-bs-placement="top" title="submit the checkbox first">
									   <input class="btn btn-secondary btn-sm d-none" type="button"
									   value="undo"
									   onclick="undo(this);">
					  
						</form>
					</div>


				  <div class="collapse" id="q_proprietary">
				  <br>
				<form class="" id ="form_proprietary" >
						<label for="input_proprietary">The name of your proprietary software is:</label>
						
						<input style="width: 26vw;" type="text" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
							   id="input_proprietary"
							   name="$_proprietary"
							   placeholder=" proprietary software "
							   onkeydown='if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.parentElement.nextElementSibling.children[1].children[1]);this.parentElement.parentElement.nextElementSibling.children[1].children[1].focus();return false;}'>
							<input class="btn btn-primary btn-sm d-none" type="button"
								   value="submit" id="input_proprietary_submit"
								   onclick="replace(this)" data-bs-toggle="tooltip" data-bs-placement="top" title="submit the checkbox first">
								   <input class="btn btn-secondary btn-sm d-none" type="button"
								   value="undo"
								   onclick="undo(this);">
				  
					</form>
				</div>



			  

			

		 
		 
		 
				
			<div class=" "  id="form_checkbox_2"   onfocusout="this.nextElementSibling.click();">
			<h2 id="scrollspyHeading2">2. What kind of data will you handle:</h2>
				  <div class="form-data_type form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="1" id="check_image" >
				  <label class="form-check-label" for="check_image">
					image
				  </label>
				</div> 
				<div class="form-data_type form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="2" id="check_RNAseq" >
				  <label class="form-check-label" for="check_RNAseq">
					RNAseq
				  </label>
				</div> 
				<div class="form-data_type form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="3" id="check_genomic" >
				  <label class="form-check-label" for="check_genomic">
					other NGS (genomics)
				  </label>
				</div>  
				  
				<div class="form-data_type  form-check">
						  <label class="form-check-label" for="check_metabolomic">
					metabolomics
				  </label>
				  <input class="form-check-input checkbox_2" type="checkbox" value="4" id="check_metabolomic" >

				</div> 


				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="5" id="check_proteomic" >
				  <label class="form-check-label" for="check_proteomic">
					proteomics
				  </label>
				</div> 

				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="6" id="check_targeted" >
				  <label class="form-check-label" for="check_targeted">
					targeted assays (e.g. glucose and fructose content)
				  </label>
				</div> 
				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="7" id="check_models" >
				  <label class="form-check-label" for="check_models">
					models
				  </label>
				</div> 
				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="8" id="check_code" >
				  <label class="form-check-label" for="check_code">
					code
				  </label>
				</div> 
				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="9" id="check_excel" >
				  <label class="form-check-label" for="check_excel">
					<a href= "#" data-toggle="tooltip" title="includes other tabular data (e.g. .csv, .tab)" data-original-title="An NFDI consortium of plant research." >excel </a>
				  </label>
				</div> 
				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="10" id="check_cloned-DNA" >
				  <label class="form-check-label" for="check_cloned-DNA">
					cloned DNA
				  </label>
				</div> 
				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="11" id="check_phenotypic" >
				  <label class="form-check-label" for="check_phenotypic">
					phenotypic
				  </label>
				</div> 

				<div class="form-data_type  form-check">
				  <input class="form-check-input checkbox_2" type="checkbox" value="12" id="check_genetic" >
				  <label class="form-check-label" for="check_genetic"> genetics
				  </label>
				</div> 
			</div> 
		
			

		<input class="btn btn-primary btn-sm d-none" type="button" id="check_box1"
						   value="submit data type" data-checkbox = "checkbox_2"
						   onclick="get_option(this)" class=> 
						   <input class="btn btn-secondary btn-sm d-none" type="button"
						   value="undo"
						   onclick="undo(this);">
		  </div>
		 <br>
		  
		  
		 <h2 id="scrollspyHeading3">3. How much data you will probably generate:</h2>
		
		  <label for="input_rawdata"><h3 >3.1 Raw data volume is :</h3> </label>
			<form class=""  style="margin-left:0px" id="form_rawdata">
				
				<div class= "align-bottom">
				<input style="width: 24vw;" type="text" class="col-auto "
					   id="input_rawdata" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
					   name="$_RAWDATA"
					   placeholder="raw data volume (in GB)"
					   onkeydown='if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.parentElement.nextElementSibling.nextElementSibling.children[0].children[0]);this.parentElement.parentElement.nextElementSibling.nextElementSibling.children[0].children[0].focus();return false;}'> 
					   GB
					<input class="btn btn-primary btn-sm col-auto ms-4 align-middle d-none" type="button" id="input_rawdata_submit" 
						   value="submit"
						   onclick="replace(this)">
						   <input class="btn btn-secondary btn-sm d-none align-middle" type="button"
						   value="undo"
						   onclick="undo(this);">
				</div>
			</form>
			<label for="input_deriveddata"><h3 >3.2 Derived data volume is :</h3> </label>
			<form class=""  style="margin-left:0px" id="form_deriveddata">
				
				<div class= "align-bottom">
				<input style="width: 24vw;" type="text" class="col-auto "
					   id="input_deriveddata" onfocusin="const oldele = this;setTimeout(() => {oldele.select();}, 0);" onfocusout="this.nextElementSibling.click();"
					   name="$_DERIVEDDATA"
					   placeholder="derived data volume (in GB)"
					   onkeydown='if (event.keyCode == 13){event.preventDefault(); event.stopPropagation();console.log(this.parentElement.parentElement.nextElementSibling.children[1].children[0]);this.parentElement.parentElement.nextElementSibling.children[1].children[0].focus();return false;}'> GB
					<input class="btn btn-primary btn-sm col-auto ms-4 align-middle d-none" type="button"
						   value="submit" id="input_deriveddata_submit"
						   onclick="replace(this)">
						   <input class="btn btn-secondary btn-sm d-none" type="button"
						   value="undo"
						   onclick="undo(this);">
				</div>
			</form>
		  
		 
		   
		   
		   
		   <div class=" "  id="form_checkbox_3"  onfocusout="this.nextElementSibling.click();" >
		   <h2 id="scrollspyHeading4">4. Are there any standards that are relevant for you?</h2>
			
			
			<div class="form-data_type  form-check" >
		  <input class="form-check-input checkbox_3" type="checkbox" value="9" id="check_MIAPPE" >
		  <label class="form-check-label" for="check_MIAPPE">
			MIAPPE
		  </label>
		</div> 
		<div class="form-data_type  form-check">
		  <input class="form-check-input checkbox_3" type="checkbox" value="10" id="check_MinSeq" >
		  <label class="form-check-label" for="check_MinSeq">
			MinSEQe
		  </label>
		</div> 
		<br>
			 <h3>4.1 Will you adhere to any high level metadata submission standards?</h3>
		<div class="form-data_type form-check">
		  <input class="form-check-input checkbox_3" type="checkbox" value="1" id="check_dublin_core" checked>
		  <label class="form-check-label" for="check_dublin_core">
			Dublin Core
		  </label>
		</div> 
		<div class="form-data_type form-check">
		  <input class="form-check-input checkbox_3" type="checkbox" value="2" id="check_marc_21" >
		  <label class="form-check-label" for="check_marc_21">
			MARC 21
		  </label>
		</div> 	
<br>		
			 <h3>4.2 When will you make your data public?</h3> 

			<div class="form-data_type form-check" >
			 <input class="form-check-input  checkbox_3" type="checkbox" id="check_early" data-bs-toggle="collapse" data-bs-target="#q6_2" aria-expanded="false" aria-controls="q6_2" role="button" checked >
		  <label class="form-check-label" for="check_early">Will you publish your data early?</label>
		</div>
		 
		<div class="collapse" id="q6_2">
			<div class="form-data_type form-check">
			<input class="form-check-input  checkbox_3" type="checkbox" id="check_ipissue"  >
		  <label class="form-check-label" for="ip_issue">Are there intellectual property (IP) issues to adhere to (e.g. patents you might want to file) or private partners involved whose IP has to be taken into account</label>
			</div>
		
		</div> 
		<br>
		</div>
			<input class="btn btn-primary form-check btn-sm d-none" type="button" id="form-check-input" value="submit" data-checkbox= "checkbox_3" onclick="get_option(this)">
								   <input class="btn btn-secondary btn-sm d-none" type="button"
						   value="undo"
						   onclick="undo(this);">
			
		   
		   
			
			  <h2 id="scrollspyHeading5">5. Do you need visualization in the project?</h2>
			 
			 <form class="col-12"  style="margin-left:0px" id="form_visualization">

				<input  type="text" 
					   id="input_visiualization"
					   name="$_VISUALIZATION"
					   placeholder=" " class="d-none" value= "and visualization" >
					<input class="btn btn-primary btn-sm" type="button"
						   value="yes" class="col-auto"
						   onclick="replace(this)">
				<input type="text" 
					   id="submit_VISUALIZATION"
					   name="$_VISUALIZATION"
					   placeholder=" " class="d-none" value= " " >	

					<input class="btn btn-primary btn-sm" type="button"
						   value="no" class="col-auto"
						   onclick="replace(this)">
						   </input>
						   <input class="btn btn-secondary btn-sm d-none" data-value="visualization" type="button"
						   value="undo"
						   onclick="undo(this);">
						
				

		  
			</form>
			

		<br>
<br><br><br><br><br>
		</div>
		</div>
		
		
		
		
		
		
		
		<div class ="row-12 position-absolute bottom-0 " style="background:lightgrey; height:80px; width:32.6vw" >
		


			<div class="progress"  style= "padding: 0px;">
				<div name="progress_bar" class="progress-bar progress-bar-striped" role="progressbar"  aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" id= "p-bar" style= "padding: 0px;">  
				</div>
			</div>
			
			
			
				
		
		
			<div class= "justify-content-center row gx-2 pt-2 ">
			
			
			<div class="btn-group-sm  d-md-none dropup col-auto" style= "padding-left: 12px; ">
			<button type="button" class="btn btn-secondary dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" id= "menu" >
				menu
			  </button>
				<ul class="dropdown-menu">
				<li>
				<button id="detailed_dmp" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="dmp1")'>detailed DMP (EU)</button>
				</li>
				<li>
				<button id="detailed_dmp" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="dfg-dmp")'>DFG DMP (DE)</button>
				</li>
				<li>
				<button id="guide_save" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="practical-guide")'>practice guide</button>
				</li>
				<li>
				<button id="proposal_save" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="proposal")'>proposal</button>
				</li>
				<li>
				<button id="template-input" class=" dropdown-item" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-template-input" aria-controls="offcanvas-template-input" onclick='document.getElementById("template-input-content").innerHTML=window.localStorage.getItem("fulltext_user");' >user defined template</button>
				</li>
			  <li><hr class="dropdown-divider"></li>
			  <li>
						<label for="upload-dropdown" class="form-label dropdown-item">upload json
						<input class="form-control d-none" type="file" id="upload-dropdown">
						</label>
				</li>
				<li>
						<button id="clear_cache" class=" dropdown-item" onclick = 'window.localStorage.clear();document.location.reload()'>clear cache</button>

				</li>
			  <li><hr class="dropdown-divider"></li>
			  <li>
				  <button id="copy_doc" class=" dropdown-item" onclick = 'copy_doc()'>copy document</button>
				
				  
				  <button id="test" class="d-none dropdown-item" onclick = "test()">debug test</button>
				  </li>
				  <li>
				  <button id="json_save" class="dropdown-item" onclick = "save_json()" >save answers to json</button>
				  <button id="print" class="dropdown-item" onclick = "printDiv();" >print document</button>
					
					<button id="odt_save" class="dropdown-item d-none" onclick = "save()">export document</button>
				  </li>
			  
			  
			  </ul>
			
			
			
			</div>
			
			<div class="btn-group-sm d-none d-md-block dropup col-auto" style= "padding-left: 12px; ">
			  <button type="button" class="btn btn-secondary dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" id= "change_template" >
				templates
			  </button>
			  <ul class="dropdown-menu">
				<li>
				<button id="detailed_dmp" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="dmp1")'>detailed DMP (EU)</button>
				</li>
				<li>
				<button id="detailed_dfg-dmp" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="dfg-dmp")'>DFG DMP (DE)</button>
				</li>
				<li>
				<button id="guide_save" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="practical-guide")'>practice guide</button>
				</li>
				<li>
				<button id="proposal_save" class=" dropdown-item" onclick = 'load_dmp(reload_answers,name="proposal")'>proposal</button>
				</li>
				<li>
				<a id="template-input" class=" dropdown-item" data-bs-toggle="offcanvas" data-bs-target="#offcanvas-template-input" aria-controls="offcanvas-template-input" onclick='document.getElementById("template-input-content").innerHTML=window.localStorage.getItem("fulltext_user");' href="#" >user defined template</a>
				</li>
			  </ul>
			</div>
			
			<div class="btn-group-sm  d-none d-md-block dropup col-auto">
			  <button type="button" class="btn btn-secondary dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" id= "upload_button" >
				(re)load 
			  </button>
			  <ul class="dropdown-menu">
				<li>
					
						<label for="upload" class="form-label dropdown-item">upload json
						<input class="form-control d-none" type="file" id="upload">
						</label>
						</li>
						<li>
						<button id="clear_cache" class=" dropdown-item" onclick = 'window.localStorage.clear();document.location.reload()'>clear cache</button>
						

			  

					
				</li>
			  </ul>
			</div>
			
			
			<div class="btn-group-sm d-none d-md-block dropup col-auto ">
			  <button type="button" class="btn btn-primary dropdown-toggle" data-bs-toggle="dropdown" aria-expanded="false" id="save_button">
				save
			  </button>
			  <ul class="dropdown-menu">
				<li>
				  <button id="copy_doc" class=" dropdown-item" onclick = 'copy_doc()'>copy document</button>
				
				  
				  <button id="test" class="d-none dropdown-item" onclick = "test()">debug test</button>
				  </li>
				  <li>
				  <button id="json_save" class="dropdown-item" onclick = "save_json()" >save answers to json</button>
				  <button id="print" class="dropdown-item" onclick = "printDiv();" >print document</button>
					
					<button id="odt_save" class="dropdown-item d-none" onclick = "save()">export document</button>
				  </li>
			  </ul>
			  <!-- Button trigger modal -->
				<button type="button" class="btn btn-primary d-none" data-bs-toggle="modal"  data-bs-target="#staticBackdrop" id= "modal-button">
				modal
				</button>
			</div>
			<div class ="inline text-center" >
			<a href="https://github.com/nfdi4plants/plan-generator/wiki/Home--%F0%9F%A4%96" style="font-size:small"  > <span class="badge btn-secondary">wiki </span></a>	<a href="javascript:void(0);" style="font-size:small" onclick="tour.show()" ><span class="badge btn-tutorial text-dark">tutorial</span></a>
			</div>
			
			  
			   
			  
			  
			

				
			
					
			
		
			
		
			
			  
				</div>
			   
			   
			
			</div>



			
		</div>	
		</div>	
			<!-- offcanvas for  study objects-->
			<div class="offcanvas offcanvas-bottom" data-bs-scroll="true" data-bs-backdrop="false" tabindex="-1" id="offcanvas-aim" aria-labelledby="offcanvasScrolling1">
			  <div class="offcanvas-header">
				<h5 class="offcanvas-title" id="offcanvasScrolling1">The project aim should be a apart of a sentence.</h5>
				<button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"></button>
			  </div>
			  <div class="offcanvas-body">
				<p><b> Example 1: </b>aims at creating a computational model of carbon and water flow within a whole plant architecture </p>
				<br>
				<p><b> Example 2: </b>aims at generating data management plan with minimal effort and making the data as open as possible</p>
			  </div>
			</div>
			
			<div class="offcanvas offcanvas-bottom" data-bs-scroll="true" data-bs-backdrop="false" tabindex="-1" id="offcanvas-obj" aria-labelledby="offcanvasScrolling2">
			  <div class="offcanvas-header">
				<h5 class="offcanvas-title" id="offcanvasScrolling2">The project object = target.</h5>
				<button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"></button>
			  </div>
			  <div class="offcanvas-body">
				<p><b> Example 1: </b> carbon and water flow in plants </p>
				<br>
				<p><b> Example 2: </b> data management plan</p>
			  </div>
			</div>
			
			<div class="offcanvas offcanvas-start" data-bs-scroll="true" data-bs-backdrop="false" tabindex="-1" id="offcanvas-template-input" aria-labelledby="offcanvasScrolling3">
			  <div class="offcanvas-header">
				<h1 class="offcanvas-title" id="offcanvasScrolling3" style="padding-bottom:0px;">User-defined template</h1>
				<button  class="btn btn-primary position-static bottom-0 start-50" data-bs-dismiss="offcanvas" aria-label="Close" onclick='document.getElementById("user-defined").innerHTML=document.getElementById("template-input-content").innerHTML;window.localStorage.setItem("fulltext_user", document.getElementById("template-input-content").innerHTML);  doc_name="user-defined";setTimeout(() => {
					 load_dmp(reload_answers,name = doc_name);
					}, 1000) ;'>submit  </button>
				<button type="button" class="btn-close text-reset" data-bs-dismiss="offcanvas" aria-label="Close"></button>
			  </div>
			  
			  <div class="offcanvas-body" style="padding-top:0px;">
			  <h2 style="padding-top:0px;">You can click the following text to edit or load our prepared template as a starting point
			  <button class="badge btn-primary" onclick=' document.getElementById("template-input-content").innerHTML= document.getElementById("dmp1").innerHTML; '> H2020 DMP </button>
			  <button class="badge btn-primary" onclick=' document.getElementById("template-input-content").innerHTML= document.getElementById("dfg-dmp").innerHTML; '> DFG DMP </button>
			  <button class="badge btn-primary" onclick=' document.getElementById("template-input-content").innerHTML= document.getElementById("practical-guide").innerHTML; '> Practical Guide </button>
			  <button class="badge btn-primary" onclick=' document.getElementById("template-input-content").innerHTML= document.getElementById("proposal").innerHTML; '> Proposal </button></h2>
			  
					<div id="template-input-content" contenteditable="true" style="border: 3px dotted black;">
					
					</div>
					
			  </div>
			
			
			</div>

			<!-- Modal -->
			<div class="modal fade" id="staticBackdrop" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-10" aria-labelledby="staticBackdropLabel" aria-hidden="true">
			  <div class="modal-dialog">
				<div class="modal-content">
				  <div class="modal-header">
					<h5 class="modal-title" id="staticBackdropLabel">Cookie consent</h5>
					<button type="button" onclick="consent()" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
				  </div>
				  <div class="modal-body">
					Currently, we don't use any cookie that needs your consent. This banner is only a mock-up. Your data is saved in local storage of the browser and is not shared with anyone.
				  </div>
				  <div class="modal-footer">
					<button type="button" class="btn btn-secondary" onclick="consent()" data-bs-dismiss="modal">Close</button>
					<button type="button" class="btn btn-primary" data-bs-dismiss="modal" onclick="consent()">consent</button>
				  </div>
				</div>
			  </div>
			</div>
			  
			
			<div id="dmp1" class="d-none">
			
			<h1 class="c13" ><span class="c2">1</span><span class="c15">    </span><span class="c2">Introduction</span></h1><p class="c0"><span class="c1">#if$_EU $_PROJECT is part of the Open Data initiative of the EU. #endif$_EU</span></p><p class="c0"><span class="c1">To best profit from open data, it is necessary to not only store data but to make data findable, accessible, interoperable and reusable (FAIR). #if$_PROTECT Open and FAIR data, however, considers the need to protect individual data sets. #endif$_PROTECT</span></p><p class="c0"><span class="c1">The aim of this document is to provide guidelines on principles guiding the data management in $_PROJECT and what data will be stored by using the answers to the EU questionnaire on Data Management Plan (DMP) as a DMP document. </span></p><p class="c0"><span class="c1">The detailed DMP instructs how data is to be handled during and after the project. The $_PROJECT DMP is modelled according to the H2020 Online Manual. #if$_UPDATE It will be updated/its validity checked during the $_PROJECT project several times. At the very least, this will happen at month $_UPDATEMONTH. #endif$_UPDATE</span></p><h1 class="c13" id="h.wup0l4f7jl9k"><span class="c2">2</span><span class="c15">    </span><span class="c2 c12">Data Management Plan EU Template</span></h1><p class="c0"><span class="c1">#if$_DATAPLANT Currently data management relies on the annotated research context ARC. It is password protected, so before any data can be obtained or samples generated an authentication needs to take place. #endif$_DATAPLANT</span></p><h2 class="c9" id="h.3l7mu272odf4"><span class="c4">2.1. Data Summary</span></h2><p class="c0"><span class="c3">What is the purpose of the data collection/generation and its relation to the objectives of the project?</span></p><p class="c0"><span class="c3"></span></p><p class="c0"><span class="c0">For $_PROJECT, data collection, integration $_VISUALIZATION #if$_DATAPLANT through the DataPLANT ARC structure is absolutely necessary #endif$_DATAPLANT as data is not only used to understand principles, but it is also used for analyzing data and in the end stakeholder involvement, which need to be informed about data provenance. It is therefore of importance that not only data is well generated, but also well annotated with metadata using open standards, as it is laid out in the following section. As $_PROJECT aims at $_PROJECTAIM.<p class="c0"><span class="c3">What types and formats of data will the project generate/collect?</span></p><p class="c0"><span class="c1">We foresee that the following data will be collected and generated at the very least: $_PHENOTYPIC, $_GENETIC, $_GENOMIC, $_METABOLOMIC, $_RNASEQ, data about $_STUDYOBJECT. In addition, derived data from the original raw data sets will also be collected. This is important, as different analytical pipelines might yield different results or include ad-hoc data analysis parts#if$_DATAPLANT and these pipelines will be tracked in the DataPLANT ARC#endif$_DATAPLANT. Therefore, specific care will be taken to document and archive these resources (including the analytic pipelines) as well#if$_DATAPLANT relying on the vast expertise in the DataPLANT consortium #endif$_DATAPLANT.</span></p><p class="c0"><span class="c3"></span></p><p class="c0"><span class="c3">Will you re-use any existing data and how?</span></p><p class="c0"><span class="c1">The project builds on existing data sets and relies on them. #if$_RNASEQ For instance, without a proper genomic reference it is very difficult to analyze NGS data sets.#endif$_RNASEQ It is also important to include existing data sets on the expression and metabolic behaviour of $_STUDYOBJECT but of course also on existing characterization and the background knowledge. #if$_PARTNERS of the partners. #endif$_PARTNERS Genomic references can simply be gathered from reference databases for genomes/sequences, like the National Center for Biotechnology Information: NCBI (US); European Bioinformatics Institute: EBI (EU); DNA Data Bank of Japan: DDBJ (JP). In addition, prior unstructured data in the form of publications and data contained therein will be used for decision making.</span></p><p class="c0"><span class="c3">What is the origin of the data?</span></p><p class="c0"><span class="c1">Public data will be extracted as described in the previous paragraph. For $_PROJECT, specific data sets will be generated by the consortium partners.</span></p>#if$_RNASEQ <p class="c0"><span class="c1"> Short read sequencing will be outsourced and raw data will be received. </span></p> #endif$_RNASEQ #if$_METABOLOMIC <p class="c0"><span class="c1"> Metabolomic data will be generated using chromatography coupled to mass spectrometry and from enzyme platforms mostly.</span></p> #endif$_METABOLOMIC #if$_PROTEOMIC <p class="c0"><span class="c1"> Proteomic data will be generated using an EU platform which are in line with community standards.</span></p> #endif$_PROTEOMIC #if$_PREVIOUSPROJECTS <p class="c0"><span class="c1"> Data from previous projects such as $_PREVIOUSPROJECTS will be considered. </span></p> #endif$_PREVIOUSPROJECTS <p class="c0"><span class="c3">What is the expected size of the data?</span></p><p class="c0"><span class="c1">We expect to generate raw data in the range of $_RAWDATA GB of data. The size of derived data will be about $_DERIVEDDATA GB.</span></p><p class="c0"><span class="c3"></span></p><p class="c0"><span class="c3">To whom might it be useful ('data utility')?</span></p><p class="c0"><span class="c1">The data will be useful for the $_PROJECT partners to the scientific community working on $_STUDYOBJECT or the general public interested in $_STUDYOBJECT. Hence, $_PROJECT also strives to collect the data that has been disseminated and potentially advertise it #if$_DATAPLANT e.g. through the DataPLANT platform or other means#endif$_DATAPLANT, if it is not included in a publication anyway which is the most likely form of dissemination.</span></p><h2 class="c9" id="h.8sk9nau3ts35"><span class="c4">2.2. FAIR data</span></h2><h3 class="c14" id="h.gtd4kdf6hncf"><span class="c3">Making data findable, including provisions for metadata</span></h3><p class="c0"><span class="c3">Are the data produced and/or used in the project discoverable with metadata, identifiable and locatable by means of a standard identification mechanism (e.g. persistent and unique identifiers such as Digital Object Identifiers)?</span></p><p class="c0"><span class="c1">All data sets will receive unique identifiers and they will be annotated with Metadata. $_PROJECT will rely on community standards plus additional recommendations necessary in the plant field adapted by e.g. using suggestions from the Minimum Information About a Plant Phenotyping Experiment (MIAPPE). These -unlike across domain minimal sets such as Dublin core which mostly defines submitter and what general type of data is being dealt with (e.g. images) - allow reusability by other researchers as it also defines properties of the plant (see preceding section). However, of course minimal cross domain annotations are part of $_PROJECT. #if$_DATAPLANT The core integration with DataPLANT will also allow to tag individual releases with a digital object identifier (DOI). #endif$_DATAPLANT</span></p><p class="c0"><span class="c3">What naming conventions do you follow?</span></p><p class="c0"><span class="c1">Data variables will use standard names. For example, this is the case for genes, metabolites and proteins. These will also be linked to free biomedical ontologies where this is possible. In the case of datasets, the dataset names will also be made to be meaningful and human readable. In addition, traditional names will of course be included where necessary as synonyms.</span></p><p class="c0"><span class="c3">Will search keywords be provided that optimize possibilities for re-use?</span></p><p class="c0"><span class="c1">Keywords about the experiment and the general consortium will be included as well as an abstract about the data, where useful. In addition, certain keywords can be auto-generated from dense metadata and its underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with standardized PLANT ontologies that are supplemented where the ontology doesnt yet include the variables. #endif$_DATAPLANT</span></p><p class="c0"><span class="c3">Do you provide clear version numbers?</span></p><p class="c0"><span class="c1">To maintain data integrity and to be able to re-analyze data, data sets will get version numbers where this is useful (e.g. raw data must not be changed and will not get a version number and is considered immutable). #if$_DATAPLANT This is automatically supported by the ARC Git DataPLANT infrastructure. #endif$_DATAPLANT</span></p><p class="c0"><span class="c3">What metadata will be created? In case metadata standards do not exist in your discipline, please outline what type of metadata will be created and how.</span><span class="c12 c17 c11"></span></p><p class="c0"><span class="c1">We foresee to use #if$_RNASEQ|$_GENOMIC e.g. </span><span class="c16">Minimum Information About a Next-generation Sequencing Experiment</span><span class="c1">MinSEQe for sequencing data and #endif$_RNASEQ|$_GENOMIC #if$_METABOLOMIC Metabolights submission compliant standards for metabolites #issuewarning Some Metabolomics partners considers Metabolights not an accepted standard. #endissuewarning #endif$_METABOLOMIC as well as MIAPPE for phenotyping data in the broadest sense, but will also be relying on specific SOPs for additional annotations #if$_DATAPLANT relying on advanced DataPLANT annotation and ontologies. #endif$_DATAPLANT</span></p><h3 class="c14" id="h.8wrxcmaxl0hv"><span class="c3">Making data openly accessible</span></h3><p class="c0"><span class="c3">Which data produced and/or used in the project will be made openly available as the default? If certain datasets cannot be shared (or need to be shared under restrictions), explain why, clearly separating legal and contractual reasons from voluntary restrictions.</span></p><p class="c0"><span class="c1">By default, all data sets from $_PROJECT will be shared with the community and made openly available. This is however after partners have had the ability to check for IP protection (according to agreements and background rights). #if$_INDUSTRY This applies in particular to data pertaining to the industry. #endif$_INDUSTRY However, all partners also strive for IP protection of data sets which will be tested and due diligence will be given.</span></p><p class="c0"><span class="c3">Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their data closed if relevant provisions are made in the consortium agreement and are in line with the reasons for opting out.</span></p><p class="c0"><span class="c3"></span></p><p class="c0"><span class="c3">How will the data be made accessible (e.g. by deposition in a repository)?</span></p><p class="c0"><span class="c1">Data will be made available via the $_PROJECT platform using a user-friendly front end which allows data visualization. Besides this it will be ensured that data which can be stored in international specialized technical discipline related repositories (Sequencing at the national US center NCBI, SRA; Sequencing and sequence data for the EU center: EBI, ENA; Proteome database: PRIDE), will be stored and processed there as well.</span></p><p class="c0"><span class="c1"></span></p><p class="c0"><span class="c3">What methods or software tools are needed to access the data?</span></p><p class="c0"><span class="c1">#if$_PROPRIETARY $_PROJECT relies on the tool(s) $_PROPRIETARY. #endif$_PROPRIETARY</span></p><p class="c0"><span class="c1">#if!$_PROPRIETARY No specialized software will be needed to access the data, usually just a modern browser. Access will be possible via web interfaces. For data processing after obtaining raw data, typical open source software can be used. #endif!$_PROPRIETARY</span></p><p class="c0"><span class="c1">#if$_DATAPLANT DataPLANT offers tools such as the open source SWATE plugin for Excel, the ARC commander and the DMP tool which while not necessarily make the interaction with data more convenient. #endif$_DATAPLANT</span></p><p class="c0"><span class="c3">Is documentation about the software needed to access the data included?</span></p><p class="c0"><span class="c1">As no software is needed, no documentation needs to be provided. #if$_DATAPLANT However, DataPLANT resources are well described and their setup is documented on their github project pages. #endif$_DATAPLANT</span></p><p class="c0"><span class="c3">Is it possible to include the relevant software (e.g. in open source code)?</span></p><p class="c0"><span class="c1">As stated above, here we use publicly available open source and well documented certified software#if$_PROPRIETARY except for $_PROPRIETARY #endif$_PROPRIETARY.</span></p><p class="c0"><span class="c12 c11 c17"></span></p><p class="c0"><span class="c3">Where will the data and associated metadata, documentation and code be deposited? Preference should be given to certified repositories which support open access where possible.</span></p><p class="c0"><span class="c1">As noted above specialized repositories like SRA /ENA, Pride /Proteomexchange are the most common ones and will be used when adequate. In the case of unstructured less standardized data (e.g. experimental phenotypic measurements) these will be metadata annotated and if complete given a digital object identifier (DOI) #if$_DATAPLANT and the whole data sets wrapped into an ARC will get DOIs as well#endif$_DATAPLANT.</span></p><p class="c0"><span class="c3">Have you explored appropriate arrangements with the identified repository?</span></p><p class="c0"><span class="c1">The submission is for free and it is the goal (at least of ENA) to obtain as much data as possible. Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required. #if$_DATAPLANT For DataPLANT this has been agreed upon. #issuewarning Please use endpoint repositories to store or archive your data after publication. #endissuewarning #endif$_DATAPLANT</span></p><p class="c0"><span class="c3">If there are restrictions on use, how will access be provided?</span></p><p class="c0"><span class="c1">There are no restrictions, beyond the aforementioned IP checks which are in line with e.g. European open data policies.</span></p><p class="c0"><span class="c7"></span></p><p class="c0"><span class="c3">Is there a need for a data access committee?</span></p><p class="c0"><span class="c1">Consequently, there is no need for a committee.</span></p><p class="c0"><span class="c3">Are there well described conditions for access (i.e. a machine-readable license)?</span></p><p class="c0"><span class="c1">Yes, where possible e.g. CC REL will be used for data not submitted to specialized repositories such as ENA.</span></p><p class="c0"><span class="c3">How will the identity of the person accessing the data be ascertained?</span></p><p class="c20"><span class="c1">In case data is only shared within the consortium, if the data is not yet finished or under IP checks, the data is hosted internally and username and password will be required (see also our GDPR rules). In the case data is made public under final EU or US repositories, completely anonymous access is normally allowed this is the case for ENA as well and both are in line with GDPR requirements.</span></p><h3 class="c14" id="h.514g6glbmw6i"><span class="c3">Making data interoperable</span></h3><p class="c10"><span class="c3">Are the data produced in the project interoperable, that is allowing data exchange and re-use between researchers, institutions, organisations, countries, etc. (i.e. adhering to standards for formats, as much as possible compliant with available (open) software applications, and in particular facilitating re-combinations with different datasets from different origins)?</span></p><p class="c0"><span class="c1">Whenever possible, data will be stored in common and openly defined formats including all necessary metadata to interpret and analyze data in a biological context. By default, no proprietary formats will be used, however Microsoft Excel files (According to ISO/IEC 29500-1:2016) might be used as intermediates by the consortium #if$_DATAPLANT and by some ARC components in form #endif$_DATAPLANT. In addition, text files might be edited in text processor files but will be shared as pdf.</span></p><p class="c10"><span class="c3">What data and metadata vocabularies, standards or methodologies will you follow to make your data interoperable?</span></p><p class="c0"><span class="c1">As mentioned above, we foresee to use #if$_RNASEQ|$_GENOMIC e.g. MinSEQe for sequencing data #endif$_RNASEQ|$_GENOMIC and Metabolights compatible forms for metabolites as well as MIAPPE for phenotyping like data. The latter will thus allow integrating data across projects and safeguards reusing established and tested protocols. Additionally, we will use ontology terms to enrich the data sets relying on free and open ontologies. In addition, additional ontology terms might be created and be canonized during $_PROJECT.</span></p><p class="c10"><span class="c3">Will you be using standard vocabularies for all data types present in your data set, to allow inter-disciplinary interoperability?</span></p><p class="c0"><span class="c1">Indeed, open biomedical ontologies will be used where they are mature. As stated in the previous question sometimes ontologies and controlled vocabularies might have to be extended. #if$_DATAPLANT Here $_PROJECT will build on the advanced ontologies developed in DataPLANT. #endif$_DATAPLANT</span></p><p class="c10"><span class="c3">In case it is unavoidable that you use uncommon or generate project specific ontologies or vocabularies, will you provide mappings to more commonly used ontologies?</span></p><p class="c0"><span class="c1">Common and open ontologies will be used thus this question does not apply.</span></p><h3 class="c14" id="h.fs2sjyqtlku7"><span class="c3">Increase data reuse (through clarifying licences)</span></h3><p class="c10"><span class="c12 c17 c11"></span></p><p class="c10"><span class="c3">How will the data be licensed to permit the widest re-use possible?</span></p><p class="c0"><span class="c1">Open licenses will be used whenever possible, such as CC.</span></p><p class="c10"><span class="c3">When will the data be made available for re-use? If an embargo is sought to give time to publish or seek patents, specify why and how long this will apply, bearing in mind that research data should be made available as soon as possible.</span></p><p class="c0"><span class="c1">#if$_early The data will be published as soon as possible to guarantee reusability. #endif$_early #if$_ipissue In general, IP issues will first be checked. #endif$_ipissue All consortium partners will be encouraged to make data available prior to publication openly and/or under pre-publication agreements #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto International Data Release Workshop. #endif$_GENOMIC</span></p><p class="c10"><span class="c3">Are the data produced and/or used in the project usable by third parties, in particular after the end of the project? If the re-use of some data is restricted, explain why.</span></p><p class="c0"><span class="c1">There will be no restrictions once data is made public.</span></p><p class="c10"><span class="c3">How long is it intended that the data remains re-usable?</span></p><p class="c0"><span class="c1">Data will be made available for many years#if$_DATAPLANT and potentially indefinitely after the end of the project#endif$_DATAPLANT.</span></p><p class="c0"><span class="c1">In any case data submitted to specific community accepted public subject area repositories (as detailed above) e.g. ENA /Pride would be subject to local data storage regulation.</span></p><p class="c10"><span class="c3">Are data quality assurance processes described?</span></p><p class="c0"><span class="c1">Data will be checked and curated. #if$_DATAPLANT In addition, data will be analyzed for quality control (QC) problems using automatic procedures as well as by manual curation. #endif$_DATAPLANT</span></p><h2 class="c9" id="h.w5pimtt9qf6p"><span class="c4">2.3. Allocation of resources</span></h2><p class="c10"><span class="c3">What are the costs for making data FAIR in your project?</span></p><p class="c0"><span class="c1">The costs comprise data curation, #if$_DATAPLANT Arc consistency checks #endif$_DATAPLANT and maintenance on $_PROJECTs side.</span></p><p class="c0"><span class="c1">Additionally, last level costs for storage are incurred by last-level repositories (e.g. ENA) but not charged against $_PROJECT or its members but by the operation budget of these repositories.</span></p><p class="c10"><span class="c3">How will these be covered? Note that costs related to open access to research data are eligible as part of the Horizon 2020 grant (if compliant with the Grant Agreement conditions).</span></p><p class="c0"><span class="c1">A large part of the cost is covered by $_PROJECT #if$_DATAPLANT and the structures tools and knowledge laid down in the DataPLANT consortium. #endif$_DATAPLANT</span></p><p class="c10"><span class="c3">Who will be responsible for data management in your project?</span></p><p class="c0"><span class="c1">The responsible will be $_DATAOFFICER.</span></p><p class="c10"><span class="c3">Are the resources for long term preservation discussed (costs and potential value, who decides and how/what data will be kept and for how long)?</span></p><p class="c0"><span class="c1"> The datao fficer #if$_PARTNERS or $_PARTNERS #endif$_PARTNERS decides on preservation of data not submitted to last level subject area repositories #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT after project end. This will be in line with EU, institute policies and data sharing based on EU and international standards.</span></p><h2 class="c9" id="h.of351jlpnl2l"><span class="c4">2.4. Data security</span></h2><p class="c10"><span class="c3">What provisions are in place for data security (including data recovery as well as secure storage and transfer of sensitive data)?</span></p><p class="c20"><span class="c1">Once data is transferred to the $_PROJECT platform #if$_DATAPLANT and ARCs have been generated in DataPLANT#endif$_DATAPLANT, data security will be imposed. This comprises secure storage and the use of passwords and usernames are generally transferred via separate safe media.</span></p><p class="c10"><span class="c3">Is the data safely stored in certified repositories for long term preservation and curation?</span></p><p class="c0"><span class="c1">Wherever there are certified repositories these will be used as last level repositories. #if$_RNASEQ Transcriptomics data and gene sequence data will be also made available upon publication via the standards ENA/SRA. #endif$_RNASEQ #if$_METABOLOMIC Metabolite data in e.g. Metabolights (and/or Nationwide repositories like the German NFDI the French INRAe). #endif$_METABOLOMIC #if$_PROTEOMIC and Proteomics data in e.g. Pride/Proteomexchange. #endif$_PROTEOMIC. In addition, the national resource will maintain safekeeping of data also after the project ends. #if$_DATAPLANT In addition, databases like e.g. Proteomexchange do not support deep plant specific metadata hence ARCs will be maintained to ensure reusability. #endif$_DATAPLANT</span></p><h2 class="c9" id="h.ozaz4qpzixaa"><span class="c4">2.5. Ethical aspects</span></h2><p class="c10"><span class="c3">Are there any ethical or legal issues that can have an impact on data sharing? These can also be discussed in the context of the ethics review. If relevant, include references to ethics deliverables and ethics chapter in the Description of the Action (DoA).</span></p><p class="c0"><span class="c1">At the moment, we do not foresee ethical or legal issues with data sharing. In terms of ethics, as this is plant data, there is no need for an ethics committee, however diligence for plant resource benefit sharing is considered (see Nagoya protocol). #issuewarning You have to check here and enter any due diligence here at the moment we are awaiting if Nagoya gets also part of sequence information. In any case if you use material not from your (partner) country and characterize this physically e.g, metabolites, proteome, biochemically RNASeq etc this might represent a Nagoya relevant action unless this is from e.g. US (non partner), Ireland (not signed still contact them) etc but other laws might apply. #endissuewarning</span></p><p class="c10"><span class="c3">Is informed consent for data sharing and long term preservation included in questionnaires dealing with personal data?</span></p><p class="c0"><span class="c1">The only personal data that will potentially be stored is the submitter name and affiliation in the metadata for data. In addition, personal data will be collected for dissemination and communication activities using specific methods and procedures developed by the $_PROJECT partners to adhere to data protection. #issuewarning You need to inform and better get WRITTEN consent that you store emails and names or even pseudonyms such as twitter handles, we are very sorry about these issues we didnt invent them #endissuewarning</span></p><h2 class="c9" id="h.slb0g3rbvjfl"><span class="c4">2.6. Other issues</span></h2><p class="c0"><span class="c3">Do you make use of other national/funder/sectorial/departmental procedures for data management? If yes, which ones?</span></p><p class="c0"><span class="c1">Yes, $_PROJECT will use common DMP tools#if$_DATAPLANT and in particular resources developed by the NFDI of Germany#endif$_DATAPLANT.</span></p><p class="c0"><span class="c1"></span></p><p class="c6"><span class="c1"></span></p><h1 class="c13" id="h.f589zyp9ll5e"><span class="c2">3</span><span class="c15">    </span><span class="c12 c2">Annexes</span></h1><p class="c0"><span class="c1"></span></p><h2 class="c9" id="h.vrmh31avvatp"><span class="c11 c18">3.1</span><span class="c15">   </span><span class="c4">Abbreviations</span></h2><p class="c0"><span class="c1"></span></p><p class="c0"><span class="c11">CC</span><span class="c1">          Creative Commons</span></p><p class="c0"><span class="c11">DDBJ</span><span class="c1">        DNA Data Bank of Japan</span></p><p class="c0"><span class="c11">DMP</span><span class="c1">        Data Management Plan</span></p><p class="c0"><span class="c11">DOI</span><span class="c1">         Digital Object Identifier</span></p><p class="c0"><span class="c11">EBI</span><span class="c1">          European Bioinformatics Institute</span></p><p class="c0"><span class="c11">ENA</span><span class="c1">         European Nucleotide Archive</span></p><p class="c0"><span class="c11">EU           </span><span class="c1">European Union</span></p><p class="c0"><span class="c11">GDPR</span><span class="c1">       General data protection regulation (of the EU)</span></p><p class="c0"><span class="c11">IP</span><span class="c1">           Intellectual Property</span></p><p class="c0"><span class="c11">MIAMET</span><span class="c1">     Minimal Information about Metabolite experiment</span></p><p class="c0"><span class="c11">MIAPPE</span><span class="c1">     Minimal Information about Plant Phenotyping Experiment</span></p><p class="c0"><span class="c11">MinSEQe</span><span class="c1">    Minimum Information about a high-throughput Sequencing Experiment</span></p><p class="c0"><span class="c11">NCBI</span><span class="c1">        National Center for Biotechnology Information</span></p><p class="c0"><span class="c11">NFDI</span><span class="c1">        National Research Data Infrastructure (of Germany)</span></p><p class="c0"><span class="c11">NGS</span><span class="c1">         Next Generation Sequencing</span></p><p class="c0"><span class="c11">RNASeq</span><span class="c1">     RNA Sequencing</span></p><p class="c0"><span class="c11">SOP</span><span class="c1">         Standard Operating Procedures</span></p><p class="c0"><span class="c11">SRA</span><span class="c1">         Short Read Archive</span></p><p class="c0"><span class="c11">ONP</span><span class="c1">         Oxford Nanopore</span></p><p class="c0"><span class="c11">qRT</span><span class="c11">PCR</span><span class="c1">     quantitative real time polymerase chain reaction</span></p><p class="c0"><span class="c11">WP</span><span class="c1">         Work Package</span></p><p class="c0"><span class="c1"></span></p><p class="c0"><span class="c1"></span></p><p class="c0"><span class="c1"></span></p><p class="c0"><span class="c1"></span></p><p class="c22"><span class="c1"></span></p>
			
			</div>
			
			
			<div id="dfg-dmp" class="d-none">
			

<h1 class="c2">DFG dmp (answers reorganized)</h1>
<h2 
    >
	
	<span class="c2" >1</span>
	<span class="c2">.     </span>
	
	<span class="c2">Data description</span>
</h2>
<h3 >
	<span class="c3">1.1 Project introduction</span>
</h3>

	#if$_EU <p class="c0"> <span class="c1"> $_PROJECT is part of the Open Data initiative of the EU. </span> </p>#endif$_EU

<p class="c0">
	<span class="c1">To best profit from open data, it is necessary to not only store data but to make data findable, accessible, interoperable and reusable (FAIR). #if$_PROTECT Open and FAIR data, however, considers the need to protect individual data sets. #endif$_PROTECT</span>
</p>
<p class="c0">
	<span class="c1">The aim of this document is to provide guidelines on principles guiding the data management in $_PROJECT and what data will be stored by using the answers to the DFG Data Management Plan (DMP) checklist to generate a DMP document. </span>
</p>
<p class="c0">
	<span class="c1">The detailed DMP instructs how data is to be handled during and after the project. The $_PROJECT DMP is modelled according to the DFG data management checklist. #if$_UPDATE It will be updated/its validity checked during the $_PROJECT project several times. At the very least, this will happen at month $_UPDATEMONTH. #endif$_UPDATE</span>
</p>

<p class="c0">
	<span class="c3">1.2 How does your project generate new data?</span>
</p>
<p class="c0">
	<span class="c1">For $_PROJECT, data collection, integration $_VISUALIZATION #if$_DATAPLANT through the DataPLANT ARC structure is absolutely necessary #endif$_DATAPLANT as data is not only used to understand principles, but it is also used for analyzing data and in the end stakeholder involvement, which need to be informed about data provenance. It is therefore of importance that not only data is well generated, but also well annotated with metadata using </span>
	<span class="c5">open standards</span>
	<span class="c1">as it is laid out in the following section. As $_PROJECT aims at $_PROJECTAIM.</span>
</p>
<p class="c0">
	<span class="c1">Public data will be extracted as described in the previous paragraph. For $_PROJECT, specific data sets will be generated by the consortium partners.</span>
</p>#if$_RNASEQ <p class="c0">
	<span class="c1"> Short read sequencing will be outsourced and raw data will be received. </span>
</p> #endif$_RNASEQ #if$_METABOLOMIC <p class="c0">
	<span class="c1"> Metabolomic data will be generated using chromatography coupled to mass spectrometry and from enzyme platforms mostly.</span>
</p> #endif$_METABOLOMIC #if$_PROTEOMIC <p class="c0">
	<span class="c1"> Proteomic data will be generated using an EU platform which are in line with community standards.</span>
</p> #endif$_PROTEOMIC #if$_PREVIOUSPROJECTS <p class="c0">
	<span class="c1"> Data from previous projects such as $_PREVIOUSPROJECTS will be considered. </span>
</p> #endif$_PREVIOUSPROJECTS <p class="c0">

<p class="c0">
	<span class="c3">1.3 Is existing data reused?</span>
</p>
<p class="c0">
	<span class="c1">The project builds on existing data sets and relies on them. #if$_RNASEQ For instance, without a proper genomic reference it is very difficult to analyze NGS data sets.#endif$_RNASEQ It is also important to include existing data sets on the expression and metabolic behaviour of $_STUDYOBJECT but of course also on existing characterization and the background knowledge. #if$_PARTNERS of the partners. #endif$_PARTNERS Genomic references can simply be gathered from reference databases for genomes/sequences, like the National Center for Biotechnology Information: NCBI (US); European Bioinformatics Institute: EBI (EU); DNA Data Bank of Japan: DDBJ (JP). In addition, prior unstructured data in the form of publications and data contained therein will be used for decision making.</span>
</p>


<p class="c0">
	<span class="c3">1.4 Which data types (in terms of data formats like image data, text data or measurement data) arise in your project and in what way are they further processed?</span>
</p>
<p class="c0">
	<span class="c1">We foresee that the following data will be collected and generated at the very least: $_PHENOTYPIC, $_GENETIC, $_GENOMIC, $_METABOLOMIC, $_RNASEQ, data about $_STUDYOBJECT. In addition, derived data from the original raw data sets will also be collected. This is important, as different analytical pipelines might yield different results or include </span>
	<span class="c5">ad-hoc</span>
	<span class="c1">data analysis parts#if$_DATAPLANT and these pipelines will be tracked in the DataPLANT ARC#endif$_DATAPLANT. Therefore, specific care will be taken to document and archive these resources (including the analytic pipelines) as well#if$_DATAPLANT relying on the vast expertise in the DataPLANT consortium #endif$_DATAPLANT.</span>
</p>



<span class="c3">1.5 To what extent do these arise or what is the anticipated data volume?</span>
</p>
<p class="c0">
	<span class="c1">We expect to generate raw data in the range of $_RAWDATA GB of data. The size of derived data will be about $_DERIVEDDATA GB.</span>
</p>







<h2 class="c13"
    id="documentation">
	<span class="c2">2</span>
	<span class="c15">    </span>
	<span class="c2 c12">Documentation and data quality</span>
</h2>
<h3 class="c3">
	<span class="c0">2.1. What approaches are being taken to describe the data in a comprehensible manner (such as the use of available metadata, documentation standards or ontologies)? </span>
</h3>

<p class="c0">
	<span class="c1">#if$_DATAPLANT Currently data management relies on the annotated research context ARC. It is password protected, so before any data can be obtained or samples generated an authentication needs to take place. #endif$_DATAPLANT</span>
</p>

<p class="c0">
	<span class="c1">All data sets will receive unique identifiers and they will be annotated with Metadata. $_PROJECT will rely on community standards plus additional recommendations necessary in the plant field adapted by e.g. using suggestions from the Minimum Information About a Plant Phenotyping Experiment (MIAPPE). These -unlike across domain minimal sets such as Dublin core which mostly defines submitter and what general type of data is being dealt with (e.g. images) - allow reusability by other researchers as it also defines properties of the plant (see preceding section). However, of course minimal cross domain annotations are part of $_PROJECT. #if$_DATAPLANT The core integration with DataPLANT will also allow to tag individual releases with a digital object identifier (DOI). #endif$_DATAPLANT</span>
</p>

<p class="c0">
	<span class="c1">Keywords about the experiment and the general consortium will be included as well as an abstract about the data, where useful. In addition, certain keywords can be auto-generated from dense metadata and its underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with standardized PLANT ontologies that are supplemented where the ontology doesnt yet include the variables. #endif$_DATAPLANT</span>
</p>

<p class="c0">
	<span class="c1">We foresee to use #if$_RNASEQ|$_GENOMIC e.g. </span>
	<span class="c16">Minimum Information About a Next-generation Sequencing Experiment</span>
	<span class="c1">MinSEQe for sequencing data and #endif$_RNASEQ|$_GENOMIC #if$_METABOLOMIC Metabolights submission compliant standards for metabolites #issuewarning Some Metabolomics partners considers Metabolights not an accepted standard. #endissuewarning #endif$_METABOLOMIC as well as MIAPPE for phenotyping data in the broadest sense, but will also be relying on specific SOPs for additional annotations #if$_DATAPLANT relying on advanced DataPLANT annotation and ontologies. #endif$_DATAPLANT</span>
</p>
<p class="c0">
	<span class="c1">Indeed, open biomedical ontologies will be used where they are mature. As stated in the previous question sometimes ontologies and controlled vocabularies might have to be extended. #if$_DATAPLANT Here $_PROJECT will build on the advanced ontologies developed in DataPLANT. #endif$_DATAPLANT</span>
</p>
<h3 class="c0">
	<span class="c3">2.2 What measures are being adopted to ensure high data quality?</span>
</h3>


<p class="c0">
	<span class="c1">Data variables will use standard names. For example, this is the case for genes, metabolites and proteins. These will also be linked to free biomedical ontologies where this is possible. In the case of datasets, the dataset names will also be made to be meaningful and human readable. In addition, traditional names will of course be included where necessary as synonyms.</span>
</p>

<p class="c0">
	<span class="c1">To maintain data integrity and to be able to re-analyze data, data sets will get version numbers where this is useful (e.g. raw data must not be changed and will not get a version number and is considered immutable). #if$_DATAPLANT this is automatically supported by the ARC Git DataPLANT infrastructure. #endif$_DATAPLANT</span>
</p>
<p class="c0">
	<span class="c1">As mentioned above, we foresee to use #if$_RNASEQ|$_GENOMIC e.g. MinSEQe for sequencing data #endif$_RNASEQ|$_GENOMIC and Metabolights compatible forms for metabolites as well as MIAPPE for phenotyping like data. The latter will thus allow integrating data across projects and safeguards reusing established and tested protocols. Additionally, we will use ontology terms to enrich the data sets relying on free and open ontologies. In addition, additional ontology terms might be created and be canonized during $_PROJECT.</span>
</p>
<h3 class="c0">
	<span class="c3">2.3 Are quality controls in place and if so, how do they operate? </span>
</h3>

<p class="c0">
	<span class="c1">Data will be checked and curated. #if$_DATAPLANT In addition data, will be analyzed for quality control (QC) problems using automatic procedures as well as by manual curation. #endif$_DATAPLANT</span>
</p>


	
<h3 class="c0">
	<span class="c3">2.4 Which digital methods and tools (e.g. software) are required to use the data?</span>
</h3>
<p class="c0">
	<span class="c1">#if$_PROPRIETARY $_PROJECT relies on the tool(s) $_PROPRIETARY. #endif$_PROPRIETARY</span>
</p>
<p class="c0">
	<span class="c1">#if!$_PROPRIETARY No specialized software will be needed to access the data, usually just a modern browser. Access will be possible via web interfaces. For data processing after obtaining raw data, typical open source software can be used. #endif!$_PROPRIETARY</span>
</p>
<p class="c0">
	<span class="c1">As no software is needed, no documentation needs to be provided. #if$_DATAPLANT However, DataPLANT resources are well described and their setup is documented on their github project pages. #endif$_DATAPLANT</span>
</p>
<p class="c0">
	<span class="c1">#if$_DATAPLANT DataPLANT offers tools such as the open source SWATE plugin for Excel, the ARC commander and the DMP tool which while not necessarily make the interaction with data more convenient. #endif$_DATAPLANT</span>
</p>

	<span class="c1">As stated above, here we use publicly available open source and well documented certified software#if$_PROPRIETARY except for $_PROPRIETARY #endif$_PROPRIETARY.</span>
<p class="c0">

</p>
<p class="c0">
	<span class="c1"> $_PROJECT will use common DMP tools#if$_DATAPLANT and in particular resources developed by the NFDI of Germany#endif$_DATAPLANT.</span>
</p>








<h2 class="c0">
	<span class="c2">3. Storage and technical archiving the project </span>
</h2>

<h3 class="c0">
	<span class="c3">3.1 How is the data to be stored and archived throughout the project duration?</span>
</h3>
<p class="c0">
	<span class="c1">Wherever there are certified repositories these will be used as last level repositories. #if$_RNASEQ Transcriptomics data and gene sequence data will be also made available upon publication via the standards ENA/SRA. #endif$_RNASEQ #if$_METABOLOMIC Metabolite data in e.g. Metabolights (and/or Nationwide repositories like the German NFDI the French INRAe). #endif$_METABOLOMIC #if$_PROTEOMIC and Proteomics data in e.g. Pride/Proteomexchange. #endif$_PROTEOMIC In addition, the national resource will maintain safekeeping of data also after the project ends. #if$_DATAPLANT In addition, databases like e.g. Proteomexchange do not support deep plant specific metadata hence ARCs will be maintained to ensure reusability. #endif$_DATAPLANT</span>
</p>
<p class="c0">
	<span class="c1">Data will be made available for many years#if$_DATAPLANT and potentially indefinitely after the end of the project#endif$_DATAPLANT.</span>
</p>
<p class="c0">
	<span class="c1">In any case data submitted to specific community accepted public subject area repositories (as detailed above) e.g. ENA /Pride would be subject to local data storage regulation.</span>
</p>
<h3 class="c0">
	<span class="c3">3.2 What is in place to secure sensitive data throughout the project duration (access and usage rights)? </span>
</h3>
<p class="c20">
	<span class="c1">In case data is only shared within the consortium, if the data is not yet finished or under IP checks, the data is hosted internally and username and password will be required (see also our GDPR rules). In the case data is made public under final EU or US repositories, completely anonymous access is normally allowed this is the case for ENA as well and both are in line with GDPR requirements.</span>
</p>
<p class="c0">
	<span class="c1">There will be no restrictions once data is made public.</span>
</p>


<h2 class="c0">
	<span class="c2">4. Legal obligations and conditions </span>
</h2>

<h3 class="c0">
	<span class="c3">4.1 What are the legal specifics associated with the handling of research data in your project?</span>
</h3>

<p class="c0">
	<span class="c1">At the moment, we do not foresee ethical or legal issues with data sharing. In terms of ethics, as this is plant data, there is no need for an ethics committee, however diligence for plant resource benefit sharing is considered (see Nagoya protocol). #issuewarning You have to check here and enter any due diligence here at the moment we are awaiting if Nagoya gets also part of sequence information. In any case if you use material not from your (partner) country and characterize this physically e.g, metabolites, proteome, biochemically RNASeq etc this might represent a Nagoya relevant action unless this is from e.g. US (non partner), Ireland (not signed still contact them) etc but other laws might apply. #endissuewarning</span>
</p>
<p class="c0">
	<span class="c1">The only personal data that will potentially be stored is the submitter name and affiliation in the metadata for data. In addition, personal data will be collected for dissemination and communication activities using specific methods and procedures developed by the $_PROJECT partners to adhere to data protection. #issuewarning You need to inform and better get WRITTEN consent that you store emails and names or even pseudonyms such as twitter handles, we are very sorry about these issues we didnt invent them #endissuewarning</span>
</p>
<h3 class="c0">
	<span class="c3">4.2 Do you anticipate any implications or restrictions regarding subsequent publication or accessibility? </span>
</h3>
<p class="c0">
	<span class="c1">Open licenses will be used whenever possible, such as CC.</span>
</p>
<h3 class="c0">
	<span class="c3">4.3 What is in place to consider aspects of use and copyright law as well as ownership issues?  </span>
</h3>
<p class="c20">
	<span class="c1">Once data is transferred to the $_PROJECT platform #if$_DATAPLANT and ARCs have been generated in DataPLANT#endif$_DATAPLANT, data security will be imposed. This comprises secure storage and the use of passwords and usernames are generally transferred via separate safe media.</span>
</p>
<h3 class="c0">
	<span class="c3">4.4 Are there any significant research codes or professional standards to be taken into account? </span>
</h3>

<p class="c0">
	<span class="c1">Whenever possible, data will be stored in common and openly defined formats including all necessary metadata to interpret and analyze data in a biological context. By default, no proprietary formats will be used, however Microsoft Excel files (According to ISO/IEC 29500-1:2016) might be used as intermediates by the consortium #if$_DATAPLANT and by some ARC components in form #endif$_DATAPLANT. In addition, text files might be edited in text processor files but will be shared as pdf.</span>
</p>


<h2 class="c0">
	<span class="c2">5. Data exchange and long-term data accessibility </span>
</h2>

<h3 class="c0">
	<span class="c3">5.1 Which data sets are especially suitable for use in other contexts? 
</span>
</h3>


<p class="c0">
	<span class="c1">The data will be useful for the $_PROJECT partners to the scientific community working on $_STUDYOBJECT or the general public interested in $_STUDYOBJECT. Hence, $_PROJECT also strives to collect the data that has been disseminated and potentially advertise it#if$_DATAPLANT e.g.through the DataPLANT platform or other means #endif$_DATAPLANT, if it is not included in a publication anyway which is the most likely form of dissemination.</span>
</p>

<h3 class="c0">
	<span class="c3">5.2 Which criteria are used to select research data to make it available for subsequent use by others?  </span>
</h3>

<p class="c0">
	<span class="c1">By default, all data sets from $_PROJECT will be shared with the community and made openly available. This is however after partners have had the ability to check for IP protection (according to agreements and background rights). #if$_INDUSTRY This applies in particular to data pertaining to the industry. #endif$_INDUSTRY However, all partners also strive for IP protection of data sets which will be tested and due diligence will be given.</span>
</p>
<p class="c0">
	<span class="c1">Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their data closed if relevant provisions are made in the consortium agreement and are in line with the reasons for opting out.</span>
</p>
<h3 class="c0">
	<span class="c3">5.3 Are you planning to archive your data in a suitable infrastructure?  </span>
</h3>
<p class="c0">
	<span class="c1">Data will be made available via the $_PROJECT platform using a user-friendly front end which allows data visualization. Besides this it will be ensured that data which can be stored in international specialized technical discipline related repositories (Sequencing at the national US center NCBI, SRA; Sequencing and sequence data for the EU center: EBI, ENA; Proteome database: PRIDE), will be stored and processed there as well.</span>
</p>
<p class="c0">
	<span class="c1">As noted above specialized repositories like SRA /ENA, Pride /Proteomexchange are the most common ones and will be used when adequate. In the case of unstructured less standardized data (e.g. experimental phenotypic measurements) these will be metadata annotated and if complete given a digital object identifier (DOI). #if$_DATAPLANT and the whole data sets wrapped into an ARC will get DOIs as well. #endif$_DATAPLANT</span>
</p>
<p class="c0">
	<span class="c1">The submission is for free and it is the goal (at least of ENA) to obtain as much data as possible. Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required. #if$_DATAPLANT For DataPLANT this has been agreed upon. #issuewarning Please use endpoint repositories to store or archive your data after publication #endissuewarning #endif$_DATAPLANT</span>
</p>
<h3 class="c0">
	<span class="c3">5.4 If so, how and where? Are there any retention periods? </span>
</h3>
<p class="c0">
	<span class="c1">There are no restrictions, beyond the aforementioned IP checks which are in line with e.g. European open data policies.</span>
</p>
<p class="c0">
	<span class="c1">The $_PARTNERS decides on preservation of data not submitted to last level subject area repositories #if$_DATAPLANT or ARCs in DataPLANT#endif$_DATAPLANT after project end. This will be in line with EU, institute policies and data sharing based on EU and international standards.</span>
</p>
<h3 class="c0">
	<span class="c3">5.4 When is the research data available for use by third parties? </span>
</h3>
<p class="c0">
	<span class="c1">#if$_early The data will be published as soon as possible to guarantee reusability. #endif$_early #if$_ipissue In general, IP issues will first be checked. #endif$_ipissue All consortium partners will be encouraged to make data available prior to publication openly and/or under pre-publication agreements #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto International Data Release Workshop. #endif$_GENOMIC</span>
</p>

<h2 class="c0">
	<span class="c2">6. Responsibilities and resources </span>
</h2>

<h3 class="c0">
	<span class="c3">6.1 Who is responsible for adequate handling of the research data (description of roles and responsibilities within the project)? </span>
</h3>
<p class="c0">
	<span class="c1">The responsible will be $_DATAOFFICER as data Officer.</span>
</p>
<h3 class="c0">
	<span class="c3">6.2 Which resources (costs; time or other) are required to implement adequate handling of research data within the project?  </span>
</h3>
<p class="c0">
	<span class="c1">The costs comprise data curation, #if$_DATAPLANT Arc consistency checks #endif$_DATAPLANT and maintenance on $_PROJECTs side.</span>
</p>
<p class="c0">
	<span class="c1">Additionally, last level costs for storage are incurred by last-level repositories (e.g. ENA) but not charged against $_PROJECT or its members but by the operation budget of these repositories.</span>
</p>
<p class="c0">
	<span class="c1">A large part of the cost is covered by $_PROJECT #if$_DATAPLANT and the structures, tools and knowledge laid down in the DataPLANT consortium. #endif$_DATAPLANT</span>
</p>

















<p class="c0">
	<span class="c1"></span>
</p>
<p class="c6">
	<span class="c1"></span>
</p>
<h1 class="c13"
    id="h.f589zyp9ll5e">
	<span class="c2">3</span>
	<span class="c15">    </span>
	<span class="c12 c2">Annexes</span>
</h1>
<p class="c0">
	<span class="c1"></span>
</p>
<h2 class="c9"
    id="h.vrmh31avvatp">
	<span class="c11 c18">3.1</span>
	<span class="c15">   </span>
	<span class="c4">Abbreviations</span>
</h2>
<p class="c0">
	<span class="c1"></span>
</p>
<p class="c0">
	<span class="c11">CC</span>
	<span class="c1">          Creative Commons</span>
</p>
<p class="c0">
	<span class="c11">DDBJ</span>
	<span class="c1">        DNA Data Bank of Japan</span>
</p>
<p class="c0">
	<span class="c11">DMP</span>
	<span class="c1">        Data Management Plan</span>
</p>
<p class="c0">
	<span class="c11">DOI</span>
	<span class="c1">         Digital Object Identifier</span>
</p>
<p class="c0">
	<span class="c11">EBI</span>
	<span class="c1">          European Bioinformatics Institute</span>
</p>
<p class="c0">
	<span class="c11">ENA</span>
	<span class="c1">         European Nucleotide Archive</span>
</p>
<p class="c0">
	<span class="c11">EU           </span>
	<span class="c1">European Union</span>
</p>
<p class="c0">
	<span class="c11">GDPR</span>
	<span class="c1">       General data protection regulation (of the EU)</span>
</p>
<p class="c0">
	<span class="c11">IP</span>
	<span class="c1">           Intellectual Property</span>
</p>
<p class="c0">
	<span class="c11">MIAMET</span>
	<span class="c1">     Minimal Information about Metabolite experiment</span>
</p>
<p class="c0">
	<span class="c11">MIAPPE</span>
	<span class="c1">     Minimal Information about Plant Phenotyping Experiment</span>
</p>
<p class="c0">
	<span class="c11">MinSEQe</span>
	<span class="c1">    Minimum Information about a high-throughput Sequencing Experiment</span>
</p>
<p class="c0">
	<span class="c11">NCBI</span>
	<span class="c1">        National Center for Biotechnology Information</span>
</p>
<p class="c0">
	<span class="c11">NFDI</span>
	<span class="c1">        National Research Data Infrastructure (of Germany)</span>
</p>
<p class="c0">
	<span class="c11">NGS</span>
	<span class="c1">         Next Generation Sequencing</span>
</p>
<p class="c0">
	<span class="c11">RNASeq</span>
	<span class="c1">     RNA Sequencing</span>
</p>
<p class="c0">
	<span class="c11">SOP</span>
	<span class="c1">         Standard Operating Procedures</span>
</p>
<p class="c0">
	<span class="c11">SRA</span>
	<span class="c1">         Short Read Archive</span>
</p>
<p class="c0">
	<span class="c11">ONP</span>
	<span class="c1">         Oxford Nanopore</span>
</p>
<p class="c0">
	<span class="c11">qRT</span>
	
	<span class="c11">PCR</span>
	<span class="c1">     quantitative real time polymerase chain reaction</span>
</p>
<p class="c0">
	<span class="c11">WP</span>
	<span class="c1">         Work Package</span>
</p>
<p class="c0">
	<span class="c1"></span>
</p>
<p class="c0">
	<span class="c1"></span>
</p>
<p class="c0">
	<span class="c1"></span>
</p>
<p class="c0">
	<span class="c1"></span>
</p>
<p class="c22">
	<span class="c1"/>
</p>
			
			</div>
			<div id="practical-guide" class="d-none">
			
			
					
					<h1 dir="ltr" style="line-height:1.38;text-align: center;margin-top:20pt;margin-bottom:6pt;"><span style="font-size:20pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Practical Data Management Guide of $_PROJECT</span></h1><h1 class="c13"><b style="font-weight:normal;" id="docs-internal-guid-dc2b5b8c-7fff-c042-d6fa-27fdc29616b1"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">This practical guide of data management in $_PROJECT should be considered as a minimum description, leaving flexibility to include additional actions of specific domain or to national or local legislation.#if$_EU $_PROJECT will follow EU FAIR principles.&nbsp; #endif$_EU&nbsp;</span></p><br><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The practical guide of data management in $_PROJECT aims at providing a complete walkthrough for the researcher. The contents are customized based on the user input in the Data Management Plant Generator (DMPG). The practices in this guide are customized to fit related legal, ethical, standardization and funding body requirements. The suitable practices will cover all steps of a data management life-cycle:</span></p><br><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;"><li dir="ltr" style="list-style-type:decimal;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="1"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data acquisition:</span></p></li><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data generation</span></p></li></ol></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data should be generated by devices that are compatible with the open source format. The $_STUDYOBJECT should be compliant to biodiversity protocols. The protocols used to collect $_PHENOTYPIC, $_GENETIC, $_GENOMIC, $_METABOLOMIC, $_RNASEQ data about $_STUDYOBJECT will be stored#if$_DATAPLANT in the assays folder of ARC repositories.#endif$_DATAPLANT#if!$_DATAPLANT in a FAIR data storage. #endif!$_DATAPLANT&nbsp;</span></p><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="2"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;margin-left: 36pt;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data collection</span></p></li></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The data collection process is conducted by experimental scientists and stewarded by $_DATAOFFICER.#if$_DATAPLANT An electronic lab notebook will be used to ensure enough metadata is recorded and guarantees that the data can be further reused.#endif$_DATAPLANT&nbsp;</span></p><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="3"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;margin-left: 36pt;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data Organization</span></p></li></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The data organization process is conducted by $_DATAOFFICER. The detailed organization method and procedure are reported to the PIs. #if$_DATAPLANT The data organization will profit from the knowledge-base and data-base of DataPLANT, elastic search will be used to find better ways to organize the data. #endif$_DATAPLANT&nbsp;</span></p><br><br><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="2"><li dir="ltr" style="list-style-type:decimal;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="1"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Annotation</span></p></li><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Workflow documentation</span></p></li></ol></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Because the data collection process is conducted by experimental scientists and stewarded by $_DATAOFFICER.#if$_DATAPLANT An electronic lab notebook was used to ensure enough metadata is recorded and guarantees that the data can be further reused. The workflow can be retrieved from the electronic workbook by using the toolkits provided from the DataPLANT such as SWATE and arccommander. #endif$_DATAPLANT&nbsp;</span></p><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="2"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;margin-left: 36pt;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Metadata completion</span></p></li></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">In case some of the metadata is still missing from the documentation from the experimental scientists and data officer. #if$_DATAPLANT Raw data identifier and parsers provided by DataPLANT will be used to get meta data directly from the raw data file. The metadata collected from the raw data file can also be used to validate the metadata previously collected in case there are any mistakes. #endif$_DATAPLANT We foresee to use #if$_RNASEQ|$_GENOMIC e.g. MinSEQe for sequencing data #endif$_RNASEQ|$_GENOMIC and Metabolights compatible forms for metabolites as well as MIAPPE for phenotyping like data. The latter will thus allow integrating data across projects and safeguards reusing established and tested protocols. Additionally, we will use ontology terms to enrich the data sets relying on free and open ontologies. In addition, additional ontology terms might be created and be canonized during $_PROJECT.</span></p><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="3"><br><li dir="ltr" style="list-style-type:decimal;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="1"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Maintenance:&nbsp;</span></p></li></ol><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;margin-left: 36pt;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data storage</span></p></li></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Raw data collected in previous steps are stored immediately by using#if$_DATAPLANT the infrastructure of DataPLANT #endif$_DATAPLANT #if!$_DATAPLANT in a secure infrastructure. ARC (Annotated Research Context) are used as a container to store the raw data as well as metadata and workflow.#endif!$_DATAPLANT</span></p><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="2"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;margin-left: 36pt;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data curation</span></p></li></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">#if$_DATAPLANT Data stored in ARC is curated regularly as long as there are needs for update or revision.#endif$_DATAPLANT #if!$_DATAPLANT Data is curated regularly as long as there are needs for update or revision.#endif!$_DATAPLANT</span></p><br><br><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="4"><li dir="ltr" style="list-style-type:decimal;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="1"><p dir="ltr" style="line-height:1.38;background-color:#ffffff;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Publication and sharing</span></p></li><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data publishing</span></p></li></ol></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">#if$_RNASEQ Transcriptomics data and gene sequence data will be also made available upon publication via the standards ENA/SRA. #endif$_RNASEQ #if$_METABOLOMIC Metabolite data in e.g. Metabolights (and/or Nationwide repositories like the German NFDI the French INRAe). #endif$_METABOLOMIC #if$_PROTEOMIC and Proteomics data in e.g. Pride/Proteomexchange. #endif$_PROTEOMIC In addition, the national resource will maintain safekeeping of data also after the project ends. #if$_DATAPLANT In addition, databases like e.g. Proteomexchange does not support deep plant specific metadata hence ARCs will be maintained to ensure reusability. #endif$_DATAPLANT</span></p><ol style="margin-top:0;margin-bottom:0;padding-inline-start:48px;" start="2"><li dir="ltr" style="list-style-type:lower-alpha;font-size:11pt;font-family:Arial;color:#000000;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;margin-left: 36pt;" aria-level="2"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;" role="presentation"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Data sharing</span></p></li></ol><p dir="ltr" style="line-height:1.38;margin-left: 72pt;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">In case data is only shared within the consortium, if the data is not yet finished or under IP checks, the data is hosted internally and username and password will be required (see also our GDPR rules). In the case data is made public under final EU or US repositories, completely anonymous access is normally allowed; this is the case for ENA as well and both are in line with GDPR requirements.</span></p><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span class="Apple-tab-span" style="white-space:pre;">	</span></span><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span style="width:100px;display:inline-block;position:relative;"></span></span></p></b></h1><h3 dir="ltr" style="line-height:1.38;margin-top:16pt;margin-bottom:4pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#434343;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Metadata focus timeline</span></h3><h1 class="c13"><b style="font-weight:normal;"><br><br><div dir="ltr" style="margin-left:0pt;" align="left"><table style="border:none;border-collapse:collapse;"><colgroup><col width="104"><col width="520"><col width="132"><col width="104"><col width="94"><col width="102"><col width="88"></colgroup><thead><tr style="height:0pt"><th style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;" scope="col"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Stages</span></p></th><th style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;" scope="col"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Actions</span></p></th></tr><tr style="height:0pt"><th style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;" scope="col"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Study</span></p><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">initialization</span></p></th><th style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;" scope="col"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The metadata of study is created at the beginning of the project and updated continuously afterwards#if$_DATAPLANT, the input of the DMP generator created during the proposal stage can be reused. #endif$_DATAPLANT&nbsp;</span></p></th></tr><tr style="height:0pt"><th style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;" scope="col"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Sample</span></p><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Collection</span></p></th><th style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;" scope="col"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The information used to identify exact samples are initiated before experiments and updated at assay creation stages.</span></p><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">#if$_DATAPLANT The sample SWATE template will be used to document the sample metadata. A part of sample metadata which can be retrieved from the raw data will be updated afterwards using the ARC parsers #endif$_DATAPLANT&nbsp;</span></p></th></tr></thead><tbody><tr style="height:0pt"><td style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Assay</span></p><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Creation</span></p></td><td style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Assay metadata must be collected as a daily routine during the experimental phrase. #if$_DATAPLANT A electronic lab notebooks will be used to guarantee the applicability and correctness of the notebook content#endif$_DATAPLANT&nbsp;</span></p></td></tr><tr style="height:0pt"><td style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Computational Analysis</span></p><br></td><td style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Workflow annotation will be conducted during the computational analysis phrase. #if$_DATAPLANT The workflow metadata will be stored in the assay folder of the ARC.#endif$_DATAPLANT&nbsp;</span></p></td></tr><tr style="height:0pt"><td style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Results Sharing</span></p></td><td style="border-left:solid #000000 1pt;border-right:solid #000000 1pt;border-bottom:solid #000000 1pt;border-top:solid #000000 1pt;vertical-align:top;padding:5pt 5pt 5pt 5pt;overflow:hidden;overflow-wrap:break-word;"><p dir="ltr" style="line-height:1.2;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">The metadata of results are collected after all modifications and should not be changed after publication. #if$_DATAPLANT Collection of result metadata before publication and the conversion from ARC to the repositories will be taken care of by the ARC2REPO converter and done with minimal efforts. #endif$_DATAPLANT&nbsp;</span></p></td></tr></tbody></table></div><br><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size:11pt;font-family:Arial;color:#000000;background-color:#ffffff;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;"><span style="width:100px;display:inline-block;position:relative;"></span></span></p></b></h1><h3 dir="ltr" style="line-height:1.38;margin-top:16pt;margin-bottom:4pt;"><span style="font-size:13.999999999999998pt;font-family:Arial;color:#434343;background-color:transparent;font-weight:400;font-style:normal;font-variant:normal;text-decoration:none;vertical-align:baseline;white-space:pre;white-space:pre-wrap;">Preferred formats for raw data</span></h3><h1 class="c13"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">#if$_GENOMIC<span>&nbsp;&nbsp;</span></span></p><div dir="ltr" style="margin-left:0pt;" align="left"><table style="border: none;"><colgroup><col width="100"><col width="190"></colgroup><tbody><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">extension_ident</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Format Name</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.h5</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Hierarchical Data Format</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.bam</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">compressed binary version of a SAM file</span></p></td></tr><tr style="height:51.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.cram</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">compressed columnar file format for storing biological sequences aligned to a reference sequence</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fa</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.faa</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fas</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fasta</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fastq</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fastq</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.ffn</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fna</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fq</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fastq</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.frn</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.sff</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">sff-trim</span></p></td></tr></tbody></table></div><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"> #endif$_GENOMIC</span></p><br><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">#if$_RNASEQ<span>&nbsp;&nbsp;</span></span></p><div dir="ltr" style="margin-left:0pt;" align="left"><table style="border: none;"><colgroup><col width="100"><col width="190"></colgroup><tbody><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.bam</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">compressed binary version of a SAM file</span></p></td></tr><tr style="height:51.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.cram</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">compressed columnar file format for storing biological sequences aligned to a reference sequence</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fa</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.faa</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fas</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fast5</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">HDF5</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fasta</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fastq</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fastq</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.ffn</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fna</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fq</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fastq</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.frn</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">fasta</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.sff</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">sff-trim</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">bas.h5</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">HDF5</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.h5</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Hierarchical Data Format</span></p></td></tr></tbody></table></div><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"> #endif$_RNASEQ</span></p><br><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">&nbsp;#if$_METABOLOMIC<span>&nbsp;&nbsp;</span></span></p><div dir="ltr" style="margin-left:0pt;" align="left"><table style="border: none;"><colgroup><col width="100"><col width="190"></colgroup><tbody><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.cdf</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">netCDF (AIA/ANDI) interchange data format</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.cmp</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">netCDF compare file</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.abf</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Axon Binary File</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.d</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Agilent</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.dat</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Chromtech, Finnigan, VG</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.idb</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; color: rgb(28, 29, 30); background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">MASSLAB binary file</span></p></td></tr><tr style="height:39.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.jpf</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Mass Center Main Mass Spectrometry Data (JEOL USA, Inc.)</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.lcd</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Shimadzu LC Solution / Labsolutions Data File</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.mgf</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Mascot Generic File</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.raw</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Thermo Xcalibur, Micromass (Waters), PerkinElmer, Waters</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.scan</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">a spectrum or a Total Ion Chromatogram (TIC)</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.wiff</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">ABI/Sciex</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.xps</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Thermo Fisher Scientific K-Alpha+ spectrometer file</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">cdf.cmp</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">netCDF compare file</span></p></td></tr></tbody></table></div><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"> #endif$_METABOLOMIC</span></p><br><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">&nbsp;#if$_PROTEOMIC<span>&nbsp;&nbsp;</span></span></p><div dir="ltr" style="margin-left:0pt;" align="left"><table style="border: none;"><colgroup><col width="100"><col width="190"></colgroup><tbody><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.baf</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Bruker</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.d</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Agilent</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.dat</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Chromtech, Finnigan, VG</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.fid</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Bruker</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.ita</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">ION-TOF</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.itm</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">ION-TOF</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.mgf&nbsp;</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Mascot Generic File</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.ms</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Finnigan (Thermo)</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.ms2</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Sequest MS/MS peak list</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.pkl</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Micromass peak list</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.qgd</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Shimadzu</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.qgd</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Shimadzu</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.raw</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Thermo Xcalibur, Micromass (Waters), PerkinElmer, Waters</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.raw</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Physical Electronics/ULVAC-PHI</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.sms</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Bruker/Varian</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.spc</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Shimadzu</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.splib&nbsp;</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">spectral library file</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.t2d</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">ABI/Sciex</span></p></td></tr><tr style="height:27.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.tdc</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Physical Electronics/ULVAC-PHI</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.wiff</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">ABI/Sciex</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.xms</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Bruker/Varian</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.yep</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Bruker</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.dta</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">Sequest MS/MS peak list</span></p></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.msp</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><br></td></tr><tr style="height:15.75pt"><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;">.nist</span></p></td><td style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;"><br></td></tr></tbody></table></div><p dir="ltr" style="line-height:1.38;margin-top:0pt;margin-bottom:0pt;"><span style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline; white-space: pre-wrap;"> #endif$_PROTEOMIC</span></p><br><br></h1>
					
					
					
					
			
			
			
			
			</div>
			
			<div id="proposal" class="d-none">
			<p class="c0"><span class="c1">Datenmanagementplan</span></p><p class="c0"><span class="c1">Projektname: $_PROJECT</span></p><p class="c0"><span class="c1">Forschungsf&ouml;rderer: Bundesministerium f&uuml;r Bildung und Forschung</span></p><p class="c0"><span class="c1">F&ouml;rderprogramm: </span></p><p class="c0"><span class="c1">FKZ: </span></p><p class="c0"><span class="c1">Prim&auml;rforscher/Wissenschaftler: </span></p><p class="c0"><span class="c1">ID Prim&auml;rforscher/Wissenschaftler: </span></p><p class="c0"><span class="c1">Kontaktperson Datenmanagement: $_DATAOFFICER</span></p><p class="c0"><span class="c1">ID Kontaktperson Datenmanagement: </span></p><p class="c0"><span class="c1">Kontakt: </span></p><p class="c0"><span class="c1">Projektbeschreibung: </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Erstellungsdatum: </span></p><p class="c0"><span class="c1">&Auml;nderungsdatum: </span></p><p class="c0"><span class="c1">Zu beachtende Vorgaben: </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Datenspeicherung</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Die Dateibenennung erfolgt nach folgendem Standard:</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Dateien werden in m&ouml;glichst offenen, standardisierten Formaten gespeichert. </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Datendokumentation</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Folgende Dokumente werden erstellt:</span></p><p class="c3"><span class="c1">Public data will be extracted as described in the previous paragraph. For $_PROJECT, specific data sets will be generated by the consortium partners.</span></p>#if$_RNASEQ <p class="c3"><span class="c1"> Short read sequencing will be outsourced and raw data will be received. </span></p> #endif$_RNASEQ#if$_METABOLOMIC <p class="c3"><span class="c1"> Metabolomic data will be generated using chromatography coupled to mass spectrometry and from enzyme platforms mostly. </span></p> #endif$_METABOLOMIC#if$_PROTEOMIC <p class="c3"><span class="c1"> proteomic data will be generated using an EU platform which are in line with community standards. </span></p> #endif$_PROTEOMIC<p class="c3"><span class="c1">#if$_PREVIOUSPROJECTS data from previous projects such as $_PREVIOUSPROJECTS will be considered. #endif$_PREVIOUSPROJECTS </span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Legitimit&auml;t</span></p><p class="c0"><span class="c1">Data Sharing</span></p><p class="c0 c2"><span class="c1"></span></p><p class="c0"><span class="c1">Datenerhalt</span></p><p class="c0 c2"><span class="c1"></span></p>
			</div>
			
			<div id="user-defined" class="d-none">
			sefegwa
			</div>

	  </body>
</html>
