
dfg_dmp = {"dfg-dmp" : `<div id="dfg-dmp" class="">
    <div id="dfg-dmp1" class="content-page">
        <br />
        <center>
            <div class="p-2" id="vis-parent" style="margin: auto; ">
                <div id="vis" style=" width:100%; max-height:600px; aspect-ratio : 1 / 1;"></div>
            </div>

            <div id="text_start" class="c2 text-center">Data Management Plan of the DFG Project $_PROJECTNAME</div>
        </center>
        <h2>
            <span class="c2">1.&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c2">Data description</span>
        </h2>
        <h3>
            <span class="c3">1.1&nbsp;&nbsp;&nbsp;&nbsp;Introduction</span>
        </h3>

        #if$_EU&nbsp;
        <p class="c0"><span class="c1">$_PROJECTNAME is part of the Open Data Initiative (ODI) of the EU.
                #endif$_EU To best profit from open data, it is necessary not only to store data but to make data
                Findable, Accessible, Interoperable and Reusable (FAIR). #if$_PROTECT Open and FAIR data, however,
                considers the need
                to protect individual data sets. #endif$_PROTECT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The aim of this document is to provide guidelines on principles guiding the data management in $_PROJECTNAME. A Data Management Plan (DMP) will be generated by using the responses to the EU questionnaire on DMP, which defines the data stored in $_PROJECTNAME.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The detailed DMP instructs how data will be handled during and after the project. This DMP of $_PROJECTNAME 
                is created according to the DFG data management checklist. #if$_UPDATE It will be updated/its
                validity
                checked during $_PROJECTNAME several times. At the very least, this will happen at month
                $_UPDATEMONTH. #endif$_UPDATE
            </span>
        </p>

        <p class="c0">
            <span class="c3">1.2&nbsp;&nbsp;&nbsp;&nbsp;How does your project generate new data?</span>
        </p>
        <p class="c0">
            <span class="c1"> Data collection or generation methods may vary depending on their types and domains. For
                example:</span>
        </p>
        <ul style="list-style-type:disc;">
            
            
            #if$_GENETIC&nbsp; <li>
                    <span class="c1"> Genetic data will be generated from targeted crosses and in breeding experiments, and
                        will include recombination frequencies and position of genetic markers. This data will be used to associate quantitative trait loci with physical genomic markers/variants.
                    </span>
            </li> #endif$_GENETIC #if$_GENOMIC&nbsp; 
            <li>
                
                    <span class="c1"> Genomic data will be created from sequencing data, which will be processed to
                        identify genes, regulatory elements, transposable elements, and physical markers such as
                        SNPs, microsatellites and structural variants. </span>

            </li>
            #if$_PANGENOMIC&nbsp; 
            <li>
                
                    <span class="c1"> Pangenomic data will be collected by sequencing the genomes of multiple individuals within a clade. Then the the sequences are assembled and aligned to create a comprehensive gene reference.  </span>

            </li>
            #endif$_PANGENOMIC
            #endif$_GENOMIC #if$_CLONED-DNA&nbsp; 
            <li>

                    <span class="c1"> The origin and assembly of cloned DNA will include (a) source of original
                        vector sequence with adding gene reference where available, and source of insert DNA (e.g.,
                        amplification by PCR from a given sample, or obtained from existing library), (b) cloning
                        strategy (e.g., restriction endonuclease digests/ligation, PCR, TOPO cloning, Gibson
                        assembly, LR recombination), and (c) verified DNA sequence data of final recombinant vector.
                    </span>

            </li>
            #endif$_CLONED-DNA  #if$_TRANSCRIPTOMIC&nbsp; 
            <li>

                    <span class="c1"> Methods of transcriptomics data collection will be selected from microarrays,
                        quantitative PCR, Northern blotting, RNA immunoprecipitation, fluorescence in situ
                        hybridization. RNA-Seq data will be collected in seperate methods.</span>

            </li>
            #if$_SPATIALTRANSCRIPTOMIC&nbsp; 
            <li>

                    <span class="c1"> Spatial transcriptomics data will be collected using methods that spatially map RNA molecules to their precise tissue locations, ensuring the preservation of RNA data alongside comprehensive metadata about its origin.</span>

            </li>
            #endif$_SPATIALTRANSCRIPTOMIC&nbsp;
            #endif$_TRANSCRIPTOMIC&nbsp;    #if$_RNASEQ&nbsp; 
            <li>

                    <span class="c1"> RNA sequencing data will be generated using short-read or long-read platforms, either in-house or outsourced to academic facilities or commercial services, and the raw data will be processed using established bioinformatics pipelines. </span>
    
            </li>
            #if$_SCRNASEQ&nbsp; 
            <li>

                    <span class="c1"> Single-cell RNA-seq data will be collected by isolating single cells, extracting and barcoding RNA, preparing sequencing libraries, and generating high-quality transcriptomic data using platforms like Illumina, with meticulous metadata recording.</span>

            </li>
            #endif$_SCRNASEQ&nbsp;
            #endif$_RNASEQ&nbsp;  #if$_METABOLOMIC&nbsp; 
            <li>

                    <span class="c1"> Metabolomic data will be generated by coupled chromatography and mass
                        spectrometry using targeted or untargeted approaches.</span>

            </li>
            #endif$_METABOLOMIC  #if$_PROTEOMIC&nbsp; 
            <li>
        
                    <span class="c1"> Proteomic data will be generated using coupled chromatography and mass
                        spectrometry for the analysis of protein abundance and protein identification, as well as
                        additional techniques for structural analysis, the identification of post-translational
                        modifications and the characterization of protein interactions.</span>

            </li>
            #endif$_PROTEOMIC  #if$_PHENOTYPIC&nbsp; 
            <li>

                    <span class="c1"> Phenotypic data will be generated using phenotyping platforms and annotated using
                        corresponding ontologies, including number/size of organs such as leaves, flowers, buds
                        etc., size of whole plant, stem/root architecture (number of lateral branches/roots etc),
                        organ structures/morphologies, quantitative metrics such as color, turgor, health/nutrition
                        indicators, among others. </span>

            </li>
            #endif$_PHENOTYPIC  #if$_TARGETED&nbsp; 
            <li>

                    <span class="c1"> Targeted assay data (e.g. glucose and fructose concentrations or
                        production/utilization rates) will be generated using specific equipment and methods that
                        are fully documented in the laboratory notebook. </span>
            
            </li>
            #endif$_TARGETED #if$_IMAGE&nbsp; 
            <li>
        
                    <span class="c1"> Image data will be generated by equipment such as cameras, scanners, and
                        microscopes combined with software. Original images which contain metadata such as EXIF
                        photo information will be archived.</span>
            
            </li>
            #endif$_IMAGE #if$_MODELS&nbsp; 
            <li>
            
                    <span class="c1"> Model data will be generated by using software simulations. The complete
                        workflow, which includes the environment, runtime, parameters, and results, will be
                        documented and archived. </span>
            
            </li>
            #endif$_MODELS #if$_CODE&nbsp; 
            <li>
                
                    <span class="c1"> Computer code will be produced by programmers. </span>
            
            </li>
            #endif$_CODE  #if$_EXCEL&nbsp; 
            <li>
                
                    <span class="c1"> Excel data will be generated by data analysts by using MS Office or
                        open-source software. </span>
            
            </li>
            #endif$_EXCEL 

            

            
        </ul>

        <p class="c0">
            <span class="c1">
                $_PROJECTNAME $_PROJECTAIM. Therefore, data collection #if!$_VVISUALIZATION
                and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION , integration and visualization
                #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT Annotated Research Context (ARC) structure are absolutely necessary,
                #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data management process is absolutely
                necessary, #endif!$_DATAPLANT because the data are used not only to understand principles, but also
                be informed about the provenance of data analysis information. Stakeholders must also be informed
                about the provenance of data. It is therefore necessary to ensure that the data are well generated
                and also well annotated with metadata using open standards, as laid out in the next section.
            </span>
        </p>

        <p class="c0">
            <span class="c1">Public data will be extracted as described in paragraph 1.3. For $_PROJECTNAME,
                specific data sets will be generated by the consortium partners.</span>
        </p>

        <p class="c0">
            <span class="c3">1.3&nbsp;&nbsp;&nbsp;&nbsp;Is existing data reused?</span>
        </p>
        <p class="c0">
            <span class="c1">
                The project builds on existing data sets. #if$_RNASEQ For instance, without a
                proper genomic reference it is very difficult to analyze NGS data sets. #endif$_RNASEQ For #if$_GENETIC genetic data, #endif$_GENETIC
                #if$_GENOMIC genomic data, 
                    #if$_PANGENOMIC Pangenomic data, #endif$_PANGENOMIC
                #endif$_GENOMIC
                #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA
                #if$_TRANSCRIPTOMIC transcriptomic data, 
                    #if$_SPATIALTRANSCRIPTOMIC spatial transcriptomic data, #endif$_SPATIALTRANSCRIPTOMIC
                #endif$_TRANSCRIPTOMIC
                #if$_RNASEQ RNAseq data, 
                    #if$_SCRNASEQ single cell RNAseq data, #endif$_SCRNASEQ
                #endif$_RNASEQ
                #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC
                #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC
                #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC
                #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED
                #if$_IMAGE image datasets, #endif$_IMAGE
                #if$_MODELS modelling data, #endif$_MODELS
                #if$_CODE computational code, #endif$_CODE
                #if$_EXCEL excel files #endif$_EXCEL data of $_STUDYOBJECT,
                existing data sets as well as additional characterizations and background knowledge from prior publications will be used. #if$_PARTNERS of the
                partners: $_PARTNERS #endif$_PARTNERS Genomic references can simply be gathered from reference databases for
                genomes/sequences, like the National Center for Biotechnology Information: NCBI (US); European
                Bioinformatics Institute: EBI (EU); DNA Data Bank of Japan: DDBJ (JP).
            </span>
        </p>

        <p class="c0">
            <span class="c3">1.4&nbsp;&nbsp;&nbsp;&nbsp;Which data types (in terms of data formats like image data,
                text data or measurement data) arise in your project and in what way are they further
                processed?</span>
        </p>
        <p class="c0">
            <span class="c1">
                As part of $_PROJECTNAME, the following types of raw data will be gathered and/or produced: 
                #if$_GENETIC genetic data, #endif$_GENETIC  
                                #if$_GENOMIC genomic data, 
                    #if$_PANGENOMIC Pangenomic data, #endif$_PANGENOMIC
                #endif$_GENOMIC
                #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA
                #if$_TRANSCRIPTOMIC transcriptomic data, 
                    #if$_SPATIALTRANSCRIPTOMIC spatial transcriptomic data, #endif$_SPATIALTRANSCRIPTOMIC
                #endif$_TRANSCRIPTOMIC
                #if$_RNASEQ RNAseq data, 
                    #if$_SCRNASEQ single cell RNAseq data, #endif$_SCRNASEQ
                #endif$_RNASEQ
                #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC  
                #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC 
                #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC  
                #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED  
                #if$_IMAGE image datasets, #endif$_IMAGE  
                #if$_MODELS modelling data, #endif$_MODELS 
                #if$_CODE computational code, #endif$_CODE  
                #if$_EXCEL excel files, #endif$_EXCEL and other result data. Furthermore, data derived from the original
                raw data sets will also be collected. As different analytical pipelines
                might yield different results or include
            </span>
            <span class="c5">ad-hoc</span>
            <span class="c1">
                data analysis parts #if$_DATAPLANT and these pipelines will be tracked in the DataPLANT
                ARC #endif$_DATAPLANT , specific care will be taken, to document and archive these
                resources (including the
                analytic pipelines) as well #if$_DATAPLANT relying on the vast expertise in the DataPLANT consortium
                #endif$_DATAPLANT.
            </span>
        </p>

        <span class="c3">1.5&nbsp;&nbsp;&nbsp;&nbsp;To what extent do these arise or what is the anticipated data
            volume?</span>

        <p class="c0">
            <span class="c1">We expect to generate raw data in the range of $_RAWDATA GB of data. The size of the
                derived data will be about $_DERIVEDDATA GB.</span>
        </p>

        <h2 class="c13" id="documentation">
            <span class="c2">2.&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"></span><span
                class="c2 c12">Documentation and data quality</span>
        </h2>
        <h3 class="c3">
            <span class="c0">2.1.&nbsp;&nbsp;&nbsp;&nbsp;What approaches are being taken to describe the data in a
                comprehensible manner (such as the use of available metadata, documentation standards or
                ontologies)? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                All datasets will be associated with unique identifiers and will be annotated with metadata. We will
                use the Investigation, Study, Assay (ISA) specification for metadata creation. $_PROJECTNAME will rely
                on community standards plus additional recommendations applicable in the plant science, such as:
                <span class="list-to-remove-comma">
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS (Minimum Information about any (X) Sequence), #endif$_MIXS
                    #if$_MIGSEU MigsEu (Minimum Information about a Genome Sequence: Eucaryote), #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg (Minimum Information about a Genome Sequence: Organelle), #endif$_MIGSORG
                    #if$_MIMS MIMS (Minimum Information about Metagenome or Environmental), #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen (Minimal Information about a Marker Specimen:
                    Specimen), #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey (Minimal Information about a Marker Specimen:
                    Survey), #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG (Minimum Information about a Single Amplified Genome), #endif$_MISAG
                    #if$_MIMAG MIMAG (Minimum Information about Metagenome-Assembled Genome), #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_TRANSCRIPTOMIC
                    #if$_MINSEQE MINSEQE (Minimum Information about a high-throughput SEQuencing
                    Experiment), #endif$_MINSEQE 
                     #if$_MIAME MIAME (Minimum Information About a Microarray
                    Experiment), #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI (Recommended Metadata for Biological Images), #endif$_REMBI
                    #endif$_IMAGE
                    #if$_RNASEQ|$_GENOMIC
 
                    #if$_MINSEQE
                        MinSEQe (Minimum Information about a high-throughput Sequencing Experiment),
                    #endif$_MINSEQE 
                    #endif$_RNASEQ|$_GENOMIC

                    #if$_METABOLOMIC
                    #if$_MMIAMET
                    MIAMET (Minimum Information About a METabolomics experiment),
                    #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE (Minimum Information About a Proteomics Experiment), #endif$_MIAPE
                    #if$_MIMIX MIMix (The Minimum Information required for reporting a Molecular Interaction Experiment), #endif$_MIMIX
                    #endif$_PROTEOMIC  
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE (Minimum Information about Plant Phenotyping Experiment) #endif$_MIAPPE  #endif$_PHENOTYPIC .
                    </span>
            </span>
            <span class="c1">#if$_METABOLOMIC #if$_METABOLIGHTS The Metabolights submission compliant standards are used for metabolomic data.  #issuewarning Some metabolomics partners considers Metabolights
                not an accepted standard. #endissuewarning #endif$_METABOLIGHTS #endif$_METABOLOMIC These domain-specific standards standards allow reusability by other researchers by defining properties of the plant and experimental processes (see the preceding section). Cross-domain minimum standards mostly define the submitter and the general type of data. $_PROJECTNAME also implement minimal cross-domain annotations #if$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 like #endif$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 <span class="list-to-remove-comma"> #if$_DUBLINCORE Dublin Core, #endif$_DUBLINCORE #if$_DARWINCORE Darwin Core, #endif$_DARWINCORE #if$_SCHEMAORG Schema.org, #endif$_SCHEMAORG #if$_BIOSCHEMAS BioSchemas, #endif$_BIOSCHEMAS #if$_MARC21 MARC 21 #endif$_MARC21 . </span> #if$_DATAPLANT The core integration with DataPLANT will also allow
                individual releases to be tagged with a Digital Object Identifier (DOI).
                #endif$_DATAPLANT #if$_OTHERSTANDARDS Other standards such as $_OTHERSTANDARDINPUT are also adhered
                to. #endif$_OTHERSTANDARDS
            </span>
        </p>

        <p class="c0">
            <span class="c1">
                Open ontologies will be used where they are mature. As stated above, some ontologies and controlled
                vocabularies might need to be extended. #if$_DATAPLANT Here, $_PROJECTNAME will build on the
                advanced ontologies integrated in DataPLANT. #endif$_DATAPLANT
                Keywords about the experiment and the general consortium will be included, as well as an abstract
                about the data, where useful. In addition, certain keywords can be auto-generated from dense
                metadata and
                its underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with
                standardized DataPLANT ontologies that are supplemented where the ontology does not yet include the
                variables.
                #endif$_DATAPLANT
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">2.2&nbsp;&nbsp;&nbsp;&nbsp;What measures are being adopted to ensure high data
                quality?</span>
        </h3>

        <p class="c0">
            <span class="c1">
                $_PROJECTNAME $_PROJECTAIM. Therefore, data
                collection#if!$_VVISUALIZATION and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION ,
                integration and visualization #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT ARC
                structure are absolutely necessary #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data
                management process is absolutely necessary, #endif!$_DATAPLANT to ensure high data quality and allow reliable interpretations. Stakeholders must also be informed about the provenance of data. The data generated not only used to
                understand principles, but can also give information of data analysis provenance. It is therefore necessary to ensure
                that the data are well generated and also well annotated with metadata using open standards.

                Data variables will be allocated standardized names. For example, genes, proteins and metabolites will
                be named according to approved nomenclature and conventions. These will also be linked to functional
                ontologies where possible. Datasets will also be named in a meaningful way to ensure readability by
                humans. Plant names will include traditional names, binomials, and all
                strain/cultivar/subspecies/variety identifiers.

                Using methods from question 1.2 ensures high-quality raw data during collection and generation.
            </span>
        </p>

        <p class="c0">
            <span class="c1">
                To maintain data integrity and to be able to re-analyze data, data sets will get version numbers
                where this is useful (e.g. raw data must not be changed and will not get a version number and is
                considered
                immutable). #if$_DATAPLANT this is automatically supported by the ARC Git DataPLANT infrastructure.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                As mentioned above, <span class="c1 list-to-remove-comma">we will use ISA specification for metadata creation.
                    The following metadata standards will also be used:
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE, #endif$_MIAPPE #endif$_PHENOTYPIC
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS, #endif$_MIXS
                    #if$_MIGSEU MigsEu, #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg, #endif$_MIGSORG
                    #if$_MIMS MIMS, #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen, #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey, #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG, #endif$_MISAG
                    #if$_MIMAG MIMAG, #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_GENOMIC|$_TRANSCRIPTOMIC #if$_MINSEQE MINSEQE, #endif$_MINSEQE #endif$_GENOMIC|$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME, #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI, #endif$_REMBI
                    #endif$_IMAGE
                    #if$_METABOLOMIC
                    #if$_MMIAMET MIAMET, #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE, #endif$_MIAPE
                    #if$_MIMIX MIMix, #endif$_MIMIX
                    #endif$_PROTEOMIC.
                    
                </span> The metadata standards will
                thus allow the integration of data across projects and safegard the established and tested
                protocols being reused.
                Additionally, we will use ontology terms to enrich the data sets relying on free and open
                ontologies. In addition, additional ontology terms might be created and be canonized during the
                $_PROJECTNAME.
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">2.3&nbsp;&nbsp;&nbsp;&nbsp;Are quality controls in place and if so, how do they
                operate? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                The data will be checked and curated through the project period. #if$_DATAPLANT Furthermore, data
                will be analyzed for quality control (QC) problems using automatic procedures as well as by manual
                curation.
                #endif$_DATAPLANT PhD students and lab professionals will be responsible for the first-hand quality
                control. Afterwards, the data will be checked and annotated by $_DATAOFFICER. #if$_RNASEQ|$_GENOMIC

                FastQC will be conducted on the base-calling. #endif$_RNASEQ|$_GENOMIC
 Before publication, the data
                will be controlled again.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">2.4&nbsp;&nbsp;&nbsp;&nbsp;Which digital methods and tools (e.g. software) are required
                to use the data?</span>
        </h3>
            <p class="c0"><span class="c1 list-to-remove-comma"> $_PROJECTNAME will use common Research Data Management (RDM) infrastructures #if$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC developed by #endif$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC #if$_DATAPLANT|$_NFDI the NFDI of Germany, #endif$_DATAPLANT|$_NFDI #if$_FRENCH INRAe from France, #endif$_FRENCH #if$_EOSC EOSC (European Open Science Cloud), #endif$_EOSC .
                    </span></p>
        <p class="c0">
            <span class="c1">#if$_PROPRIETARY $_PROJECTNAME relies on the tool(s) $_PROPRIETARY.
                #endif$_PROPRIETARY</span>
        </p>
        <p class="c0">
            <span class="c1">
                #if!$_PROPRIETARY No specialized software will be needed to access the data, usually just a modern
                browser. Access will be possible through web interfaces. For data processing after obtaining raw
                data,
                typical open-source software can be used. As no proprietary software is needed, no documentation
                needs to be provided. #endif!$_PROPRIETARY
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT However, DataPLANT resources are well described, and their setup is documented on
                their github project pages.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT DataPLANT offers opensource data curation tools such as the <a target="_blank" href="https://nfdi4plants.github.io/nfdi4plants.knowledgebase/arcitect/">ARC management tool ARCitect </a>, command line tool <a target="_blank" href="https://nfdi4plants.github.io/nfdi4plants.knowledgebase/arc-commander/">ARCcommander </a>, <a target="_blank" href="https://github.com/nfdi4plants/nfdi4plants_ontology"> DataPLANT Biological Ontology (DPBO)</a>, <a target="_blank" href="https://nfdi4plants.github.io/nfdi4plants.knowledgebase/swate/">metadata annotation tool swate</a>, <a target="_blank" href="https://nfdi4plants.github.io/nfdi4plants.knowledgebase/resources/metadata-quiz/">the Metadata Quiz</a> and <a target="_blank" href="https://nfdi4plants.github.io/nfdi4plants.knowledgebase/resources/dataplan/">DataPLAN DMP generator</a>. #endif$_DATAPLANT
            </span>
        </p>

        <span class="c1">As stated above, here we use publicly available open-source and well-documented certified
            software, #if$_PROPRIETARY except for $_PROPRIETARY #endif$_PROPRIETARY.</span>
        <p class="c0"></p>

        <h2 class="c0">
            <span class="c2">3.&nbsp;&nbsp;&nbsp;&nbsp;Storage and technical archiving the project </span>
        </h2>

        <h3 class="c0">
            <span class="c3">3.1&nbsp;&nbsp;&nbsp;&nbsp;How is the data to be stored and archived throughout the
                project duration?</span>
        </h3>

          <p class="c0">
                        <span class="c1 list-to-remove-comma"> #if$_GENETIC|$_GENOMIC|$_RNASEQ For genetic or genomic data: #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC|$_RNASEQ #if$_GENETIC #if$_SRA NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                            #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #endif$_GENETIC #if$_GENETIC|$_GENOMIC|$_RNASEQ . #endif$_GENETIC|$_GENOMIC|$_RNASEQ </span>
                    </p>

        <p class="c0">
            <span class="c1  list-to-remove-comma">#if$_TRANSCRIPTOMIC For Transcriptomic data: #if$_SRA
                NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #if$_ARRAYEXPRESS
                EBI-ArrayExpress, #endif$_ARRAYEXPRESS . #endif$_TRANSCRIPTOMIC</span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_IMAGE For image data: #if$_BIOIMAGE EBI-BioImage
                Archive, #endif$_BIOIMAGE #if$_IDR IDR (Image Data Resource), #endif$_IDR .
                #endif$_IMAGE </span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_METABOLOMIC For metabolomic data: #if$_METABOLIGHTS
                EBI-Metabolights, #endif$_METABOLIGHTS #if$_METAWORKBENCH Metabolomics
                Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct (Molecular
                interactions), #endif$_INTACT . #endif$_METABOLOMIC </span>
        </p>
        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_PROTEOMIC For proteomics data: #if$_PRIDE
                EBI-PRIDE (PRoteomics IDEntifications Database), #endif$_PRIDE #if$_PDB PDB (Protein Data Bank), #endif$_PDB
                #if$_CHEBI Chebi (Chemical Entities of Biological Interest), #endif$_CHEBI .
                #endif$_PROTEOMIC </span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_PHENOTYPIC For phenotypic data: #if$_EDAL e!DAL-PGP (Plant
                Genomics & Phenomics Research Data Repository), #endif$_EDAL . #endif$_PHENOTYPIC
                #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                well. #endif$_OTHEREP
            </span>
        </p>
    
        <p class="c0">
            <span class="c1">Data will be made available for many years#if$_DATAPLANT and potentially indefinitely
                after the end of the project #endif$_DATAPLANT.</span>
        </p>
        <p class="c0">
            <span class="c1">In any case data submitted to international
                discipline related repositories which use specialized technologies (as detailed above) would be subject to local data storage regulation.</span>
        </p>
        <h3 class="c0">
            <span class="c3">3.2&nbsp;&nbsp;&nbsp;&nbsp;What is in place to secure sensitive data throughout the
                project duration (access and usage rights)? </span>
        </h3>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT In DataPLANT, data management relies on the Annotated Research Context (ARC). It is
                password protected, so before any data can be obtained or samples generated, an authentication needs
                to
                take place. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c20">
            <span class="c1">
                In case data is only shared within the consortium, because the data is not yet finished or under IP
                checks, the data is hosted internally, and the username and the password will be required (see also GDPR rules).
                In the case data is made public under final EU or US repositories, completely anonymous access is
                normally allowed. This is the case for ENA as well and both are in line with GDPR requirements.
            </span>
        </p>
        <p class="c0">
            <span class="c1">There will be no restrictions once the data is made public.</span>
        </p>

        <h2 class="c0">
            <span class="c2">4.&nbsp;&nbsp;&nbsp;&nbsp;Legal obligations and conditions </span>
        </h2>

        <h3 class="c0">
            <span class="c3">4.1&nbsp;&nbsp;&nbsp;&nbsp;What are the legal specifics associated with the handling of
                research data in your project?</span>
        </h3>

        <p class="c0">
            <span class="c1">
                At the moment, we do not anticipate ethical or legal issues with data sharing. In terms of ethics,
                since this is plant data, there is no need for an ethics committee, however, diligence for plant
                resource
                benefit sharing is considered. #issuewarning Please ensure that you complete any necessary due diligence. Currently, we are awaiting clarification on whether the Nagoya Protocol (ðŸ¡º see Nagoya Protocol) will encompass sequence information. Regardless, if you use material from a country other than your own (or that of your partner), and you conduct physical or biochemical characterization (e.g., metabolites, proteome, RNASeq, etc.), this may constitute an action relevant under the Nagoya Protocol. Exceptions might include materials from countries such as the U.S. (non-partner), Ireland (has not signedâ€”still contact them), etc., though other laws could apply. #endissuewarning
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The only personal data that will potentially be stored is the submitter name and affiliation in the
                metadata for data. In addition, personal data will be collected for dissemination and communication
                activities using specific methods and procedures developed by $_PROJECTNAME partners to adhere to
                data protection. #issuewarning You need to inform and better get WRITTEN consent that you store
                emails and
                names or even pseudonyms such as twitter handles, we are very sorry about these issues we didnâ€™t
                invent them #endissuewarning
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">4.2&nbsp;&nbsp;&nbsp;&nbsp;Do you anticipate any implications or restrictions regarding
                subsequent publication or accessibility? </span>
        </h3>
        <p class="c20">
            <span class="c1">
                Once data is transferred to the $_PROJECTNAME platform #if$_DATAPLANT and ARCs have been generated in
                DataPLANT #endif$_DATAPLANT , data security will be imposed. This comprises secure storage, and the
                use of
                passwords and usernames is generally transferred via separate safe media.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">4.3&nbsp;&nbsp;&nbsp;&nbsp;What is in place to consider aspects of use and copyright
                law as well as ownership issues? </span>
        </h3>
        <p class="c0">
            <span class="c1">Open licenses, such as Creative Commons (CC), will be used whenever possible.</span>
        </p>
        <h3 class="c0">
            <span class="c3">4.4&nbsp;&nbsp;&nbsp;&nbsp;Are there any significant research codes or professional
                standards to be taken into account? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                Whenever possible, data will be stored in common and openly defined formats including all the
                necessary metadata to interpret and analyze data in a biological context. By default, no proprietary
                formats will
                be used; however, Microsoft Excel files (according to ISO/IEC 29500-1:2016) might be used as
                intermediates by the consortium#if$_DATAPLANT and by some ARC components in form #endif$_DATAPLANT.
                In addition,
                text files might be edited in text processor files, but will be shared as pdf.
            </span>
        </p>

        <h2 class="c0">
            <span class="c2">5.&nbsp;&nbsp;&nbsp;&nbsp;Data exchange and long-term data accessibility </span>
        </h2>

        <h3 class="c0">
            <span class="c3">5.1&nbsp;&nbsp;&nbsp;&nbsp;Which data sets are especially suitable for use in other
                contexts? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                The data will be useful for $_PROJECTNAME partners, the scientific community working on
                $_STUDYOBJECT or the general public interested in $_STUDYOBJECT. Hence, $_PROJECTNAME also strives
                to collect the data
                that has been disseminated and potentially advertise it #if$_DATAPLANT e.g. through the DataPLANT
                platform or other means #endif$_DATAPLANT , if it is not included in a publication anyway, which is
                the most
                likely form of dissemination.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">5.2&nbsp;&nbsp;&nbsp;&nbsp;Which criteria are used to select research data to make it
                available for subsequent use by others? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                By default, all data sets from $_PROJECTNAME will be shared with the community and made openly
                available. This is, however, after partners have had the ability to check for IP protection
                (according to
                agreements and background rights). #if$_INDUSTRY This applies in particular to data pertaining to
                the industry. #endif$_INDUSTRY However, all partners also strive for IP protection of data sets
                which will
                be tested and due diligence will be given.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their
                data closed if relevant provisions are made in the consortium agreement and are in line with the
                reasons for opting out.
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">5.3&nbsp;&nbsp;&nbsp;&nbsp;Are you planning to archive your data in a suitable
                infrastructure? </span>
        </h3>
        <p class="c0">
            #if$_DATAPLANT As $_PROJECTNAME is closely aligned with DataPLANT, the ARC converter and DataHUB will be
            used to find the end-point repositories and upload to the repositories automatically. #endif$_DATAPLANT
            <span class="c1"> </span>
        </p>

        <p class="c0">
            
                As mentioned above, data will be archived in
                international discipline related repositories which use specialized technologies:
                <span class="c1 list-to-remove-comma">
                #if$_GENETIC|$_GENOMIC|$_RNASEQ #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC|$_RNASEQ #if$_TRANSCRIPTOMIC|$_GENETIC 
                #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_ARRAYEXPRESS
                EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC|$_GENOMIC #if$_IMAGE #if$_BIOIMAGE
                EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-Metabolights, #endif$_METABOLIGHTS
                #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct, #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                #endif$_EDAL #endif$_PHENOTYPIC.
                #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                well. #endif$_OTHEREP
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The submission is for free, and it is the goal #if$_ENA (at least of ENA) #endif$_ENA to obtain as much data as possible.
                Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required.
                #if$_DATAPLANT For DataPLANT, this has been agreed upon. #endif$_DATAPLANT #issuewarning If no data
                management platform such as DataPLANT is used, then you need to find appropriate repository to store
                or archive your data after publication. #endissuewarning
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">5.4&nbsp;&nbsp;&nbsp;&nbsp;If so, how and where? Are there any retention periods?
            </span>
        </h3>
        <p class="c0">
            <span class="c1">There are no restrictions, beyond the aforementioned IP checks, which are in line with
                e.g. European open data policies.</span>
        </p>
        <p class="c0">
            <span class="c1">
                The $_PARTNERS decides on preservation of data not submitted to end-point subject area repositories
                #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT after project end. This will be in line with EU
                institute policies and data sharing based on EU and international standards.
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">5.5&nbsp;&nbsp;&nbsp;&nbsp;When is the research data available for use by third
                parties? </span>
        </h3>
        <p class="c0">
            <span class="c1">
                #if$_EARLY Some raw data is made public as soon as it is collected and processed. #endif$_EARLY
                #if$_BEFOREPUBLICATION Relevant processed datasets are made public when the research findings are
                published. #endif$_BEFOREPUBLICATION #if$_ENDOFPROJECT At the end of the project, all data without
                embargo period will be published. #endif$_ENDOFPROJECT #if$_EMBARGO Data, which is subject to an
                embargo period, is not publicly accessible until the end of embargo period. #endif$_EMBARGO
                #if$_REQUEST Data is made available upon request, allowing controlled sharing while ensuring
                responsible use. #endif$_REQUEST #if$_IPISSUE IP issues will be checked before publication.
                #endif$_IPISSUE All consortium partners will be
                encouraged to make data available before publication, openly and/or under pre-publication agreements
                #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto International
                Data
                Release Workshop #endif$_GENOMIC . This will be implemented as soon as IP-related checks are
                complete.
            </span>
        </p>

        <h2 class="c0">
            <span class="c2">6.&nbsp;&nbsp;&nbsp;&nbsp;Responsibilities and resources </span>
        </h2>

        <h3 class="c0">
            <span class="c3">6.1&nbsp;&nbsp;&nbsp;&nbsp;Who is responsible for adequate handling of the research
                data (description of roles and responsibilities within the project)? </span>
        </h3>
        <p class="c0">
            <span class="c1">The responsible will be $_DATAOFFICER as data officer.</span>
            <span class="c1">The data responsible(s) (data officer#if$_PARTNERS or $_PARTNERS #endif$_PARTNERS)
                decides on the preservation of data not submitted to end-point subject area repositories
                #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT after the
                project end. This will be in line with EU institute policies, and data sharing based on EU and
                international standards.</span>
        </p>
        <h3 class="c0">
            <span class="c3">6.2&nbsp;&nbsp;&nbsp;&nbsp;Which resources (costs; time or other) are required to
                implement adequate handling of research data within the project? </span>
        </h3>
        <p class="c0">
            <span class="c1">The costs comprise data curation, #if$_DATAPLANT ARC consistency checks,
                #endif$_DATAPLANT and maintenance on $_PROJECTNAMEÂ´s side.</span>        
            <span class="c1">A large part of the cost is covered by $_PROJECTNAME #if$_DATAPLANT and the structures,
                tools and knowledge laid down in the DataPLANT consortium. #endif$_DATAPLANT</span>
        
        </p>
        <p class="c0">
            <span class="c1">
                Additionally, last-level costs for storage are incurred by end-point repositories (e.g. ENA) but not
                charged against $_PROJECTNAME or its members but by the operation budget of these repositories.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">6.3&nbsp;&nbsp;&nbsp;&nbsp;Who is responsible for curating the data once the project
                has ended? </span>
        </h3>
        <p class="c0">
            <span class="c1">As applicable, $_DATAOFFICER, who is responsible for ongoing data maintenance will also
                take care of it after the finish of $_PROJECTNAME. #if$_DATAPLANT DataPLANT as external data
                archives may provide such services in some cases. #endif$_DATAPLANT</span>
        </p>

        <p class="c0">
            <span class="c1"></span>
        </p>
        <p class="c6">
            <span class="c1"></span>
        </p>
        <h1 class="c13" ><span class="c2">7&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15">
        </span><span class="c12 c2">Annexes</span></h1>
    <p class="c0"><span class="c1"></span></p>
    <h2 class="c9" ><span class="c11 c18">7.1&nbsp;&nbsp;&nbsp;&nbsp;</span><span
            class="c15"> </span><span class="c4">Abbreviations</span></h2>
    <p class="c0"><span class="c1"></span></p>
    #if$_DATAPLANT  <span class="c11">  ARC </span> <span class="c1"> Annotated Research Context</span> <br>  #endif$_DATAPLANT 

    <span class="c11"> CC</span><span class="c1"> Creative Commons</span> <br>
    <span class="c11"> CC CEL</span><span class="c1"> Creative Commons Rights Expression Language</span> <br>
    #if$_PROTEOMIC #if$_CHEBI <span class="c11"> ChEBI</span><span class="c1"> Chemical Entities of Biological Interest </span> <br>  #endif$_CHEBI #endif$_PROTEOMIC      
    <span class="c11"> DDBJ</span><span class="c1"> DNA Data Bank of Japan</span> <br>
    <span class="c11"> DMP</span><span class="c1"> Data Management Plan</span> <br>
    <span class="c11"> DoA</span><span class="c1"> Description of Action</span> <br>
    <span class="c11"> DOI</span><span class="c1"> Digital Object Identifier</span> <br>
    <span class="c11"> EBI</span><span class="c1"> European Bioinformatics Institute</span> <br>
    #if$_PHENOTYPIC #if$_EDAL  <span class="c11"> e!DAL-PGP</span><span class="c1"> Plant Genomics & Phenomics Research Data Repository</span> <br>  #endif$_EDAL   #endif$_PHENOTYPIC  
    #if$_GENETIC  #if$_ENA   <span class="c11">  ENA </span> <span class="c1"> European Nucleotide Archive </span> <br>  #endif$_ENA        #endif$_GENETIC   
    <span class="c11"> EU </span><span class="c1">European Union</span> <br>
    <span class="c11"> FAIR </span><span class="c1">Findable Accessible Interoperable Reproducible</span> <br>
    <span class="c11"> GDPR</span><span class="c1"> General data protection regulation (of the EU)</span> <br>
    #if$_TRANSCRIPTOMIC|$_GENETIC #if$_GEO  <span class="c11"> GEO</span><span class="c1"> Gene Expression Omnibus </span>  <br>  #endif$_GEO   #endif$_TRANSCRIPTOMIC|$_GENETIC   
    #if$_IMAGE #if$_IDR  <span class="c11"> IDR</span><span class="c1"> Image Data Resource </span>  <br>  #endif$_IDR  #endif$_IMAGE 
    <span class="c11"> IP</span><span class="c1"> Intellectual Property</span> <br>
    <span class="c11"> ISO</span><span class="c1"> International Organization for Standardization</span>  <br>
    #if$_TRANSCRIPTOMIC #if$_MIAME <span class="c11"> MIAME</span><span class="c1"> Minimum Information About a Microarray Experiment</span> <br>  #endif$_MIAME  #endif$_TRANSCRIPTOMIC  
    #if$_METABOLOMIC #if$_MMIAMET <span class="c11"> MIAMET</span><span class="c1"> Minimum Information About a METabolomics experiment</span> <br>  #endif$_MMIAMET   #endif$_METABOLOMIC  
    #if$_PROTEOMIC #if$_MIAPE <span class="c11"> MIAPE</span><span class="c1"> Minimum Information About a Proteomics Experiment</span> <br>  #endif$_MIAPE  #endif$_PROTEOMIC  
    #if$_PHENOTYPIC #if$_MIAPPE <span class="c11"> MIAPPE</span><span class="c1"> Minimum Information about Plant Phenotyping Experiment</span> <br>  #endif$_MIAPPE  #endif$_PHENOTYPIC 
    #if$_GENOMIC|$_GENETIC #if$_MIGSEU <span class="c11"> MigsEu </span><span class="c1"> Minimum Information about a Genome Sequence: Eucaryote </span> <br>  #endif$_MIGSEU  
    #if$_MIGSORG <span class="c11"> MigsOrg </span><span class="c1"> Minimum Information about a Genome Sequence: Organelle </span> <br>  #endif$_MIGSORG  
    #if$_MIMAG <span class="c11"> MIMAG </span><span class="c1"> Minimum Information about Metagenome-Assembled Genome </span> <br>  #endif$_MIMAG  
    #if$_MIMARKSSPECIMEN <span class="c11"> MIMARKSSpecimen</span><span class="c1"> Minimal Information about a Marker Specimen: Specimen </span> <br>  #endif$_MIMARKSSPECIMEN 
    #if$_MIMARKSSURVEY  <span class="c11">  MIMARKSSurvey </span><span class="c1"> Minimal Information about a Marker Specimen: Survey </span> <br>  #endif$_MIMARKSSURVEY    #endif$_GENOMIC|$_GENETIC  
    #if$_PROTEOMIC #if$_MIMIX <span class="c11"> MIMIX</span><span class="c1"> The Minimum Information required for reporting a Molecular Interaction Experiment </span> <br>  #endif$_MIMIX   #endif$_PROTEOMIC 
    #if$_GENOMIC|$_GENETIC #if$_MIMS  <span class="c11">  MIMS</span><span class="c1"> Molecular Interactions </span> <br>  #endif$_MIMS    #endif$_GENOMIC|$_GENETIC 
    #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_MINSEQE <span class="c11"> MinSEQe</span><span class="c1"> Minimum Information about a high-throughput Sequencing Experiment</span> <br>  #endif$_MINSEQE #endif$_TRANSCRIPTOMIC|$_GENOMIC 
    #if$_GENOMIC|$_GENETIC #if$_MISAG <span class="c11"> MISAG</span><span class="c1"> Minimum Information about a Single Amplified Genome </span> <br>  #endif$_MISAG
    #if$_MIXS <span class="c11"> MIxS</span><span class="c1"> Minimum Information about any (X) Sequence </span> <br>  #endif$_MIXS #endif$_GENOMIC|$_GENETIC 
    <span class="c11"> NCBI</span><span class="c1"> National Center for Biotechnology Information</span> <br>
    <span class="c11"> NFDI</span><span class="c1"> National Research Data Infrastructure (of Germany)</span> <br>
    <span class="c11"> NGS</span><span class="c1"> Next Generation Sequencing</span> <br>
    #if$_PROTEOMIC #if$_PRIDE  <span class="c11"> PRIDE</span><span class="c1"> PRoteomics IDEntifications Database</span> <br>  #endif$_PRIDE
    #if$_PDB  <span class="c11"> PDB</span><span class="c1"> Protein Data Bank </span> <br>  #endif$_PDB #endif$_PROTEOMIC
    <span class="c11"> RDM</span><span class="c1"> Research Data Management</span> <br>
    #if$_IMAGE  #if$_REMBI  <span class="c11"> REMBI</span><span class="c1"> Recommended Metadata for Biological Images </span> <br>  #endif$_REMBI #endif$_IMAGE 

    #if$_RNASEQ  <span class="c11"> RNASeq</span><span class="c1"> RNA Sequencing</span> <br>  #endif$_RNASEQ 
    <span class="c11"> SOP</span><span class="c1"> Standard Operating Procedures</span>  <br>
    #if$_GENETIC #if$_SRA  <span class="c11"> SRA</span><span class="c1"> Sequence Read Archive</span> <br>  #endif$_SRA #endif$_GENETIC
    #if$_DATAPLANT  <span class="c11">  SWATE</span><span class="c1"> Swate Workflow Annotation Tool for Excel</span> <br>  #endif$_DATAPLANT 
    <span class="c11"> ONP</span><span class="c1"> Oxford Nanopore</span>  <br>
    <span class="c11"> qRT</span> <span class="c11"> PCR</span><span class="c1"> quantitative real time polymerase chain reaction</span>  <br>
    <span class="c11"> WP</span><span class="c1"> Work Package</span>  <br>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c22"><span class="c1"></span></p>
    </div>
</div>`}