
dmpStrings={
"Horizon2020_DMP" : `   <div id="Horizon2020_DMP" >
        <div class="content-page">
            <br />
            <center>
                <div class="p-2" id="vis-parent" style="margin: auto; width:100%";>
                    <div id="vis" style=" width:100%; max-height:600px; aspect-ratio : 1 / 1;" ></div>
                    <br />
                    <div class="p-2" style="margin: auto;">


                        <div class="c2 text-center ">
                            <h1>Data Management Plan of the H2020 Project $_PROJECTNAME </h1>
                        </div>

                        <div class="d-flex justify-content-center">



                            <br />
                            <table>
                                <tr>
                                    <td>
                                        <h3>Action Number: </h3>
                                    </td>
                                    <td>
                                        <h3> $_FUNDINGPROGRAMME </h3>
                                    </td>
                                </tr>
                                <tr>
                                    <td>
                                        <h3>Action Acronym:</h3>
                                    </td>
                                    <td>
                                        <h3> $_PROJECTNAME</h3>
                                    </td>
                                </tr>
                                <tr>
                                    <td>
                                        <h3>Action Title:</h3>
                                    </td>
                                    <td>
                                        <h3> $_PROJECTNAME</h3>
                                    </td>

                                </tr>
                                <tr>
                                    <td>
                                        <h3>Creation Date:</h2>
                                    </td>
                                    <td>
                                        <h3>$_CREATIONDATE</h3>
                                    </td>

                                </tr>
                                <tr>
                                    <td>
                                        <h3>Modification Date:</h2>
                                    </td>
                                    <td>
                                        <h3>$_MODIFICATIONDATE</h3>
                                    </td>

                                </tr>
                                <tr>
                                    <td>
                                        <h3>DMP version:</h3>
                                    </td>
                                    <td id="dmp_version">
                                        <h3> $_DMPVERSION</h3>
                                    </td>

                                </tr>
                            </table>

                        </div>
                        <br />

                    </div>
                </div>
            </center>

            <h1 class="c13"><span class="c2">1&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"></span><span
                    class="c2">Introduction</span></h1>
            <p class="c0"><span class="c1">#if$_EU $_PROJECTNAME is part of the Open Data Initiative (ODI) of the EU.
                    #endif$_EU To best profit from open data, it is necessary not only to store the data but to make it
                    Findable, Accessible, Interoperable, and Reusable (FAIR).#if$_PROTECT We support open and FAIR data,
                    however, we also consider the need to protect individual data sets. #endif$_PROTECT
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    The aim of this document is to provide guidelines on the principles of data management in $_PROJECTNAME. A Data Management Plan (DMP) will be generated by using the responses to the EU questionnaire on DMP, which defines the data stored in $_PROJECTNAME.
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    The detailed DMP states how data will be handled during and after the project. The $_PROJECTNAME DMP is
                    prepared according to the Horizon 2020 and Horizon Europe online manual. #if$_UPDATE It will be
                    updated/its validity checked during the
                    $_PROJECTNAME project several times. At the very least, this will happen at month $_UPDATEMONTH.
                    #endif$_UPDATE
                </span>
            </p>
            <h1 class="c13" id="h.wup0l4f7jl9k"><span class="c2">2&nbsp;&nbsp;&nbsp;&nbsp;</span><span
                    class="c15"></span><span class="c2 c12">Data Management Plan EU Template</span></h1>

            <h2 class="c9" id="h.3l7mu272odf4"><span class="c4">2.1&nbsp;&nbsp;&nbsp;&nbsp;Data Summary</span></h2>
            <p class="c0"><span class="c3">What is the purpose of the data collection/generation and its relation to the
                    objectives of the project?</span></p>
            <p class="c0">

                <span class="c1">
                    $_PROJECTNAME has the following aim: $_PROJECTAIM. Therefore, data collection #if!$_VVISUALIZATION and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION , integration and visualization #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT Annotated Research Context (ARC) structure are absolutely necessary, #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data management process is absolutely necessary, #endif!$_DATAPLANT because the data are used not only to understand principles, but also
                    be informed about the provenance of data analysis information. Stakeholders must also be informed
                    about the provenance of data. It is therefore necessary to ensure that the data are well generated
                    and also well annotated with metadata using open standards, as laid out in the next section.
                    <p class="c0"><span class="c3">What types and formats of data will the project
                            generate/collect?</span></p>
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    $_PROJECTNAME will collect and/or generate the following types of raw data: 
                    #if$_GENETIC genetic data, #endif$_GENETIC  
                    #if$_GENOMIC genomic data, #endif$_GENOMIC 
                    #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA 
                    #if$_TRANSCRIPTOMIC transcriptomic data, #endif$_TRANSCRIPTOMIC  
                    #if$_RNASEQ RNAseq data, #endif$_RNASEQ 
                    #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC  
                    #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC 
                    #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC  
                    #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED  
                    #if$_IMAGE image datasets, #endif$_IMAGE  
                    #if$_MODELS modelling data, #endif$_MODELS 
                    #if$_CODE computational code, #endif$_CODE  
                    #if$_EXCEL excel files, #endif$_EXCEL 
                     and other types of data which are related to $_STUDYOBJECT. In addition, the raw data
                    will also be processed and modified using analytical pipelines, which may yield different results or
                    include ad hoc data analysis parts. #if$_DATAPLANT These pipelines will be
                    tracked in the DataPLANT ARC. #endif$_DATAPLANT Therefore, care will be taken to document and archive
                    these resources (including the analytical pipelines) as well, #if$_DATAPLANT relying on the expertise in the DataPLANT consortium #endif$_DATAPLANT.
                </span>
            </p>
            <p class="c0"><span class="c3"></span></p>
            <p class="c0"><span class="c3">Will you re-use any existing data and how?</span></p>
            <p class="c0">
                <span class="c1">
                    The project builds on existing data sets and relies on them. #if$_RNASEQ|$_GENOMIC For example,
                    without a proper genomic reference it is very difficult to analyze next-generation sequencing (NGS)
                    data sets. #endif$_RNASEQ|$_GENOMIC It is also important to include existing data-sets on the
                    expression and metabolic behavior of the $_STUDYOBJECT, and on existing background knowledge
                    #if$_PARTNERS of the partners: $_PARTNERS #endif$_PARTNERS.
                    Genomic references can be gathered from reference databases for genomes/ and sequences, like the the NCBI (US National Center for Biotechnology Information),
                    EBI (European Bioinformatics Institute) and DDBJ (DNA Data Bank of Japan). Furthermore, prior 'unstructured' data in the form of publications and data
                    contained therein will be used for decision making.
                </span>
            </p>


            <p class="c0"><span class="c3">What is the origin of the data?</span></p>
            <p class="c0"><span class="c1">Public data will be extracted as described in the previous paragraph. For $_PROJECTNAME, specific data sets will be generated by the consortium partners.</span></p>


            <p class="c0">
                <span class="c1"> Data of different types or representing different domains will be generated using
                    unique approaches. For example:</span>
            </p>
            <ul style="list-style-type:disc;">
                #if$_GENETIC&nbsp; <li>
                        <span class="c1"> Genetic data will be generated targeting crosses and breeding experiments, and
                            will include recombination frequencies and crossover event that position genetic markers and
                            quantitative trait loci that can be associated with physical genomic markers/variants.
                        </span>
                </li> #endif$_GENETIC #if$_GENOMIC&nbsp; 
                <li>
                    
                        <span class="c1"> Genomic data will be created from sequencing data, which will be processed to
                            identify genes, regulatory elements, transposable elements, and physical markers such as
                            SNPs, microsatellites and structural variants. </span>
   
                </li>
                #endif$_GENOMIC #if$_CLONED-DNA&nbsp; 
                <li>
     
                        <span class="c1"> The origin and assembly of cloned DNA will include (a) source of original
                            vector sequence with Add gene reference where available, and source of insert DNA (e.g.,
                            amplification by PCR from a given sample, or obtained from existing library), (b) cloning
                            strategy (e.g., restriction endonuclease digests/ligation, PCR, TOPO cloning, Gibson
                            assembly, LR recombination), and (c) verified DNA data sequence of final recombinant vector.
                        </span>

                </li>
                #endif$_CLONED-DNA  #if$_TRANSCRIPTOMIC&nbsp; 
                <li>

                        <span class="c1"> Methods of transcriptomics data collection will be selected from microarrays,
                            quantitative PCR, Northern blotting, RNA immunoprecipitation, fluorescence in situ
                            hybridization. RNA-Seq data will be collected in seperate methods.</span>

                </li>
                #endif$_TRANSCRIPTOMIC&nbsp;  #if$_RNASEQ&nbsp; 
                <li>
   
                        <span class="c1"> RNA sequencing will be generated using short-read or long-read plantforms,
                            either in house or outsourced to academic facilities or commercial services, and the raw
                            data will be processed using estabilished biofirmatics piplines. </span>
         
                </li>
                #endif$_RNASEQ&nbsp;  #if$_METABOLOMIC&nbsp; 
                <li>

                        <span class="c1"> Metabolomic data will be generated by coupled chromatography and mass
                            spectrometry using targeted or untargeted approaches.</span>

                </li>
                #endif$_METABOLOMIC  #if$_PROTEOMIC&nbsp; 
                <li>
            
                        <span class="c1"> Proteomic data will be generated using coupled chromatography and mass
                            spectrometry for the analysis of protein abundance and protein identification, as well as
                            additional techniques for structural analysis, the identification of post-translational
                            modifications and the characterization of protein interactions.</span>
      
                </li>
                #endif$_PROTEOMIC  #if$_PHENOTYPIC&nbsp; 
                <li>
    
                        <span class="c1"> Phenotypic data will be generated using phenotyping platforms and
                            corresponding ontologies, including number/size of organs such as leaves, flowers, buds
                            etc., size of whole plant, stem/root architecture (number of lateral branches/roots etc),
                            organ structures/morphologies, quantitative metrics such as color, turgor, health/nutrition
                            indicators, among others. </span>
       
                </li>
                #endif$_PHENOTYPIC  #if$_TARGETED&nbsp; 
                <li>
        
                        <span class="c1"> Targeted assays data (e.g. glucose and fructose concentrations or
                            production/ultilization rates) will be generated using specific equipment and methods that
                            are fully documented in the laboratory notebook. </span>
                 
                </li>
                #endif$_TARGETED #if$_IMAGE&nbsp; 
                <li>
              
                        <span class="c1"> Image data will be generated by equipment such as cameras, scanners, and
                            microscopes combined with software. Original images which contain metadata such as exif
                            photo information will be archived.</span>
                  
                </li>
                #endif$_IMAGE #if$_MODELS&nbsp; 
                <li>
                   
                        <span class="c1"> Model data will be generated by using software simulations. The complete
                            workflow, which includes the environment, runtime, parameters, and results, will be
                            documented and archived. </span>
                  
                </li>
                #endif$_MODELS #if$_CODE&nbsp; 
                <li>
                    
                        <span class="c1"> Computer code will be produced by programmers. </span>
                  
                </li>
                #endif$_CODE  #if$_EXCEL&nbsp; 
                <li>
                    
                        <span class="c1"> Excel data will be generated by data analysts by using MS Office or
                            open-source software. </span>
                 
                </li>
                #endif$_EXCEL 

                

                
            </ul>


            #if$_PREVIOUSPROJECTS&nbsp; 
            <p class="c0"><span class="c1"> Data from previous projects such as $_PREVIOUSPROJECTS will be considered.
                </span></p>
            #endif$_PREVIOUSPROJECTS
            <p class="c0"><span class="c3">What is the expected size of the data?</span></p>
            <p class="c0"><span class="c1">We expect to generate $_RAWDATA GB of raw data and up to $_DERIVEDDATA GB of
                    processed data.</span></p>
            <p class="c0"><span class="c3"></span></p>
            <p class="c0"><span class="c3">To whom might it be useful ('data utility')?</span></p>
            <p class="c0">
                <span class="c1">
                    The data will initially benefit $_PROJECTNAME partners, but will also be made available to selected
                    stakeholders closely involved in the project, and then the scientific community working on
                    $_STUDYOBJECT. $_DATAUTILITY In addition, the general public interested in $_STUDYOBJECT can also
                    use the data after publication. The data will be disseminated according to $_PROJECTNAME's
                    dissemination and communication plan, #if$_DATAPLANT which aligns with DataPLANT platform or other means #endif$_DATAPLANT.
                </span>
            </p>
            <p class="c0">
                <span class="c1">

                </span>
            </p>
            <h2 class="c9" id="h.8sk9nau3ts35"><span class="c4">2.2&nbsp;&nbsp;&nbsp;&nbsp;FAIR data</span></h2>
            <h3 class="c14" id="h.gtd4kdf6hncf"><span class="c7">Making data findable, including provisions for
                    metadata</span></h3>
            <p class="c0">
                <span class="c3">
                    Are the data produced and/or used in the project discoverable with metadata, identifiable and
                    locatable by means of a standard identification mechanism (e.g. persistent and unique identifiers
                    such as Digital
                    Object Identifiers)?
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    All datasets will be associated with unique identifiers and will be annotated with metadata. We will
                    use Investigation, Study, Assay (ISA) specification for metadata creation. $_PROJECTNAME will rely
                    on community standards plus additional recommendations applicable in the plant science, such as the
                        <span class="list-to-remove-comma">
                            The following metadata/ minimum informatin standards will be used to collect metadata:
                            #if$_GENOMIC|$_GENETIC 
                            #if$_MIXS MIxS (Minimum Information about any (X) Sequence), #endif$_MIXS
                            #if$_MIGSEU MigsEu (Minimum Information about a Genome Sequence: Eucaryote), #endif$_MIGSEU
                            #if$_MIGSORG MigsOrg (Minimum Information about a Genome Sequence: Organelle), #endif$_MIGSORG
                            #if$_MIMS MIMS (Minimum Information about Metagenome or Environmental), #endif$_MIMS
                            #if$_MIMARKSSPECIMEN MIMARKSSpecimen (Minimal Information about a Marker Specimen:
                            Specimen), #endif$_MIMARKSSPECIMEN
                            #if$_MIMARKSSURVEY MIMARKSSurvey (Minimal Information about a Marker Specimen:
                            Survey), #endif$_MIMARKSSURVEY
                            #if$_MISAG MISAG (Minimum Information about a Single Amplified Genome), #endif$_MISAG
                            #if$_MIMAG MIMAG (Minimum Information about Metagenome-Assembled Genome), #endif$_MIMAG 
                            #endif$_GENOMIC|$_GENETIC
                            #if$_TRANSCRIPTOMIC 
                            #if$_MIAME MIAME (Minimum Information About a Microarray Experiment), #endif$_MIAME 
                            
                            #endif$_TRANSCRIPTOMIC
                            #if$_IMAGE
                            #if$_REMBI REMBI (Recommended Metadata for Biological Images), #endif$_REMBI
                            #endif$_IMAGE
                            #if$_RNASEQ|$_GENOMIC 
                            #if$_MINSEQE
                                MinSEQe (Minimum Information about a high-throughput Sequencing Experiment),
                            #endif$_MINSEQE 
                            #endif$_RNASEQ|$_GENOMIC
                            #if$_METABOLOMIC
                            #if$_MMIAMET
                            MIAMET (Minimum Information About a METabolomics experiment),
                            #endif$_MMIAMET
                            #endif$_METABOLOMIC
                            #if$_PROTEOMIC
                            #if$_MIAPE MIAPE (Minimum Information About a Proteomics Experiment), #endif$_MIAPE
                            #if$_MIMIX MIMix (The Minimum Information required for reporting a Molecular Interaction Experiment), #endif$_MIMIX
                            #endif$_PROTEOMIC 
                            #if$_PHENOTYPIC #if$_MIAPPE MIAPPE (Minimum Information about Plant Phenotyping Experiment) #endif$_MIAPPE  #endif$_PHENOTYPIC .
                        </span>
                </span>
                <span class="c1">#if$_METABOLOMIC #if$_METABOLIGHTS The Metabolights submission compliant standards are used for metabolomic data. #issuewarning Some metabolomics partners considers Metabolights
                    not an accepted standard. #endissuewarning #endif$_METABOLIGHTS #endif$_METABOLOMIC These specific standard unlike cross-domain minimal sets such as the Dublin core, which
                    mostly define the submitter and the general type of data, allow reusability by other researchers by
                    defining properties of the plant (see the preceding section). However, $_PROJECTNAME also implement minimal cross-domain annotations #if$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 such as #endif$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 <span class="list-to-remove-comma"> #if$_DUBLINCORE Dublin Core, #endif$_DUBLINCORE #if$_DARWINCORE Darwin Core, #endif$_DARWINCORE #if$_SCHEMAORG Schema.org, #endif$_SCHEMAORG #if$_BIOSCHEMAS BioSchemas, #endif$_BIOSCHEMAS #if$_MARC21 MARC 21 #endif$_MARC21 . </span> #if$_DATAPLANT The core integration with DataPLANT will also allow
                    individual releases to be tagged with a Digital Object Identifier (DOI).
                    #endif$_DATAPLANT #if$_OTHERSTANDARDS Other standards such as $_OTHERSTANDARDINPUT are also adhered
                    to. #endif$_OTHERSTANDARDS
                </span>
            </p>
            <p class="c0"><span class="c3">What naming conventions do you follow?</span></p>
            <p class="c0">
                <span class="c1">
                    Data variables will be allocated standard names. For example, genes, proteins and metabolites will
                    be named according to approved nomenclature and conventions. These will also be linked to functional
                    ontologies where possible. Datasets will also be named in a meaningful way to ensure readability by
                    humans. Plant names will include traditional names, binomials, and all
                    strain/cultivar/subspecies/variety identifiers.
                </span>
            </p>
            <p class="c0"><span class="c3">Will search keywords be provided that optimize possibilities for
                    re-use?</span></p>
            <p class="c0">
                <span class="c1">
                    Keywords about the experiment and consortium will be included, as well as an abstract about the
                    data, where useful. In addition, certain keywords can be auto-generated from dense metadata and its
                    underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with standardized
                    DataPLANT ontologies that are provided where the ontology does not yet include such variables.
                    #endif$_DATAPLANT
                </span>
            </p>
            <p class="c0"><span class="c3">Do you provide clear version numbers?</span></p>
            <p class="c0">
                <span class="c1">
                    To maintain data integrity and facilitate reanalysis, data sets will be allocated version numbers
                    where this is useful (e.g. raw data must not be changed and will not get a version number and is
                    considered
                    immutable). #if$_DATAPLANT This is automatically supported by the ARC Git DataPLANT infrastructure.
                    #endif$_DATAPLANT
                </span>
            </p>
            <p class="c0">
                <span class="c3">What metadata will be created? In case metadata standards do not exist in your
                    discipline, please outline what type of metadata will be created and how.</span><span
                    class="c12 c17 c11"></span>
            </p>
            <p class="c0">
                <span class="c1 list-to-remove-comma">We will use Investigation, Study, Assay (ISA) specification for metadata creation.
                    The following metadata standards will also be used:
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE, #endif$_MIAPPE #endif$_PHENOTYPIC
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS, #endif$_MIXS
                    #if$_MIGSEU MigsEu, #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg, #endif$_MIGSORG
                    #if$_MIMS MIMS, #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen, #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey, #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG, #endif$_MISAG
                    #if$_MIMAG MIMAG, #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_GENOMIC|$_TRANSCRIPTOMIC #if$_MINSEQE MINSEQE, #endif$_MINSEQE #endif$_GENOMIC|$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME, #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI, #endif$_REMBI
                    #endif$_IMAGE
                    #if$_METABOLOMIC
                    #if$_MMIAMET MIAMET, #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE, #endif$_MIAPE
                    #if$_MIMIX MIMix, #endif$_MIMIX
                    #endif$_PROTEOMIC.
                    
                </span>
            </p>


            <h3 class="c14" id="h.8wrxcmaxl0hv"><span class="c7">Making data openly accessible</span></h3>
            <p class="c0">
                <span class="c3">
                    Which data produced and/or used in the project will be made openly available as the default? If
                    certain datasets cannot be shared (or need to be shared under restrictions), we explain why, clearly
                    separating
                    legal and contractual reasons from voluntary restrictions.
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    By default, all data sets from $_PROJECTNAME will be shared with the community and made openly
                    available. However, before the data are released, all will be provided with an opportunity to check
                    for potential IP (according to the consortium agreement and background IP rights). #if$_INDUSTRY
                    This applies in particular to data pertaining to the industry. #endif$_INDUSTRY IP protection will
                    be prioritized for datasets that offer the potential for exploitation.
                </span>
            </p>
            <p class="c0">
                <span class="c3">
                    Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their
                    data closed if relevant provisions are made in the consortium agreement and are in line with the
                    reasons
                    for opting out.
                </span>
            </p>
            <p class="c0"><span class="c3"></span></p>
            <p class="c0"><span class="c3">How will the data be made accessible (e.g. by deposition in a
                    repository)?</span></p>
            <p class="c0">
            <p class="c0">
                <span class="c1">
                    Data will be made available via the $_PROJECTNAME platform using a user-friendly front end that allows
                    data visualization. Besides this it will be ensured that data which can be stored in
                    international discipline related repositories which use specialized technologies:
                   


                    <p class="c0">
                        <span class="c1 list-to-remove-comma"> #if$_GENETIC|$_GENOMIC For genetic or genomic data: #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_GENETIC #if$_SRA NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                            #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #endif$_GENETIC #if$_GENETIC|$_GENOMIC . #endif$_GENETIC|$_GENOMIC </span>
                    </p>
        
                    <p class="c0">
                        <span class="c1  list-to-remove-comma">#if$_TRANSCRIPTOMIC For Transcriptomic data: #if$_SRA
                            NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #if$_ARRAYEXPRESS
                            EBI-ArrayExpress, #endif$_ARRAYEXPRESS . #endif$_TRANSCRIPTOMIC</span>
                    </p>
        
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_IMAGE For image data: #if$_BIOIMAGE EBI-BioImage
                            Archive, #endif$_BIOIMAGE #if$_IDR IDR (Image Data Resource), #endif$_IDR .
                            #endif$_IMAGE </span>
                    </p>
        
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_METABOLOMIC For metabolomic data: #if$_METABOLIGHTS
                            EBI-Metabolights, #endif$_METABOLIGHTS #if$_METAWORKBENCH Metabolomics
                            Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct (Molecular
                            interactions), #endif$_INTACT . #endif$_METABOLOMIC </span>
                    </p>
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_PROTEOMIC For proteomics data: #if$_PRIDE
                            EBI-PRIDE (PRoteomics IDEntifications Database), #endif$_PRIDE #if$_PDB PDB (Protein Data Bank), #endif$_PDB
                            #if$_CHEBI Chebi (Chemical Entities of Biological Interest), #endif$_CHEBI .
                            #endif$_PROTEOMIC </span>
                    </p>
        
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_PHENOTYPIC For phenotypic data: #if$_EDAL e!DAL-PGP (Plant
                            Genomics & Phenomics Research Data Repository), #endif$_EDAL . #endif$_PHENOTYPIC
                            <br>
                            #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                            well. #endif$_OTHEREP
                        </span>
                    </p>
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    For unstructured and less standardized data (e.g., experimental phenotypic measurements), these will
                    be annotated with metadata and if complete allocated a digital object identifier (DOI).
                    #if$_DATAPLANT Whole datasets will also be wrapped into an ARC with allocated DOIs. The ARC and the
                    converters provided by DataPLANT will ensure that the upload into the endpoint repositories is fast
                    and easy.
                    #endif$_DATAPLANT
                </span>
            </p>
            </p>
            <p class="c0"><span class="c1"></span></p>
            <p class="c0"><span class="c3">What methods or software tools are needed to access the data?</span></p>
            <p class="c0"><span class="c1">#if$_PROPRIETARY $_PROJECTNAME relies on the tool(s) $_PROPRIETARY . #endif$_PROPRIETARY </span></p>
            <p class="c0">
                <span class="c1">
                    #if!$_PROPRIETARY No specialized software will be needed to access the data, just a modern browser.
                    Access will be possible through web interfaces. For data processing after obtaining raw data,
                    typical
                    open-source software can be used. #endif!$_PROPRIETARY
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    #if$_DATAPLANT DataPLANT offers tools such as the open-source SWATE plugin for Excel, the ARC
                    commander, arcCommander, and DataPLAN, #endif$_DATAPLANT.
                </span>
            </p>
            <p class="c0"><span class="c3">Is documentation about the software needed to access the data
                    included?</span></p>
            <p class="c0">
                <span class="c1">
                    #if$_DATAPLANT DataPLANT resources are well described, and their setup is documented on a github
                    project guide is provided on the GitHub project pages. #endif$_DATAPLANT
                    All external software documentation will be duplicated locally and stored near the software.
                </span>
            </p>
            <p class="c0"><span class="c3">Is it possible to include the relevant software (e.g. in open-source
                    code)?</span></p>
            <p class="c0"><span class="c1">As stated above, $_PROJECTNAME will use publicly available open-source and
                    well-documented certified software, #if$_PROPRIETARY except for $_PROPRIETARY as proprietary softare #endif$_PROPRIETARY.</span></p>
            <p class="c0"><span class="c12 c11 c17"></span></p>
            <p class="c0">
                <span class="c3">Where will the data and associated metadata, documentation and code be deposited?
                    Preference should be given to certified repositories that support open access, where
                    possible.</span>
            </p>
            <p class="c0">
                <span class="c1">
                    As noted above, specialized repositories will be used for common data types. For unstructured and
                    less standardized data (e.g., experimental phenotypic measurements), these will be annotated with
                    metadata and if complete allocated a digital object identifier (DOI).#if$_DATAPLANT The Whole
                    datasets will also be wrapped into an ARC with allocated DOIs. #endif$_DATAPLANT
                </span>
            </p>
            <p class="c0"><span class="c3">Have you explored appropriate arrangements with the identified
                    repository?</span></p>
            <p class="c0">
                <span class="c1">
                    The submission is for free, and it is the goal (at least of ENA) to obtain as much data as possible.
                    Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required, #if$_DATAPLANT and this has been confirmed for data associated with DataPLANT #endif$_DATAPLANT.
                    #issuewarning If no data management platform such as DataPLANT is used, then you need to find
                    appropriate repository to store or archive your data after publication. #endissuewarning
                </span>
            </p>
            <p class="c0"><span class="c3">If there are restrictions on use, how will access be provided?</span></p>
            <p class="c0"><span class="c1">There are no restrictions beyond the IP screening described above, which is
                    in line with European open data policies.</span></p>
            <p class="c0"><span class="c7"></span></p>


            <p class="c0"><span class="c3">Is there a need for a data access committee?</span></p>
            <p class="c0"><span class="c1">There is no need for a data access committee.</span></p>
            <p class="c0"><span class="c3">Are there well described conditions for access (i.e. a machine-readable
                    license)?</span></p>
            <p class="c0"><span class="c1">Yes, where possible, e.g. CC REL will be used for data not submitted to
                    specialized repositories such as ENA.</span></p>
            <p class="c0"><span class="c3">How will the identity of the person accessing the data be ascertained?</span>
            </p>
            <p class="c0">
                <span class="c1">
                    When data are shared only within the consortium, if the datasets are not yet complete or are undergoing 
                    IP checks, the data will be hosted internally, requiring a username and password for access (see GDPR rules). 
                    Once the data are made public in EU or US repositories, completely anonymous access is typically allowed. 
                    This applies to ENA as well, and both are in line with GDPR requirements.
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    #if$_DATAPLANT Currently, data management relies on the annotated research context (ARC). It is
                    password protected, so before any data or samples can be obtained, user authentication is required.
                    #endif$_DATAPLANT
                </span>
            </p>
            <h3 class="c14" id="h.514g6glbmw6i"><span class="c7">Making data interoperable</span></h3>
            <p class="c10">
                <span class="c3">
                    Are the data produced in the project interoperable, that is allowing data exchange and re-use
                    between researchers, institutions, organizations, countries, etc. (i.e. adhering to standards for
                    formats, as much
                    as possible compliant with available (open) software applications, and in particular facilitating
                    re-combinations with different datasets from different origins)?
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    Whenever possible, data will be stored in common and openly defined formats including all the
                    necessary metadata to interpret and analyze data in a biological context. By default, no proprietary
                    formats will be
                    used. However Microsoft Excel files (according to ISO/IEC 29500-1:2016) might be used as
                    intermediates by the consortium#if$_DATAPLANT and by some ARC components #endif$_DATAPLANT. In
                    addition, text
                    files might be edited in text processor files, but will be shared as pdf.
                </span>
            </p>
            <p class="c10"><span class="c3">What data and metadata vocabularies, standards or methodologies will you
                    follow to make your data interoperable?</span></p>
          
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">We will use Investigation, Study, Assay (ISA) specification for metadata creation.
                            The following metadata standards will also be used:
                            #if$_PHENOTYPIC #if$_MIAPPE MIAPPE, #endif$_MIAPPE #endif$_PHENOTYPIC
                            #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS, #endif$_MIXS
                            #if$_MIGSEU MigsEu, #endif$_MIGSEU
                            #if$_MIGSORG MigsOrg, #endif$_MIGSORG
                            #if$_MIMS MIMS, #endif$_MIMS
                            #if$_MIMARKSSPECIMEN MIMARKSSpecimen, #endif$_MIMARKSSPECIMEN
                            #if$_MIMARKSSURVEY MIMARKSSurvey, #endif$_MIMARKSSURVEY
                            #if$_MISAG MISAG, #endif$_MISAG
                            #if$_MIMAG MIMAG, #endif$_MIMAG
                            #endif$_GENOMIC|$_GENETIC
                            #if$_GENOMIC|$_TRANSCRIPTOMIC #if$_MINSEQE MINSEQE, #endif$_MINSEQE #endif$_GENOMIC|$_TRANSCRIPTOMIC
                            #if$_TRANSCRIPTOMIC #if$_MIAME MIAME, #endif$_MIAME #endif$_TRANSCRIPTOMIC
                            #if$_IMAGE
                            #if$_REMBI REMBI, #endif$_REMBI
                            #endif$_IMAGE
                            #if$_METABOLOMIC
                            #if$_MMIAMET MIAMET, #endif$_MMIAMET
                            #endif$_METABOLOMIC
                            #if$_PROTEOMIC
                            #if$_MIAPE MIAPE, #endif$_MIAPE
                            #if$_MIMIX MIMix, #endif$_MIMIX
                            #endif$_PROTEOMIC.
                            
                        </span>
                    </p>
                    <p class="c0">
                        <span class="c1">
                    The minimal information standards will allow
                    the integration of data across projects, and its reuse according to established and tested
                    protocols. We will also use
                    ontological terms to enrich the data sets relying on free and open ontologies where possible.
                    Additional ontology terms might be created and canonized during $_PROJECTNAME.
                </span>
            </p>
            <p class="c10"><span class="c3">Will you be using standard vocabularies for all data types present in your
                    data set, to allow inter-disciplinary interoperability?</span></p>
            <p class="c0">
                <span class="c1">
                    Open ontologies will be used where they are mature. As stated above, some ontologies and controlled vocabularies might need to be extended. #if$_DATAPLANT Here, $_PROJECTNAME will build on the advanced ontologies integrated in DataPLANT. #endif$_DATAPLANT
                </span>
            </p>
            <p class="c10"><span class="c3">In case it is unavoidable that you use uncommon or generate project specific
                    ontologies or vocabularies, will you provide mappings to more commonly used ontologies?</span></p>
            <p class="c0"><span class="c1">Common and open ontologies will be used, so this question does not
                    apply.</span></p>
            <h3 class="c14" id="h.fs2sjyqtlku7"><span class="c7">Increase data reuse (by clarifying licences)</span>
            </h3>
            <p class="c10"><span class="c12 c17 c11"></span></p>
            <p class="c10"><span class="c3">How will the data be licensed to permit the widest re-use possible?</span>
            </p>
            <p class="c0"><span class="c1">Open licenses, such as Creative Commons (CC), will be used whenever
                    possible.</span></p>


            <p class="c10">
                <span class="c3">
                    When will the data be made available for re-use? If an embargo is sought to give time to publish or
                    seek patents, specify why and how long this will apply, bearing in mind that research data should be
                    made
                    available as soon as possible.
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    <span class="c1">
                        #if$_EARLY Some raw data is made public as soon as it is collected and processed. #endif$_EARLY
                        #if$_BEFOREPUBLICATION Relevant processed datasets are made public when the research findings
                        are published. #endif$_BEFOREPUBLICATION #if$_ENDOFPROJECT At the end of the project, all data
                        without embargo period will be published. #endif$_ENDOFPROJECT #if$_EMBARGO Data, which is
                        subject to an embargo period, is not publicly accessible until the end of embargo
                        period. #endif$_EMBARGO #if$_REQUEST Data is made available upon request, allowing controlled
                        sharing while ensuring responsible use. #endif$_REQUEST #if$_IPISSUE IP issues will be checked
                        before publication. #endif$_IPISSUE All consortium partners will be
                        encouraged to make data available before publication, openly and/or under pre-publication
                        agreements #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto
                        International Data
                        Release Workshop #endif$_GENOMIC . This will be implemented as soon as IP-related checks are
                        complete.
                    </span>
                </span>
            </p>
            <p class="c10"><span class="c3">Are the data produced and/or used in the project usable by third parties, in
                    particular after the end of the project? If the re-use of some data is restricted, explain
                    why.</span></p>
            <p class="c0"><span class="c1">There will be no restrictions once the data are made public.</span></p>
            <p class="c10"><span class="c3">How long is it intended that the data remains re-usable?</span></p>
            <p class="c0"><span class="c1">The data will be made available for many years#if$_DATAPLANT and ideally
                    indefinitely after the end of the project #endif$_DATAPLANT.</span></p>
            <p class="c0">
                <span class="c1">Data submitted to repositories (as detailed above) e.g. ENA /PRIDE would be subject to
                    local data storage regulation.</span>
            </p>
            <p class="c10"><span class="c3">Are data quality assurance processes described?</span></p>
            <p class="c0">
                <span class="c1">
                    The data will be checked and curated. #if$_DATAPLANT Furthermore, data will be quality controlled
                    (QC) using automatic procedures as well as manual curation. #endif$_DATAPLANT
                </span>
            </p>
            <h2 class="c9" id="h.w5pimtt9qf6p"><span class="c4">2.3&nbsp;&nbsp;&nbsp;&nbsp;Allocation of
                    resources</span></h2>
            <p class="c10"><span class="c3">What are the costs for making data FAIR in your project?</span></p>
            <p class="c0"><span class="c1">$_PROJECTNAME will bear the costs of data curation, #if$_DATAPLANT ARC
                    consistency checks, #endif$_DATAPLANT and data maintenance/security before transfer to public
                    repositories. Subsequent costs are then borne by the operators of these repositories.</span></p>
            <p class="c0">
                <span class="c1">
                    Additionally, costs for after publication storage are incurred by end-point repositories (e.g. ENA)
                    but not charged against $_PROJECTNAME or its members but by the operation budget of these
                    repositories.
                </span>
            </p>
            <p class="c10">
                <span class="c3">How will these be covered? Note that costs related to open access to research data are
                    eligible as part of the Horizon 2020 or Horizon Europe grant (if compliant with the Grant Agreement
                    conditions).</span>
            </p>
            <p class="c0"><span class="c1">The cost born by $_PROJECTNAME are covered by the project funding.
                    Pre-existing structures #if$_DATAPLANT such as structures, tools, and knowledge laid down in the
                    DataPLANT consortium #endif$_DATAPLANT will also be used.</span></p>
            <p class="c10"><span class="c3">Who will be responsible for data management in your project?</span></p>
            <p class="c0"><span class="c1">The responsible person will be $_DATAOFFICER of $_PROJECTNAME.</span></p>
            <p class="c10"><span class="c3">Are the resources for long term preservation discussed (costs and potential
                    value, who decides and how/what data will be kept and for how long)?</span></p>
            <p class="c0">
                <span class="c1">
                    The data officer #if$_PARTNERS or $_PARTNERS #endif$_PARTNERS will ultimately decide on the
                    strategy to preserve data that are not submitted to end-point subject area repositories
                    #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT when the
                    project ends. This will be in line with EU guidlines, institute policies, and data sharing based on
                    EU and international standards.
                </span>
            </p>
            <h2 class="c9" id="h.of351jlpnl2l"><span class="c4">2.4&nbsp;&nbsp;&nbsp;&nbsp;Data security</span></h2>
            <p class="c10"><span class="c3">What provisions are in place for data security (including data recovery as
                    well as secure storage and transfer of sensitive data)?</span></p>
            <p class="c20">
                <span class="c1">
                    Online platforms will be protected by vulnerability scanning, two-factor authorization and daily
                    automatic backups allowing immediate recovery. All partners holding confidential project data will use
                    secure platforms with automatic backups and offsite secure copies.
                    #if$_DATAPLANT As ARCs are stored in the PlantDataHub of DataPLANT, data security will be imposed.
                    This comprises secure storage, and the use of password and usernames is generally transferred via
                    separate safe media. #endif$_DATAPLANT
                </span>
            </p>
            <p class="c10"><span class="c3">Is the data safely stored in certified repositories for long term
                    preservation and curation?</span></p>
            
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">
                            As mentioned above, data will be made archived in
                            international discipline related repositories which use specialized technologies: #if$_GENETIC|$_GENOMIC #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_TRANSCRIPTOMIC|$_GENETIC 
                            #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_ARRAYEXPRESS
                            EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC|$_GENOMIC #if$_IMAGE #if$_BIOIMAGE
                            EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                            #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-Metabolights, #endif$_METABOLIGHTS
                            #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct, #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                            EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                            Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                            #endif$_EDAL #endif$_PHENOTYPIC.
                            #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                            well. #endif$_OTHEREP
                        </span>
                    </p>
            <h2 class="c9" ><span class="c4">2.5&nbsp;&nbsp;&nbsp;&nbsp;Ethical aspects</span></h2>
            <p class="c10">
                <span class="c3">
                    Are there any ethical or legal issues that can have an impact on data sharing? These can also be
                    discussed in the context of an ethics review. If relevant, include references to ethics deliverables
                    and
                    ethics chapter in the Description of the Action (DoA).
                </span>
            </p>
            <p class="c0">
                <span class="c1">
                    At the moment, we do not anticipate ethical or legal issues with data sharing. In terms of ethics,
                    since this is plant data, there is no need for an ethics committee to deal with data from plants,
                    although we will diligently follow the Nagoya protocol on access and benefit sharing. #issuewarning
                    Please ensure that you complete any necessary due diligence. Currently, we are awaiting clarification on whether the Nagoya Protocol ( see Nagoya Protocol) will encompass sequence information. Regardless, if you use material from a country other than your own (or that of your partner), and you conduct physical or biochemical characterization (e.g., metabolites, proteome, RNASeq, etc.), this may constitute an action relevant under the Nagoya Protocol. Exceptions might include materials from countries such as the U.S. (non-partner), Ireland (has not signedstill contact them), etc., though other laws could apply.
                    #endissuewarning
                </span>
            </p>
            <p class="c10"><span class="c3">Is informed consent for data sharing and long term preservation included in
                    questionnaires dealing with personal data?</span></p>
            <p class="c0">
                <span class="c1">
                    The only personal data that will potentially be stored is the submitter name and affiliation in the
                    metadata for data. In addition, personal data will be collected for dissemination and communication
                    activities using specific methods and procedures developed by $_PROJECTNAME partners to adhere to
                    data protection. #issuewarning You need to inform and better get WRITTEN consent that you store
                    emails and
                    names or even pseudonyms such as twitter handles, we are very sorry about these issues we didnt
                    invent them #endissuewarning
                </span>
            </p>
            <h2 class="c9" id="h.slb0g3rbvjfl"><span class="c4">2.6&nbsp;&nbsp;&nbsp;&nbsp;Other issues</span></h2>
            <p class="c0"><span class="c3">Do you make use of other national/funder/sectorial/departmental procedures
                    for data management? If yes, which ones?</span></p>
                    <p class="c0"><span class="c1 list-to-remove-comma">Yes, $_PROJECTNAME will use common Research Data Management (RDM) infrastructures #if$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC developed by #endif$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC #if$_DATAPLANT|$_NFDI the NFDI of Germany, #endif$_DATAPLANT|$_NFDI #if$_FRENCH INRAe from France, #endif$_FRENCH #if$_EOSC EOSC (European Open Science Cloud), #endif$_EOSC .
                    </span></p>
            <p class="c6"><span class="c1"></span></p>
            <h1 class="c13">
                <span class="c2">3&nbsp;&nbsp;&nbsp;&nbsp;</span>
                <span class="c12 c2">Annexes</span>
            </h1>
            <p class="c0"><span class="c1"></span></p>
            
            <h2 class="c9"><span class="c11 c18">3.1&nbsp;&nbsp;&nbsp;&nbsp;</span>
                <span class="c4">Abbreviations</span>
            </h2>
            <p class="c0"><span class="c1"></span></p>
            
             #if$_DATAPLANT  <span class="c11">  ARC </span> <span class="c1"> Annotated Research Context</span> <br>  #endif$_DATAPLANT 

             <span class="c11"> CC</span><span class="c1"> Creative Commons</span> <br>
             <span class="c11"> CC CEL</span><span class="c1"> Creative Commons Rights Expressio Language</span> <br>
             #if$_PROTEOMIC #if$_CHEBI <span class="c11"> ChEBI</span><span class="c1"> Chemical Entities of Biological Interest </span> <br>  #endif$_CHEBI #endif$_PROTEOMIC      
             <span class="c11"> DDBJ</span><span class="c1"> DNA Data Bank of Japan</span> <br>
             <span class="c11"> DMP</span><span class="c1"> Data Management Plan</span> <br>
             <span class="c11"> DoA</span><span class="c1"> Description of Action</span> <br>
             <span class="c11"> DOI</span><span class="c1"> Digital Object Identifier</span> <br>
             <span class="c11"> EBI</span><span class="c1"> European Bioinformatics Institute</span> <br>
             #if$_PHENOTYPIC #if$_EDAL  <span class="c11"> e!DAL-PGP</span><span class="c1"> Plant Genomics & Phenomics Research Data Repository</span> <br>  #endif$_EDAL   #endif$_PHENOTYPIC  
               #if$_GENETIC  #if$_ENA   <span class="c11">  ENA </span> <span class="c1"> European Nucleotide Archive </span> <br>  #endif$_ENA        #endif$_GENETIC   
             <span class="c11"> EU </span><span class="c1">European Union</span> <br>
             <span class="c11"> FAIR </span><span class="c1">Findable Accessible Interoperable Reproducible</span> <br>
             <span class="c11"> GDPR</span><span class="c1"> General data protection regulation (of the EU)</span> <br>
              #if$_TRANSCRIPTOMIC|$_GENETIC #if$_GEO  <span class="c11"> GEO</span><span class="c1"> Gene Expression Omnibus </span>  <br>  #endif$_GEO   #endif$_TRANSCRIPTOMIC|$_GENETIC   
               #if$_IMAGE #if$_IDR  <span class="c11"> IDR</span><span class="c1"> Image Data Resource </span>  <br>  #endif$_IDR  #endif$_IMAGE 
             <span class="c11"> IP</span><span class="c1"> Intellectual Property</span> <br>
            <span class="c11"> ISO</span><span class="c1"> International Organization for Standardization</span>  <br>
            #if$_TRANSCRIPTOMIC #if$_MIAME <span class="c11"> MIAME</span><span class="c1"> Minimum Information About a Microarray Experiment</span> <br>  #endif$_MIAME  #endif$_TRANSCRIPTOMIC  
            #if$_METABOLOMIC #if$_MMIAMET <span class="c11"> MIAMET</span><span class="c1"> Minimum Information About a METabolomics experiment</span> <br>  #endif$_MMIAMET   #endif$_METABOLOMIC  
            #if$_PROTEOMIC #if$_MIAPE <span class="c11"> MIAPE</span><span class="c1"> Minimum Information About a Proteomics Experiment</span> <br>  #endif$_MIAPE  #endif$_PROTEOMIC  
             #if$_PHENOTYPIC #if$_MIAPPE <span class="c11"> MIAPPE</span><span class="c1"> Minimum Information about Plant Phenotyping Experiment</span> <br>  #endif$_MIAPPE  #endif$_PHENOTYPIC 
            #if$_GENOMIC|$_GENETIC #if$_MIGSEU <span class="c11"> MigsEu </span><span class="c1"> Minimum Information about a Genome Sequence: Eucaryote </span> <br>  #endif$_MIGSEU  
             #if$_MIGSORG <span class="c11"> MigsOrg </span><span class="c1"> Minimum Information about a Genome Sequence: Organelle </span> <br>  #endif$_MIGSORG  
             #if$_MIMAG <span class="c11"> MIMAG </span><span class="c1"> Minimum Information about Metagenome-Assembled Genome </span> <br>  #endif$_MIMAG  
            #if$_MIMARKSSPECIMEN <span class="c11"> MIMARKSSpecimen</span><span class="c1"> Minimal Information about a Marker Specimen: Specimen </span> <br>  #endif$_MIMARKSSPECIMEN 
             #if$_MIMARKSSURVEY  <span class="c11">  MIMARKSSurvey </span><span class="c1"> Minimal Information about a Marker Specimen: Survey </span> <br>  #endif$_MIMARKSSURVEY    #endif$_GENOMIC|$_GENETIC  
           #if$_PROTEOMIC #if$_MIMIX <span class="c11"> MIMIX</span><span class="c1"> The Minimum Information required for reporting a Molecular Interaction Experiment </span> <br>  #endif$_MIMIX   #endif$_PROTEOMIC 
            #if$_GENOMIC|$_GENETIC #if$_MIMS  <span class="c11">  MIMS</span><span class="c1"> Molecular Interactions </span> <br>  #endif$_MIMS    #endif$_GENOMIC|$_GENETIC 
            #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_MINSEQE <span class="c11"> MinSEQe</span><span class="c1"> Minimum Information about a high-throughput Sequencing Experiment</span> <br>  #endif$_MINSEQE #endif$_TRANSCRIPTOMIC|$_GENOMIC 
            #if$_GENOMIC|$_GENETIC #if$_MISAG <span class="c11"> MISAG</span><span class="c1"> Minimum Information about a Single Amplified Genome </span> <br>  #endif$_MISAG
            #if$_MIXS <span class="c11"> MIxS</span><span class="c1"> Minimum Information about any (X) Sequence </span> <br>  #endif$_MIXS #endif$_GENOMIC|$_GENETIC 
             <span class="c11"> NCBI</span><span class="c1"> National Center for Biotechnology Information</span> <br>
             <span class="c11"> NFDI</span><span class="c1"> National Research Data Infrastructure (of Germany)</span> <br>
             <span class="c11"> NGS</span><span class="c1"> Next Generation Sequencing</span> <br>
             #if$_PROTEOMIC #if$_PRIDE  <span class="c11"> PRIDE</span><span class="c1"> PRoteomics IDEntifications Database</span> <br>  #endif$_PRIDE
            #if$_PDB  <span class="c11"> PDB</span><span class="c1"> Protein Data Bank </span> <br>  #endif$_PDB #endif$_PROTEOMIC
             <span class="c11"> RDM</span><span class="c1"> Research Data Management</span> <br>
            #if$_IMAGE  #if$_REMBI  <span class="c11"> REMBI</span><span class="c1"> Recommended Metadata for Biological Images </span> <br>  #endif$_REMBI #endif$_IMAGE 

            #if$_RNASEQ  <span class="c11"> RNASeq</span><span class="c1"> RNA Sequencing</span> <br>  #endif$_RNASEQ 
            <span class="c11"> SOP</span><span class="c1"> Standard Operating Procedures</span>  <br>
            #if$_GENETIC|$_GENOMIC #if$_SRA  <span class="c11"> SRA</span><span class="c1"> Sequence Read Archive</span> <br>  #endif$_SRA #endif$_GENETIC|$_GENOMIC
            #if$_DATAPLANT  <span class="c11">  SWATE</span><span class="c1"> Swate Workflow Annotation Tool for Excel</span> <br>  #endif$_DATAPLANT 
         <span class="c11"> ONP</span><span class="c1"> Oxford Nanopore</span>  <br>
            <span class="c11"> qRT</span> <span class="c11"> PCR</span><span class="c1"> quantitative real time polymerase chain reaction</span>  <br>
             <span class="c11"> WP</span><span class="c1"> Work Package</span>  <br>
            

        </div>
    </div>`, 
"horizon_europe":`<div id="horizon_europe" >
    <div class="content-page">
        <br />
        <center>
            <div class="p-2" id="vis-parent" style="margin: auto;">

                <div id="vis" style=" width:100%;max-height:600px;  aspect-ratio : 1 / 1;"></div>

            </div>
            <br />
            <div class="p-2" style="margin: auto;">


                <div class="c2 text-center">
                    <h1>Data Management Plan of the Horizon Europe Project $_PROJECTNAME</h1>
                </div>

                <div class="d-flex justify-content-center">


                    <table>
                        <tbody>
                            <tr>
                                <td>
                                    <h3>Action Number: </h3>
                                </td>
                                <td>
                                    <h3> $_FUNDINGPROGRAMME </h3>
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    <h3>Action Acronym:</h3>
                                </td>
                                <td>
                                    <h3> $_PROJECTNAME</h3>
                                </td>
                            </tr>
                            <tr>
                                <td>
                                    <h3>Action Title:</h3>
                                </td>
                                <td>
                                    <h3> $_PROJECTNAME</h3>
                                </td>

                            </tr>
                            <tr>
                                <td>
                                    <h2>Creation Date:</h2>
                                </td>
                                <td>
                                    <h3>$_CREATIONDATE</h3>
                                </td>

                            </tr>
                            <tr>
                                <td>
                                    <h3>Modification Date:</h2>
                                </td>
                                <td>
                                    <h3>$_MODIFICATIONDATE</h3>
                                </td>

                            </tr>
                        </tbody>
                    </table>

                </div>
                <br>

            </div>

        </center>

        <h1 class="c13"><span class="c2">Introduction</span></h1>
        <p class="c0"><span class="c1">#if$_EU $_PROJECTNAME is part of the Open Data Initiative (ODI) of the EU.
                #endif$_EU To best profit from open data, it is necessary not only to store data but to make data
                Findable, Accessible, Interoperable, and Reusable (FAIR).#if$_PROTECT We support open and FAIR data,
                however, we also consider the need to protect individual data sets. #endif$_PROTECT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The aim of this document is to provide guidelines on principles guiding the data management in $_PROJECTNAME. A Data Management Plan (DMP) will generated by using the responses to the EU questionnaire on DMP, which defines the data stored in $_PROJECTNAME.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The detailed DMP instructs how data will be handled during and after the project. The $_PROJECTNAME DMP
                is modified according to the Horizon Europe and Horizon Europe online Manual. #if$_UPDATE It will be
                updated/its validity checked during the
                $_PROJECTNAME project several times. At the very least, this will happen at month $_UPDATEMONTH.
                #endif$_UPDATE</span>
        </p>

        <h2 class="c9" id="h.3l7mu272odf4"><span class="c4">1.&nbsp; &nbsp; Data Summary</span></h2>
        <p class="c0"><span class="c3">Will you re-use any existing data and what will you re-use it for? State the
                reasons if re-use of any existing data has been considered but discarded.</span></p>
        <p class="c0">

            <span class="c1">
                The project builds on existing data sets and relies on them. #if$_RNASEQ For instance, without a
                proper genomic reference it is very difficult to analyze NGS data sets. #endif$_RNASEQ It is also
                important to include existing data sets on the expression and metabolic behaviour of $_STUDYOBJECT,
                but of course, also on existing characterization and the background knowledge. #if$_PARTNERS of the
                partners: $_PARTNERS #endif$_PARTNERS Genomic references can simply be gathered from reference databases for
                genomes/sequences, like the National Center for Biotechnology Information: NCBI (US); European
                Bioinformatics Institute: EBI (EU); DNA Data Bank of Japan: DDBJ (JP). Furthermore, prior
                'unstructured' data in the form of publications and data contained therein will be used for decision
                making.
            </span>
        </p>
        <p class="c0"><span class="c3">What types and formats of data will the project generate or re-use?</span>
        </p>


        <p class="c0">
            <span class="c1">
                $_PROJECTNAME will collect and/or generate the following types of raw data: 
                #if$_GENETIC genetic data, #endif$_GENETIC  
                #if$_GENOMIC genomic data, #endif$_GENOMIC 
                #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA 
                #if$_TRANSCRIPTOMIC transcriptomic data, #endif$_TRANSCRIPTOMIC  
                #if$_RNASEQ RNAseq data, #endif$_RNASEQ 
                #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC  
                #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC 
                #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC  
                #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED  
                #if$_IMAGE image datasets, #endif$_IMAGE  
                #if$_MODELS modelling data, #endif$_MODELS 
                #if$_CODE computational code, #endif$_CODE  
                #if$_EXCEL excel files, #endif$_EXCEL and other types of data which are related to $_STUDYOBJECT. In addition, the raw data will also
                be processed and modified using analytical pipelines, which may yield different results or include
                ad hoc data analysis parts. #if$_DATAPLANT These pipelines will be tracked in the DataPLANT
                Annotated Research Context (ARC) . #endif$_DATAPLANT Therefore, care will be taken to document and archive these resources
                (including the analytical pipelines) as well#if$_DATAPLANT relying on the expertise in the DataPLANT
                consortium #endif$_DATAPLANT.
            </span>
        </p>

        <p class="c0"><span class="c3">What is the purpose of the data generation or re-use and its relation to the
                objectives of the project?</span></p>
        <p class="c0">
            <span class="c1">
                $_PROJECTNAME has the following aim: $_PROJECTAIM. Therefore, data collection #if!$_VVISUALIZATION
                and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION , integration and visualization
                #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT ARC structure are absolutely necessary,
                #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data management process is absolutely
                necessary, #endif!$_DATAPLANT because the data are used not only to understand principles, but also
                be informed about the provenance of data analysis information. Stakeholders must also be informed
                about the provenance of data. It is therefore necessary to ensure that the data are well generated
                and also well annotated with metadata using open standards, as laid out in the next section.
            </span>
        </p>


        <p class="c0"><span class="c3">What is the expected size of the data that you intend to generate or
                re-use?</span></p>
        <p class="c0"><span class="c1">We expect to generate raw data in the range of $_RAWDATA GB of data. The size
                of the derived data will be about $_DERIVEDDATA GB.</span></p>

        <p class="c0"><span class="c3">What is the origin/provenance of the data, either generated or
                re-used?</span></p>
        <p class="c0"><span class="c1">Public data will be extracted as described in the previous paragraph. For $_PROJECTNAME, specific data sets will be generated by the consortium partners.</span></p>


        <p class="c0">
            <span class="c1"> Data of different types or representing different domains will be generated using
                unique approaches. For example:</span>
        </p>
        <ul style="list-style-type:disc;">
            #if$_GENETIC&nbsp; <li>
                    <span class="c1"> Genetic data will be generated targeting crosses and breeding experiments, and
                        will include recombination frequencies and crossover event that position genetic markers and
                        quantitative trait loci that can be associated with physical genomic markers/variants.
                    </span>
            </li> #endif$_GENETIC #if$_GENOMIC&nbsp; 
            <li>
                
                    <span class="c1"> Genomic data will be created from sequencing data, which will be processed to
                        identify genes, regulatory elements, transposable elements, and physical markers such as
                        SNPs, microsatellites and structural variants. </span>

            </li>
            #endif$_GENOMIC #if$_CLONED-DNA&nbsp; 
            <li>

                    <span class="c1"> The origin and assembly of cloned DNA will include (a) source of original
                        vector sequence with Add gene reference where available, and source of insert DNA (e.g.,
                        amplification by PCR from a given sample, or obtained from existing library), (b) cloning
                        strategy (e.g., restriction endonuclease digests/ligation, PCR, TOPO cloning, Gibson
                        assembly, LR recombination), and (c) verified DNA data sequence of final recombinant vector.
                    </span>

            </li>
            #endif$_CLONED-DNA  #if$_TRANSCRIPTOMIC&nbsp; 
            <li>

                    <span class="c1"> Methods of transcriptomics data collection will be selected from microarrays,
                        quantitative PCR, Northern blotting, RNA immunoprecipitation, fluorescence in situ
                        hybridization. RNA-Seq data will be collected in seperate methods.</span>

            </li>
            #endif$_TRANSCRIPTOMIC&nbsp;  #if$_RNASEQ&nbsp; 
            <li>

                    <span class="c1"> RNA sequencing will be generated using short-read or long-read plantforms,
                        either in house or outsourced to academic facilities or commercial services, and the raw
                        data will be processed using estabilished bioinformatics piplines. </span>
    
            </li>
            #endif$_RNASEQ&nbsp;  #if$_METABOLOMIC&nbsp; 
            <li>

                    <span class="c1"> Metabolomic data will be generated by coupled chromatography and mass
                        spectrometry using targeted or untargeted approaches.</span>

            </li>
            #endif$_METABOLOMIC  #if$_PROTEOMIC&nbsp; 
            <li>
        
                    <span class="c1"> Proteomic data will be generated using coupled chromatography and mass
                        spectrometry for the analysis of protein abundance and protein identification, as well as
                        additional techniques for structural analysis, the identification of post-translational
                        modifications and the characterization of protein interactions.</span>

            </li>
            #endif$_PROTEOMIC  #if$_PHENOTYPIC&nbsp; 
            <li>

                    <span class="c1"> Phenotypic data will be generated using phenotyping platforms and
                        corresponding ontologies, including number/size of organs such as leaves, flowers, buds
                        etc., size of whole plant, stem/root architecture (number of lateral branches/roots etc),
                        organ structures/morphologies, quantitative metrics such as color, turgor, health/nutrition
                        indicators, among others. </span>

            </li>
            #endif$_PHENOTYPIC  #if$_TARGETED&nbsp; 
            <li>

                    <span class="c1"> Targeted assays data (e.g. glucose and fructose concentrations or
                        production/ultilization rates) will be generated using specific equipment and methods that
                        are fully documented in the laboratory notebook. </span>
            
            </li>
            #endif$_TARGETED #if$_IMAGE&nbsp; 
            <li>
        
                    <span class="c1"> Image data will be generated by equipment such as cameras, scanners, and
                        microscopes combined with software. Original images which contain metadata such as exif
                        photo information will be archived.</span>
            
            </li>
            #endif$_IMAGE #if$_MODELS&nbsp; 
            <li>
            
                    <span class="c1"> Model data will be generated by using software simulations. The complete
                        workflow, which includes the environment, runtime, parameters, and results, will be
                        documented and archived. </span>
            
            </li>
            #endif$_MODELS #if$_CODE&nbsp; 
            <li>
                
                    <span class="c1"> Computer code will be produced by programmers. </span>
            
            </li>
            #endif$_CODE  #if$_EXCEL&nbsp; 
            <li>
                
                    <span class="c1"> Excel data will be generated by data analysts by using MS Office or
                        open-source software. </span>
            
            </li>
            #endif$_EXCEL 

            

            
        </ul>


        #if$_PREVIOUSPROJECTS&nbsp;
        <p class="c0"><span class="c1"> Data from previous projects such as $_PREVIOUSPROJECTS will be considered.
            </span></p>
        #endif$_PREVIOUSPROJECTS

        <p class="c0"><span class="c3">To whom might it be useful ('data utility'), outside your project?</span></p>
        <p class="c0">
            <span class="c1">
                The data will initially benefit $_PROJECTNAME partners, but will also be made available to selected
                stakeholders closely involved in the project, and then the scientific community working on
                $_STUDYOBJECT. $_DATAUTILITY In addition, the general public interested in $_STUDYOBJECT can also
                use the data after publication. The data will be disseminated according to $_PROJECTNAME's
                dissemination and communication plan #if$_DATAPLANT , which aligns with DataPLANT platform or other means #endif$_DATAPLANT.
            </span>
        </p>

        <h2 class="c9"><span class="c4">2&nbsp; &nbsp; FAIR data</span></h2>
        <h3 class="c14"><span class="c7">2.1. Making data findable, including provisions for metadata</span></h3>
        <p class="c0">
            <span class="c3">
                Will data be identified by a persistent identifier?
            </span>
        </p>
        <p class="c0">
            <span class="c1">All data sets will receive unique identifiers, and they will be annotated with
                metadata.
            </span>
        </p>

        <p class="c0">
            <span class="c3">Will rich metadata be provided to allow discovery? What metadata will be created? What
                disciplinary or general standards will be followed? In case metadata standards do not exist in your
                discipline, please outline what type of metadata will be created and how.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                All datasets will be associated with unique identifiers and will be annotated with metadata. We will
                use Investigation, Study, Assay (ISA) specification for metadata creation. $_PROJECTNAME will rely
                on community standards plus additional recommendations applicable in the plant science, such as the
                <span class="list-to-remove-comma">
                    The following metadata/ minimum informatin standards will be used to collect metadata:
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS (Minimum Information about any (X) Sequence), #endif$_MIXS
                    #if$_MIGSEU MigsEu (Minimum Information about a Genome Sequence: Eucaryote), #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg (Minimum Information about a Genome Sequence: Organelle), #endif$_MIGSORG
                    #if$_MIMS MIMS (Minimum Information about Metagenome or Environmental), #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen (Minimal Information about a Marker Specimen:
                    Specimen), #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey (Minimal Information about a Marker Specimen:
                    Survey), #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG (Minimum Information about a Single Amplified Genome), #endif$_MISAG
                    #if$_MIMAG MIMAG (Minimum Information about Metagenome-Assembled Genome), #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_TRANSCRIPTOMIC
                    #if$_MINSEQE MINSEQE (Minimum Information about a high-throughput SEQuencing
                    Experiment), #endif$_MINSEQE #endif$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME (Minimum Information About a Microarray
                    Experiment), #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI (Recommended Metadata for Biological Images), #endif$_REMBI
                    #endif$_IMAGE
                    #if$_RNASEQ|$_GENOMIC 
                    #if$_MINSEQE
                        MinSEQe (Minimum Information about a high-throughput Sequencing Experiment),
                    #endif$_MINSEQE 
                    #endif$_RNASEQ|$_GENOMIC
                    #if$_METABOLOMIC
                    #if$_MMIAMET
                    MIAMET (Minimum Information About a METabolomics experiment),
                    #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE (Minimum Information About a Proteomics Experiment), #endif$_MIAPE
                    #if$_MIMIX MIMix (The Minimum Information required for reporting a Molecular Interaction Experiment), #endif$_MIMIX
                    #endif$_PROTEOMIC  
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE (Minimum Information about Plant Phenotyping Experiment) #endif$_MIAPPE  #endif$_PHENOTYPIC .
                    </span>
            </span>
            <span class="c1"> #if$_METABOLOMIC #if$_METABOLIGHTS The Metabolights submission compliant standards are used for metabolomic data. #issuewarning Some metabolomics partners considers Metabolights
                not an accepted standard. #endissuewarning #endif$_METABOLIGHTS #endif$_METABOLOMIC These specific standard unlike cross-domain minimal sets such as the Dublin core, which
                mostly define the submitter and the general type of data, allow reusability by other researchers by
                defining properties of the plant (see the preceding section). However, $_PROJECTNAME also implement minimal cross-domain annotations #if$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 such as #endif$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 <span class="list-to-remove-comma"> #if$_DUBLINCORE Dublin Core, #endif$_DUBLINCORE #if$_DARWINCORE Darwin Core, #endif$_DARWINCORE #if$_SCHEMAORG Schema.org, #endif$_SCHEMAORG #if$_BIOSCHEMAS BioSchemas, #endif$_BIOSCHEMAS #if$_MARC21 MARC 21 #endif$_MARC21 . </span> #if$_DATAPLANT The core integration with DataPLANT will also allow
                individual releases to be tagged with a Digital Object Identifier (DOI).
                #endif$_DATAPLANT #if$_OTHERSTANDARDS Other standards such as $_OTHERSTANDARDINPUT are also adhered
                to. #endif$_OTHERSTANDARDS
            </span>
        </p>
        <p class="c0"><span class="c3">Will search keywords be provided in the metadata to optimize the possibility
                for discovery and then potential re-use?</span></p>
        <p class="c0">
            <span class="c1">
                Keywords about the experiment and the general consortium will be included, as well as an abstract
                about the data, where useful. In addition, certain keywords can be auto-generated from dense
                metadata and its
                underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with standardized
                DataPLANT ontologies that are supplemented where the ontology does not yet include the variables.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0"><span class="c3">Will metadata be offered in such a way that it can be harvested and
                indexed?</span></p>
        <p class="c0">
            <span class="c1">
                To maintain data integrity and to be able to re-analyze data, data sets will get version numbers
                where this is useful (e.g. raw data must not be changed and will not get a version number and is
                considered
                immutable). #if$_DATAPLANT This is automatically supported by the ARC Git DataPLANT infrastructure.
                #endif$_DATAPLANT</span>

            <span class="c1">
                Data variables will be allocated standard names. For example, genes, proteins and metabolites will
                be named according to approved nomenclature and conventions. These will also be linked to functional
                ontologies where possible. Datasets will also be named I a meaningful way to ensure readability by
                humans. Plant names will include traditional names, binomials, and all
                strain/cultivar/subspecies/variety identifiers.</span>
        </p>



        <h2 class="c9" id="h.3l7mu272odf4"><span class="c4">2.2.&nbsp; &nbsp; Making data accessible</span></h2>
        <p class="c0"><span class="c3">Repository</span></p>
        <p class="c0"><span class="c3">Will the data be deposited in a trusted repository?</span></p>

        <p class="c0">
            <span class="c1">
                Data will be made available via the $_PROJECTNAME platform using a user-friendly front end that allows
                data visualization. Besides this it will be ensured that data which can be stored in
                international discipline related repositories which use specialized technologies:
                


                 <p class="c0">
                        <span class="c1 list-to-remove-comma"> #if$_GENETIC|$_GENOMIC For genetic or genomic data: #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_GENETIC #if$_SRA NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                            #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #endif$_GENETIC #if$_GENETIC|$_GENOMIC . #endif$_GENETIC|$_GENOMIC </span>
                    </p>

                <p class="c0">
                    <span class="c1  list-to-remove-comma">#if$_TRANSCRIPTOMIC For Transcriptomic data: #if$_SRA
                        NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #if$_ARRAYEXPRESS
                        EBI-ArrayExpress, #endif$_ARRAYEXPRESS . #endif$_TRANSCRIPTOMIC</span>
                </p>

                <p class="c0">
                    <span class="c1 list-to-remove-comma">#if$_IMAGE For image data: #if$_BIOIMAGE EBI-BioImage
                        Archive, #endif$_BIOIMAGE #if$_IDR IDR (Image Data Resource), #endif$_IDR .
                        #endif$_IMAGE </span>
                </p>

                <p class="c0">
                    <span class="c1 list-to-remove-comma">#if$_METABOLOMIC For metabolomic data: #if$_METABOLIGHTS
                        EBI-Metabolights, #endif$_METABOLIGHTS #if$_METAWORKBENCH Metabolomics
                        Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct (Molecular
                        interactions), #endif$_INTACT . #endif$_METABOLOMIC </span>
                </p>
                <p class="c0">
                    <span class="c1 list-to-remove-comma">#if$_PROTEOMIC For proteomics data: #if$_PRIDE
                        EBI-PRIDE (PRoteomics IDEntifications Database), #endif$_PRIDE #if$_PDB PDB (Protein Data Bank), #endif$_PDB
                        #if$_CHEBI Chebi (Chemical Entities of Biological Interest), #endif$_CHEBI .
                        #endif$_PROTEOMIC </span>
                </p>

                <p class="c0">
                    <span class="c1 list-to-remove-comma">#if$_PHENOTYPIC For phenotypic data: #if$_EDAL e!DAL-PGP (Plant
                        Genomics & Phenomics Research Data Repository), #endif$_EDAL . #endif$_PHENOTYPIC
                        #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                        well. #endif$_OTHEREP
                    </span>
                </p>
            </span>
        </p>


        <p class="c0"><span class="c3">Have you explored appropriate arrangements with the identified repository
                where your data will be deposited? </span></p>
        <p class="c0">
            <span class="c1">
                The submission is for free, and it is the goal (at least of ENA) to obtain as much data as possible.
                Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required.
                #if$_DATAPLANT For DataPLANT, this has been agreed upon, as all the omics repositories of
                International Nucleotide Sequence Database Collaboration (INSDC) will be used. #endif$_DATAPLANT
                #issuewarning If no data management platform such as DataPLANT is used, then you need to find
                appropriate repository to store or archive your data after publication. #endissuewarning
            </span>
        </p>

        <p class="c0"><span class="c3">Does the repository ensure that the data is assigned an identifier? Will the
                repository resolve the identifier to a digital object? </span></p>
        <p class="c0">
            <span class="c1">
                
                <p class="c0">
                    <span class="c1 list-to-remove-comma">
                        As mentioned above, data will be made archived in
                        international discipline related repositories which use specialized technologies: #if$_GENETIC|$_GENOMIC #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_TRANSCRIPTOMIC|$_GENETIC 
                        #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_ARRAYEXPRESS
                        EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC|$_GENOMIC #if$_IMAGE #if$_BIOIMAGE
                        EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                        #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-Metabolights, #endif$_METABOLIGHTS
                        #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct, #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                        EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                        Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                        #endif$_EDAL #endif$_PHENOTYPIC.
                        #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                        well. #endif$_OTHEREP
                    </span>
                </p> 
                In the case of unstructured less standardized data (e.g. experimental
                phenotypic measurements), these will be metadata annotated and if complete given a digital object
                identifier (DOI) #if$_DATAPLANT and the whole data sets wrapped into an ARC will get DOIs as well
                #endif$_DATAPLANT .
            </span>
        </p>
        Those repositories are the most appropriate ones.
        <br>
        <br>
        <p class="c0"><span class="c3">Data:</span></p>
        <p class="c0"><span class="c3">Will all data be made openly available? If certain datasets cannot be shared
                (or need to be shared under restricted access conditions), explain why, clearly separating legal and
                contractual reasons from intentional restrictions. Note that in multi-beneficiary projects it is
                also possible for specific beneficiaries to keep their data closed if opening their data goes
                against their legitimate interests or other constraints as per the Grant Agreement.</span></p>
        <p class="c0">
            <span class="c1">
                By default, all data sets from $_PROJECTNAME will be shared with the community and made openly
                available. However, before the data are released, all will be provided with an opportunity to check
                for potential IP (according to the consortium agreement and background IP rights). #if$_INDUSTRY
                This applies in particular to data pertaining to the industry. #endif$_INDUSTRY IP protection will
                be prioritized for datasets that offer the potential for exploitation.
            </span>
        </p>
        <p class="c0">
            <span class="c3">
                Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their
                data closed if relevant provisions are made in the consortium agreement and are in line with the
                reasons
                for opting out.
            </span>
        </p>

        <p class="c0">
        <span class="c3">If an embargo is applied to give time to publish or seek protection of the
                intellectual property (e.g. patents), specify why and how long this will apply, bearing in mind that
                research data should be made available as soon as possible.</span></p>
        <p class="c0">
            <span class="c1">
                #if$_EARLY Some raw data is made public as soon as it is collected and processed. #endif$_EARLY
                #if$_BEFOREPUBLICATION Relevant processed datasets are made public when the research findings are
                published. #endif$_BEFOREPUBLICATION #if$_ENDOFPROJECT At the end of the project, all data without
                embargo period will be published. #endif$_ENDOFPROJECT #if$_EMBARGO Data, which is subject to an
                embargo period, is not publicly accessible until the end of embargo period. #endif$_EMBARGO
                #if$_REQUEST Data is made available upon request, allowing controlled sharing while ensuring
                responsible use. #endif$_REQUEST #if$_IPISSUE IP issues will be checked before publication.
                #endif$_IPISSUE All consortium partners will be
                encouraged to make data available before publication, openly and/or under pre-publication agreements
                #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto International
                Data
                Release Workshop #endif$_GENOMIC . This will be implemented as soon as IP-related checks are
                complete.
            </span>
        </p>

        <p class="c0"><span class="c3">Will the data be accessible through a free and standardized access protocol?
            </span></p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT DataPLANT stores data in the ARC, which is a git repo. The DataHUB shares data and
                metadata as a gitlab instance. The "Git" and "Web" protocol are opensourced and freely accessible.
                In addition, #endif$_DATAPLANT Zenodo and the endpoint repositories will also be used for access. In
                General, web-based protocols are free and standardized for access.
            </span>
        </p>

        <p class="c0"><span class="c3">If there are restrictions on use, how will access be provided to the data,
                both during and after the end of the project?
            </span></p>
        <p class="c0">
            <span class="c1">
                There are no restrictions, beyond the aforementioned IP checks, which are in line with e.g. European
                open data policies.
            </span>
        </p>


        <p class="c0"><span class="c3">How will the identity of the person accessing the data be ascertained?
            </span></p>
        <p class="c0">
            <span class="c1">
                In case data is only shared within the consortium, if the data is not yet finished or under IP
                checks, the data is hosted internally and username and password will be required (see also our GDPR
                rules). In the case data is made public under final EU or US repositories, completely anonymous
                access is normally allowed. This is the case for ENA as well and both are in line with GDPR
                requirements.
                #if$_DATAPLANT Currently, data management relies on the annotated research context ARC. It is
                password protected, so before any data can be obtained or samples generated an authentication needs
                to take place. #endif$_DATAPLANT

            </span>
        </p>
        <p class="c0"><span class="c3">Is there a need for a data access committee (e.g. to evaluate/approve access
                requests to personal/sensitive data)?
            </span></p>
        <p class="c0">
            <span class="c1">
                Consequently, there is no need for a committee.

            </span>
        </p>
        <p class="c0">
            <span class="c3">
                Metadata:
            </span>
        </p>
        <p class="c0">
            <span class="c3">
                Will metadata be made openly available and licenced under a public domain dedication CC0, as per the
                Grant Agreement? If not, please clarify why. Will metadata contain information to enable the user to
                access the data?
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                Yes, where possible, e.g. CC REL will be used for data not submitted to specialized repositories
                such as ENA.
            </span>
        </p>
        <p class="c0">
            <span class="c3">
                How long will the data remain available and findable? Will metadata be guaranteed to remain
                available after data is no longer available?
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                <p class="c0"><span class="c1">The data will be made available for many years#if$_DATAPLANT and
                        ideally indefinitely after the end of the project #endif$_DATAPLANT.
                        In any case data submitted to repositories (as detailed above) e.g. ENA /PRIDE would be
                        subject to local data storage regulation.

                    </span></p>

            </span>
        </p>

        <p class="c0">
            <span class="c3">
                Will documentation or reference about any software be needed to access or read the data be included?
                Will it be possible to include the relevant software (e.g. in open source code)?
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_PROPRIETARY $_PROJECTNAME relies on the tool(s) $_PROPRIETARY. #endif$_PROPRIETARY
                #if!$_PROPRIETARY No specialized software will be needed to access the data, usually just a modern
                browser. Access will be possible through web interfaces. For data processing after obtaining raw
                data, typical open-source software can be used. #endif!$_PROPRIETARY
                #if$_DATAPLANT DataPLANT offers tools such as the open-source SWATE plugin for Excel, the ARC
                commander, and the DMP tool which will not necessarily make the interaction with data more
                convenient. #endif$_DATAPLANT #if$_DATAPLANT However, DataPLANT resources are well described, and
                their setup is documented on their github project pages. #endif$_DATAPLANT
                As stated above, here we use publicly available open-source and well-documented certified software, #if$_PROPRIETARY except for $_PROPRIETARY #endif$_PROPRIETARY.
            </span>
        </p>

        <h3><span class="c7"> 2.3. Making data interoperable </span></h3>
        <p class="c0">
            <span class="c3">
                What data and metadata vocabularies, standards, formats or methodologies will you follow to make
                your data interoperable to allow data exchange and re-use within and across disciplines? Will you
                follow community-endorsed interoperability best practices? Which ones?
            </span>
        </p>
        <p class="c0">
            
        
                <span class="c1 list-to-remove-comma">We will use Investigation, Study, Assay (ISA) specification for metadata creation.
                    The following metadata standards will also be used:
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE, #endif$_MIAPPE #endif$_PHENOTYPIC
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS, #endif$_MIXS
                    #if$_MIGSEU MigsEu, #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg, #endif$_MIGSORG
                    #if$_MIMS MIMS, #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen, #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey, #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG, #endif$_MISAG
                    #if$_MIMAG MIMAG, #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_GENOMIC|$_TRANSCRIPTOMIC #if$_MINSEQE MINSEQE, #endif$_MINSEQE #endif$_GENOMIC|$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME, #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI, #endif$_REMBI
                    #endif$_IMAGE
                    #if$_METABOLOMIC
                    #if$_MMIAMET MIAMET, #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE, #endif$_MIAPE
                    #if$_MIMIX MIMix, #endif$_MIMIX
                    #endif$_PROTEOMIC.
                    
                </span>
        
            
            <span class="c1">#if$_METABOLOMIC #if$_METABOLIGHTS The Metabolights submission compliant standards are used for metabolomic data. #issuewarning Some metabolomics partners considers Metabolights
                not an accepted standard. #endissuewarning #endif$_METABOLIGHTS #endif$_METABOLOMIC These specific standard unlike cross-domain minimal sets such as the Dublin core, which
                mostly define the submitter and the general type of data, allow reusability by other researchers by
                defining properties of the plant (see the preceding section). However, $_PROJECTNAME also implement minimal cross-domain annotations #if$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 such as #endif$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 <span class="list-to-remove-comma"> #if$_DUBLINCORE Dublin Core, #endif$_DUBLINCORE #if$_DARWINCORE Darwin Core, #endif$_DARWINCORE #if$_SCHEMAORG Schema.org, #endif$_SCHEMAORG #if$_BIOSCHEMAS BioSchemas, #endif$_BIOSCHEMAS #if$_MARC21 MARC 21 #endif$_MARC21 . </span> #if$_DATAPLANT The core integration with DataPLANT will also allow
                individual releases to be tagged with a Digital Object Identifier (DOI).
                #endif$_DATAPLANT #if$_OTHERSTANDARDS Other standards such as $_OTHERSTANDARDINPUT are also adhered
                to. #endif$_OTHERSTANDARDS
            </span>
        </p>
        Whenever possible, data will be stored in common and openly defined formats including all the necessary
        metadata to interpret and analyze data in a biological context. By default, no proprietary formats will be
        used. However Microsoft Excel files (according to ISO/IEC 29500-1:2016) might be used as intermediates by
        the consortium#if$_DATAPLANT and by some ARC components #endif$_DATAPLANT. In addition, text files might be
        edited in text processor files, but will be shared as pdf.
        Open ontologies will be used where they are mature. As stated above, some ontologies and controlled
        vocabularies might need to be extended. #if$_DATAPLANT Here, $_PROJECTNAME will build on the advanced
        ontologies integrated in DataPLANT. #endif$_DATAPLANT


        </span>
        </p>



        <p class="c0">
            <span class="c3">
                In case it is unavoidable that you use uncommon or generate project specific ontologies or
                vocabularies, will you provide mappings to more commonly used ontologies? Will you openly publish
                the generated ontologies or vocabularies to allow reusing, refining or extending them?
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                Common and open ontologies will be used. In fact, open biomedical ontologies will be used where they
                are mature. As stated in the previous question, sometimes ontologies and controlled vocabularies
                might have to be extended. #if$_DATAPLANT Here, $_PROJECTNAME will build on the DataPLANT biology
                ontology (DPBO) developed in DataPLANT. #endif$_DATAPLANT Ontology databases such as OBO Foundry
                will be used to publish ontology. #if$_DATAPLANT The DPBO is also published in GitHub
                https://github.com/nfdi4plants/nfdi4plants_ontology #endif$_DATAPLANT.

            </span>
        </p>

        <p class="c0">
            <span class="c3">
                Will your data include qualified references to other data (e.g. other data from your project, or
                datasets from previous research)?
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The references to other data will be made in the form of DOI and ontology terms.
            </span>
        </p>

        <h3><span class="c7"> 2.4. Increase data re-use </span></h3>
        <p class="c0"><span class="c3">How will you provide documentation needed to validate data analysis and
                facilitate data re-use (e.g. readme files with information on methodology, codebooks, data cleaning,
                analyses, variable definitions, units of measurement, etc.)?</span></p>

        <p class="c0">
            <span class="c1">
                The documentation will be provided in the form of ISA (Investigation Study Assay) and CWL (Common
                Workflow Language). #if$_DATAPLANT Here, $_PROJECTNAME will build on the ARC container, which
                includes all the data, metadata, and documentations. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0"><span class="c3">Will your data be made freely available in the public domain to permit the
                widest re-use possible? Will your data be licensed using standard reuse licenses, in line with the
                obligations set out in the Grant Agreement?
            </span></p>

        <p class="c0">
            <span class="c1">
                Yes, our data will be made freely available in the public domain to permit the widest re-use
                possible. Open licenses, such as Creative Commons (CC), will be used whenever possible.
            </span>
        </p>
        <p class="c0"><span class="c3">Will the data produced in the project be useable by third parties, in
                particular after the end of the project?
            </span></p>

        <p class="c0">
            <span class="c1">
                There will be no restrictions once the data is made public.
            </span>
        </p>
        <p class="c0"><span class="c3">Will the provenance of the data be thoroughly documented using the
                appropriate standards? Describe all relevant data quality assurance processes.
            </span></p>

        <p class="c0">
            <span class="c1">
                $_PROJECTNAME has the following aim: $_PROJECTAIM. Therefore, data collection #if!$_VVISUALIZATION
                and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION , integration and visualization
                #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT ARC structure are absolutely necessary,
                #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data management process is absolutely
                necessary, #endif!$_DATAPLANT because the data are used not only to understand principles, but also
                be informed about the provenance of data analysis information. Stakeholders must also be informed
                about the provenance of data. It is therefore necessary to ensure that the data are well generated
                and also well annotated with metadata using open standards, as laid out in the next section.
            </span>
        </p>

        <p class="c0"><span class="c3">Describe all relevant data quality assurance processes. Further to the FAIR
                principles, DMPs should also address research outputs other than data, and should carefully consider
                aspects related to the allocation of resources, data security and ethical aspects.
            </span></p>

        <p class="c0">
            <span class="c1">
                The data will be checked and curated by using data collection protocol, personnel training, data
                cleaning, data analysis, and quality control #if$_DATAPLANT Furthermore, data will be analyzed for
                quality control (QC) problems using automatic procedures as well as by manual curation.
                #endif$_DATAPLANT Document all data quality assurance processes, including the data collection
                protocol, data cleaning procedures, data analysis techniques, and quality control measures. This
                documentation should be kept for future reference and should be made available to stakeholders upon
                request.

            </span>
        </p>
        <h2 class="c9"><span class="c4">3&nbsp; &nbsp; Other research outputs</span></h2>
        <p class="c0"><span class="c3">In addition to the management of data, beneficiaries should also consider and
                plan for the management of other research outputs that may be generated or re-used throughout their
                projects. Such outputs can be either digital (e.g. software, workflows, protocols, models, etc.) or
                physical (e.g. new materials, antibodies, reagents, samples, etc.).

            </span></p>
        <p class="c0">
            <span class="c1">
                In the current data management plan, any digital output including but not limited to software,
                workflows, protocols, models, documents, templates, notebooks are all treated as data. Therefore,
                all aforementioned digital objects are already described in detail. For the non-digital objects, the
                data management plan will be closely connected to the digitalisation of the physical objects. #if$_DATAPLANT $_PROJECTNAME will build a workflow which connects the ARC with an electronic lab
                notebook in order to also manage the physical objects. #endif$_DATAPLANT

            </span>
        </p>

        <p class="c0"><span class="c3">Beneficiaries should consider which of the questions pertaining to FAIR data
                above, can apply to the management of other research outputs, and should strive to provide
                sufficient detail on how their research outputs will be managed and shared, or made available for
                re-use, in line with the FAIR principles.

            </span></p>
        <p class="c0">
            <span class="c1">
                Open licenses, such as Creative Commons CC, will be used whenever possible even on the other digital
                objects.

            </span>
        </p>





        <h2 class="c9"><span class="c4">4.&nbsp; &nbsp; Allocation of resources</span></h2>
        <p class="c10"><span class="c3">What will the costs be for making data or other research outputs FAIR in
                your project (e.g. direct and indirect costs related to storage, archiving, re-use, security,
                etc.)?</span></p>
        <p class="c0"><span class="c1">$_PROJECTNAME will bear the costs of data curation, #if$_DATAPLANT ARC
                consistency checks, #endif$_DATAPLANT and data maintenance/security before transfer to public
                repositories. Subsequent costs are then borne by the operators of these repositories.</span></p>
        <p class="c0">
            <span class="c1">
                Additionally, costs for after publication storage are incurred by end-point repositories (e.g. ENA)
                but not charged against $_PROJECTNAME or its members but by the operation budget of these
                repositories.
            </span>
        </p>
        <p class="c10">
            <span class="c3">How will these be covered? Note that costs related to research data/output management
                are eligible as part of the Horizon Europe grant (if compliant with the Grant Agreement
                conditions)</span>
        </p>
        <p class="c0"><span class="c1">The cost born by $_PROJECTNAME are covered by the project funding.
                Pre-existing structures #if$_DATAPLANT such as structures, tools, and knowledge laid down in the
                DataPLANT consortium #endif$_DATAPLANT will also be used.</span></p>
        <p class="c10"><span class="c3">Who will be responsible for data management in your project?</span></p>
        <p class="c0"><span class="c1">The responsible person will be $_DATAOFFICER of $_PROJECTNAME.</span></p>
        <p class="c10"><span class="c3">How will long term preservation be ensured? Discuss the necessary resources
                to accomplish this (costs and potential value, who decides and how, what data will be kept and for
                how long)?</span></p>
        <p class="c0">
            <span class="c1">
                The data officer #if$_PARTNERS or $_PARTNERS #endif$_PARTNERS will ultimately decide on the
                strategy to preserve data that are not submitted to end-point subject area repositories
                #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT when the
                project ends. This will be in line with EU guidlines, institute policies, and data sharing based on
                EU and international standards.
            </span>
        </p>
        <h2 class="c9" id="h.of351jlpnl2l"><span class="c4">5.&nbsp; &nbsp; Data security</span></h2>
        <p class="c10"><span class="c3">What provisions are or will be in place for data security (including data
                recovery as well as secure storage/archiving and transfer of sensitive data)?</span></p>
        <p class="c20">
            <span class="c1">
                Online platforms will be protected by vulnerability scanning, two-factor authorization and daily
                automatic backups allowing immediate recovery. All partners holding confidential project data will use secure platforms with automatic backups and offsite secure copies.
                #if$_DATAPLANT As ARCs are stored in the PLANTDataHUB of DataPLANT, data security will be imposed.
                This comprises secure storage, and the use of password and usernames is generally transferred via
                separate safe media. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c10"><span class="c3">Will the data be safely stored in trusted repositories for long term
                preservation and curation?</span></p>
    
                <p class="c0">
                    <span class="c1 list-to-remove-comma">
                        As mentioned above, data will be archived in
                        international discipline related repositories which use specialized technologies: 
                        #if$_GENETIC|$_GENOMIC #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_TRANSCRIPTOMIC|$_GENETIC 
                        #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_ARRAYEXPRESS
                        EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC|$_GENOMIC #if$_IMAGE #if$_BIOIMAGE
                        EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                        #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-Metabolights, #endif$_METABOLIGHTS
                        #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct, #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                        EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                        Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                        #endif$_EDAL #endif$_PHENOTYPIC.
                        #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                        well. #endif$_OTHEREP
                    </span>
                </p>
        <h2 class="c9" id="h.ozaz4qpzixaa"><span class="c4">6.&nbsp; &nbsp; Ethics</span></h2>
        <p class="c10">
            <span class="c3">
                Are there, or could there be, any ethics or legal issues that can have an impact on data sharing?
                These can also be discussed in the context of the ethics review. If relevant, include references to
                ethics deliverables and ethics chapter in the Description of the Action (DoA).
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                At the moment, we do not anticipate ethical or legal issues with data sharing. In terms of ethics,
                since this is plant data, there is no need for an ethics committee, however, diligence for plant
                resource benefit
                sharing is considered. #issuewarning Please ensure that you complete any necessary due diligence. Currently, we are awaiting clarification on whether the Nagoya Protocol ( see Nagoya Protocol) will encompass sequence information. Regardless, if you use material from a country other than your own (or that of your partner), and you conduct physical or biochemical characterization (e.g., metabolites, proteome, RNASeq, etc.), this may constitute an action relevant under the Nagoya Protocol. Exceptions might include materials from countries such as the U.S. (non-partner), Ireland (has not signedstill contact them), etc., though other laws could apply.
                #endissuewarning
            </span>
        </p>
        <p class="c10"><span class="c3">Will informed consent for data sharing and long term preservation be
                included in questionnaires dealing with personal data?</span></p>
        <p class="c0">
            <span class="c1">
                The only personal data that will potentially be stored is the submitter name and affiliation in the
                metadata for data. In addition, personal data will be collected for dissemination and communication
                activities using specific methods and procedures developed by $_PROJECTNAME partners to adhere to
                data protection. #issuewarning You need to inform and better get WRITTEN consent that you store
                emails and
                names or even pseudonyms such as twitter handles, we are very sorry about these issues we didnt
                invent them #endissuewarning
            </span>
        </p>
        <h2 class="c9" id="h.slb0g3rbvjfl"><span class="c4">7.&nbsp; &nbsp; Other issues</span></h2>
        <p class="c0"><span class="c3">Do you, or will you, make use of other national/funder/sectorial/departmental
                procedures for data management? If yes, which ones (please list and briefly describe them)?</span>
        </p>
        <p class="c0"><span class="c1 list-to-remove-comma">Yes, $_PROJECTNAME will use common Research Data Management (RDM) infrastructures #if$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC developed by #endif$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC #if$_DATAPLANT|$_NFDI the NFDI of Germany, #endif$_DATAPLANT|$_NFDI #if$_FRENCH INRAe from France, #endif$_FRENCH #if$_EOSC EOSC (European Open Science Cloud), #endif$_EOSC .
                    </span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c6"><span class="c1"></span></p>
        <h1 class="c13" ><span class="c2">3&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15">
            </span><span class="c12 c2">Annexes</span></h1>
        <p class="c0"><span class="c1"></span></p>
        <h2 class="c9" ><span class="c11 c18">3.1&nbsp;&nbsp;&nbsp;&nbsp;</span><span
                class="c15"> </span><span class="c4">Abbreviations</span></h2>
        <p class="c0"><span class="c1"></span></p>
        #if$_DATAPLANT  <span class="c11">  ARC </span> <span class="c1"> Annotated Research Context</span> <br>  #endif$_DATAPLANT 

        <span class="c11"> CC</span><span class="c1"> Creative Commons</span> <br>
        <span class="c11"> CC CEL</span><span class="c1"> Creative Commons Rights Expressio Language</span> <br>
        #if$_PROTEOMIC #if$_CHEBI <span class="c11"> ChEBI</span><span class="c1"> Chemical Entities of Biological Interest </span> <br>  #endif$_CHEBI #endif$_PROTEOMIC      
        <span class="c11"> DDBJ</span><span class="c1"> DNA Data Bank of Japan</span> <br>
        <span class="c11"> DMP</span><span class="c1"> Data Management Plan</span> <br>
        <span class="c11"> DoA</span><span class="c1"> Description of Action</span> <br>
        <span class="c11"> DOI</span><span class="c1"> Digital Object Identifier</span> <br>
        <span class="c11"> EBI</span><span class="c1"> European Bioinformatics Institute</span> <br>
        #if$_PHENOTYPIC #if$_EDAL  <span class="c11"> e!DAL-PGP</span><span class="c1"> Plant Genomics & Phenomics Research Data Repository</span> <br>  #endif$_EDAL   #endif$_PHENOTYPIC  
        #if$_GENETIC  #if$_ENA   <span class="c11">  ENA </span> <span class="c1"> European Nucleotide Archive </span> <br>  #endif$_ENA        #endif$_GENETIC   
        <span class="c11"> EU </span><span class="c1">European Union</span> <br>
        <span class="c11"> FAIR </span><span class="c1">Findable Accessible Interoperable Reproducible</span> <br>
        <span class="c11"> GDPR</span><span class="c1"> General data protection regulation (of the EU)</span> <br>
        #if$_TRANSCRIPTOMIC|$_GENETIC #if$_GEO  <span class="c11"> GEO</span><span class="c1"> Gene Expression Omnibus </span>  <br>  #endif$_GEO   #endif$_TRANSCRIPTOMIC|$_GENETIC   
        #if$_IMAGE #if$_IDR  <span class="c11"> IDR</span><span class="c1"> Image Data Resource </span>  <br>  #endif$_IDR  #endif$_IMAGE 
        <span class="c11"> IP</span><span class="c1"> Intellectual Property</span> <br>
        <span class="c11"> ISO</span><span class="c1"> International Organization for Standardization</span>  <br>
        #if$_TRANSCRIPTOMIC #if$_MIAME <span class="c11"> MIAME</span><span class="c1"> Minimum Information About a Microarray Experiment</span> <br>  #endif$_MIAME  #endif$_TRANSCRIPTOMIC  
        #if$_METABOLOMIC #if$_MMIAMET <span class="c11"> MIAMET</span><span class="c1"> Minimum Information About a METabolomics experiment</span> <br>  #endif$_MMIAMET   #endif$_METABOLOMIC  
        #if$_PROTEOMIC #if$_MIAPE <span class="c11"> MIAPE</span><span class="c1"> Minimum Information About a Proteomics Experiment</span> <br>  #endif$_MIAPE  #endif$_PROTEOMIC  
        #if$_PHENOTYPIC #if$_MIAPPE <span class="c11"> MIAPPE</span><span class="c1"> Minimum Information about Plant Phenotyping Experiment</span> <br>  #endif$_MIAPPE  #endif$_PHENOTYPIC 
        #if$_GENOMIC|$_GENETIC #if$_MIGSEU <span class="c11"> MigsEu </span><span class="c1"> Minimum Information about a Genome Sequence: Eucaryote </span> <br>  #endif$_MIGSEU  
        #if$_MIGSORG <span class="c11"> MigsOrg </span><span class="c1"> Minimum Information about a Genome Sequence: Organelle </span> <br>  #endif$_MIGSORG  
        #if$_MIMAG <span class="c11"> MIMAG </span><span class="c1"> Minimum Information about Metagenome-Assembled Genome </span> <br>  #endif$_MIMAG  
        #if$_MIMARKSSPECIMEN <span class="c11"> MIMARKSSpecimen</span><span class="c1"> Minimal Information about a Marker Specimen: Specimen </span> <br>  #endif$_MIMARKSSPECIMEN 
        #if$_MIMARKSSURVEY  <span class="c11">  MIMARKSSurvey </span><span class="c1"> Minimal Information about a Marker Specimen: Survey </span> <br>  #endif$_MIMARKSSURVEY    #endif$_GENOMIC|$_GENETIC  
        #if$_PROTEOMIC #if$_MIMIX <span class="c11"> MIMIX</span><span class="c1"> The Minimum Information required for reporting a Molecular Interaction Experiment </span> <br>  #endif$_MIMIX   #endif$_PROTEOMIC 
        #if$_GENOMIC|$_GENETIC #if$_MIMS  <span class="c11">  MIMS</span><span class="c1"> Molecular Interactions </span> <br>  #endif$_MIMS    #endif$_GENOMIC|$_GENETIC 
        #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_MINSEQE <span class="c11"> MinSEQe</span><span class="c1"> Minimum Information about a high-throughput Sequencing Experiment</span> <br>  #endif$_MINSEQE #endif$_TRANSCRIPTOMIC|$_GENOMIC 
        #if$_GENOMIC|$_GENETIC #if$_MISAG <span class="c11"> MISAG</span><span class="c1"> Minimum Information about a Single Amplified Genome </span> <br>  #endif$_MISAG
        #if$_MIXS <span class="c11"> MIxS</span><span class="c1"> Minimum Information about any (X) Sequence </span> <br>  #endif$_MIXS #endif$_GENOMIC|$_GENETIC 
        <span class="c11"> NCBI</span><span class="c1"> National Center for Biotechnology Information</span> <br>
        <span class="c11"> NFDI</span><span class="c1"> National Research Data Infrastructure (of Germany)</span> <br>
        <span class="c11"> NGS</span><span class="c1"> Next Generation Sequencing</span> <br>
        #if$_PROTEOMIC #if$_PRIDE  <span class="c11"> PRIDE</span><span class="c1"> PRoteomics IDEntifications Database</span> <br>  #endif$_PRIDE
        #if$_PDB  <span class="c11"> PDB</span><span class="c1"> Protein Data Bank </span> <br>  #endif$_PDB #endif$_PROTEOMIC
        <span class="c11"> RDM</span><span class="c1"> Research Data Management</span> <br>
        #if$_IMAGE  #if$_REMBI  <span class="c11"> REMBI</span><span class="c1"> Recommended Metadata for Biological Images </span> <br>  #endif$_REMBI #endif$_IMAGE 

        #if$_RNASEQ  <span class="c11"> RNASeq</span><span class="c1"> RNA Sequencing</span> <br>  #endif$_RNASEQ 
        <span class="c11"> SOP</span><span class="c1"> Standard Operating Procedures</span>  <br>
        #if$_GENETIC #if$_SRA  <span class="c11"> SRA</span><span class="c1"> Sequence Read Archive</span> <br>  #endif$_SRA #endif$_GENETIC
        #if$_DATAPLANT  <span class="c11">  SWATE</span><span class="c1"> Swate Workflow Annotation Tool for Excel</span> <br>  #endif$_DATAPLANT 
        <span class="c11"> ONP</span><span class="c1"> Oxford Nanopore</span>  <br>
        <span class="c11"> qRT</span> <span class="c11"> PCR</span><span class="c1"> quantitative real time polymerase chain reaction</span>  <br>
        <span class="c11"> WP</span><span class="c1"> Work Package</span>  <br>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c22"><span class="c1"></span></p>

    </div>
</div>`,

"dfg-dmp" : `<div id="dfg-dmp" class="">
    <div id="dfg-dmp1" class="content-page">
        <br />
        <center>
            <div class="p-2" id="vis-parent" style="margin: auto; ">
                <div id="vis" style=" width:100%; max-height:600px; aspect-ratio : 1 / 1;"></div>
            </div>

            <div id="text_start" class="c2 text-center">Data Management Plan of the DFG Project $_PROJECTNAME</div>
        </center>
        <h2>
            <span class="c2">1.&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c2">Data description</span>
        </h2>
        <h3>
            <span class="c3">1.1&nbsp;&nbsp;&nbsp;&nbsp;Introduction</span>
        </h3>

        #if$_EU&nbsp;
        <p class="c0"><span class="c1">$_PROJECTNAME is part of the Open Data Initiative (ODI) of the EU.
                #endif$_EU To best profit from open data, it is necessary not only to store data but to make data
                Findable, Accessible, Interoperable and Reusable (FAIR). #if$_PROTECT Open and FAIR data, however,
                considers the need
                to protect individual data sets. #endif$_PROTECT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The aim of this document is to provide guidelines on principles guiding the data management in $_PROJECTNAME. A Data Management Plan (DMP) will generated by using the responses to the EU questionnaire on DMP, which defines the data stored in $_PROJECTNAME.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The detailed DMP instructs how data will be handled during and after the project. The $_PROJECTNAME DMP
                is modified according to the DFG data management checklist. #if$_UPDATE It will be updated/its
                validity
                checked during $_PROJECTNAME several times. At the very least, this will happen at month
                $_UPDATEMONTH. #endif$_UPDATE
            </span>
        </p>

        <p class="c0">
            <span class="c3">1.2&nbsp;&nbsp;&nbsp;&nbsp;How does your project generate new data?</span>
        </p>
        <p class="c0">
            <span class="c1"> Data of different types or of different domains will be generated differently. For
                example:</span>
        </p>
        <ul style="list-style-type:disc;">
            #if$_GENETIC&nbsp; <li>
                    <span class="c1"> Genetic data will be generated targeting crosses and breeding experiments, and
                        will include recombination frequencies and crossover event that position genetic markers and
                        quantitative trait loci that can be associated with physical genomic markers/variants.
                    </span>
            </li> #endif$_GENETIC #if$_GENOMIC&nbsp; 
            <li>
                
                    <span class="c1"> Genomic data will be created from sequencing data, which will be processed to
                        identify genes, regulatory elements, transposable elements, and physical markers such as
                        SNPs, microsatellites and structural variants. </span>

            </li>
            #endif$_GENOMIC #if$_CLONED-DNA&nbsp; 
            <li>

                    <span class="c1"> The origin and assembly of cloned DNA will include (a) source of original
                        vector sequence with Add gene reference where available, and source of insert DNA (e.g.,
                        amplification by PCR from a given sample, or obtained from existing library), (b) cloning
                        strategy (e.g., restriction endonuclease digests/ligation, PCR, TOPO cloning, Gibson
                        assembly, LR recombination), and (c) verified DNA data sequence of final recombinant vector.
                    </span>

            </li>
            #endif$_CLONED-DNA  #if$_TRANSCRIPTOMIC&nbsp; 
            <li>

                    <span class="c1"> Methods of transcriptomics data collection will be selected from microarrays,
                        quantitative PCR, Northern blotting, RNA immunoprecipitation, fluorescence in situ
                        hybridization. RNA-Seq data will be collected in seperate methods.</span>

            </li>
            #endif$_TRANSCRIPTOMIC&nbsp;  #if$_RNASEQ&nbsp; 
            <li>

                    <span class="c1"> RNA sequencing will be generated using short-read or long-read plantforms,
                        either in house or outsourced to academic facilities or commercial services, and the raw
                        data will be processed using estabilished bioinformatics piplines. </span>
    
            </li>
            #endif$_RNASEQ&nbsp;  #if$_METABOLOMIC&nbsp; 
            <li>

                    <span class="c1"> Metabolomic data will be generated by coupled chromatography and mass
                        spectrometry using targeted or untargeted approaches.</span>

            </li>
            #endif$_METABOLOMIC  #if$_PROTEOMIC&nbsp; 
            <li>
        
                    <span class="c1"> Proteomic data will be generated using coupled chromatography and mass
                        spectrometry for the analysis of protein abundance and protein identification, as well as
                        additional techniques for structural analysis, the identification of post-translational
                        modifications and the characterization of protein interactions.</span>

            </li>
            #endif$_PROTEOMIC  #if$_PHENOTYPIC&nbsp; 
            <li>

                    <span class="c1"> Phenotypic data will be generated using phenotyping platforms and
                        corresponding ontologies, including number/size of organs such as leaves, flowers, buds
                        etc., size of whole plant, stem/root architecture (number of lateral branches/roots etc),
                        organ structures/morphologies, quantitative metrics such as color, turgor, health/nutrition
                        indicators, among others. </span>

            </li>
            #endif$_PHENOTYPIC  #if$_TARGETED&nbsp; 
            <li>

                    <span class="c1"> Targeted assays data (e.g. glucose and fructose concentrations or
                        production/ultilization rates) will be generated using specific equipment and methods that
                        are fully documented in the laboratory notebook. </span>
            
            </li>
            #endif$_TARGETED #if$_IMAGE&nbsp; 
            <li>
        
                    <span class="c1"> Image data will be generated by equipment such as cameras, scanners, and
                        microscopes combined with software. Original images which contain metadata such as exif
                        photo information will be archived.</span>
            
            </li>
            #endif$_IMAGE #if$_MODELS&nbsp; 
            <li>
            
                    <span class="c1"> Model data will be generated by using software simulations. The complete
                        workflow, which includes the environment, runtime, parameters, and results, will be
                        documented and archived. </span>
            
            </li>
            #endif$_MODELS #if$_CODE&nbsp; 
            <li>
                
                    <span class="c1"> Computer code will be produced by programmers. </span>
            
            </li>
            #endif$_CODE  #if$_EXCEL&nbsp; 
            <li>
                
                    <span class="c1"> Excel data will be generated by data analysts by using MS Office or
                        open-source software. </span>
            
            </li>
            #endif$_EXCEL 

            

            
        </ul>

        <p class="c0">
            <span class="c1">
                $_PROJECTNAME has the following aim: $_PROJECTAIM. Therefore, data collection #if!$_VVISUALIZATION
                and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION , integration and visualization
                #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT Annotated Research Context (ARC) structure are absolutely necessary,
                #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data management process is absolutely
                necessary, #endif!$_DATAPLANT because the data are used not only to understand principles, but also
                be informed about the provenance of data analysis information. Stakeholders must also be informed
                about the provenance of data. It is therefore necessary to ensure that the data are well generated
                and also well annotated with metadata using open standards, as laid out in the next section.
            </span>
        </p>

        <p class="c0">
            <span class="c1">Public data will be extracted as described in paragraph 1.3. For $_PROJECTNAME,
                specific data sets will be generated by the consortium partners.</span>
        </p>

        <p class="c0">
            <span class="c3">1.3&nbsp;&nbsp;&nbsp;&nbsp;Is existing data reused?</span>
        </p>
        <p class="c0">
            <span class="c1">
                The project builds on existing data sets and relies on them. #if$_RNASEQ For instance, without a
                proper genomic reference it is very difficult to analyze NGS data sets. #endif$_RNASEQ It is also
                important to include existing data sets on the expression and metabolic behaviour of $_STUDYOBJECT,
                but of course, also on existing characterization and the background knowledge. #if$_PARTNERS of the
                partners: $_PARTNERS #endif$_PARTNERS Genomic references can simply be gathered from reference databases for
                genomes/sequences, like the National Center for Biotechnology Information: NCBI (US); European
                Bioinformatics Institute: EBI (EU); DNA Data Bank of Japan: DDBJ (JP). Furthermore, prior
                'unstructured' data in the form of publications and data contained therein will be used for decision
                making.
            </span>
        </p>

        <p class="c0">
            <span class="c3">1.4&nbsp;&nbsp;&nbsp;&nbsp;Which data types (in terms of data formats like image data,
                text data or measurement data) arise in your project and in what way are they further
                processed?</span>
        </p>
        <p class="c0">
            <span class="c1">
                We foresee that the following data about $_STUDYOBJECT will be collected and generated at the very
                least: $_PROJECTNAME will collect and/or generate the following types of raw data: 
                #if$_GENETIC genetic data, #endif$_GENETIC  
                #if$_GENOMIC genomic data, #endif$_GENOMIC 
                #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA 
                #if$_TRANSCRIPTOMIC transcriptomic data, #endif$_TRANSCRIPTOMIC  
                #if$_RNASEQ RNAseq data, #endif$_RNASEQ 
                #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC  
                #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC 
                #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC  
                #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED  
                #if$_IMAGE image datasets, #endif$_IMAGE  
                #if$_MODELS modelling data, #endif$_MODELS 
                #if$_CODE computational code, #endif$_CODE  
                #if$_EXCEL excel files, #endif$_EXCEL and result data. Furthermore, data derived from the original
                raw data sets will also be collected. This is important, as different analytical pipelines
                might yield different results or include
            </span>
            <span class="c5">ad-hoc</span>
            <span class="c1">
                data analysis parts #if$_DATAPLANT and these pipelines will be tracked in the DataPLANT
                ARC #endif$_DATAPLANT. Therefore, specific care will be taken, to document and archive these
                resources (including the
                analytic pipelines) as well #if$_DATAPLANT relying on the vast expertise in the DataPLANT consortium
                #endif$_DATAPLANT.
            </span>
        </p>

        <span class="c3">1.5&nbsp;&nbsp;&nbsp;&nbsp;To what extent do these arise or what is the anticipated data
            volume?</span>

        <p class="c0">
            <span class="c1">We expect to generate raw data in the range of $_RAWDATA GB of data. The size of the
                derived data will be about $_DERIVEDDATA GB.</span>
        </p>

        <h2 class="c13" id="documentation">
            <span class="c2">2.&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15"></span><span
                class="c2 c12">Documentation and data quality</span>
        </h2>
        <h3 class="c3">
            <span class="c0">2.1.&nbsp;&nbsp;&nbsp;&nbsp;What approaches are being taken to describe the data in a
                comprehensible manner (such as the use of available metadata, documentation standards or
                ontologies)? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                All datasets will be associated with unique identifiers and will be annotated with metadata. We will
                use Investigation, Study, Assay (ISA) specification for metadata creation. $_PROJECTNAME will rely
                on community standards plus additional recommendations applicable in the plant science, such as the
                <span class="list-to-remove-comma">
                    The following metadata/ minimum informatin standards will be used to collect metadata:
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS (Minimum Information about any (X) Sequence), #endif$_MIXS
                    #if$_MIGSEU MigsEu (Minimum Information about a Genome Sequence: Eucaryote), #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg (Minimum Information about a Genome Sequence: Organelle), #endif$_MIGSORG
                    #if$_MIMS MIMS (Minimum Information about Metagenome or Environmental), #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen (Minimal Information about a Marker Specimen:
                    Specimen), #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey (Minimal Information about a Marker Specimen:
                    Survey), #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG (Minimum Information about a Single Amplified Genome), #endif$_MISAG
                    #if$_MIMAG MIMAG (Minimum Information about Metagenome-Assembled Genome), #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_TRANSCRIPTOMIC
                    #if$_MINSEQE MINSEQE (Minimum Information about a high-throughput SEQuencing
                    Experiment), #endif$_MINSEQE #endif$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME (Minimum Information About a Microarray
                    Experiment), #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI (Recommended Metadata for Biological Images), #endif$_REMBI
                    #endif$_IMAGE
                    #if$_RNASEQ|$_GENOMIC 
                    #if$_MINSEQE
                        MinSEQe (Minimum Information about a high-throughput Sequencing Experiment),
                    #endif$_MINSEQE 
                    #endif$_RNASEQ|$_GENOMIC
                    #if$_METABOLOMIC
                    #if$_MMIAMET
                    MIAMET (Minimum Information About a METabolomics experiment),
                    #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE (Minimum Information About a Proteomics Experiment), #endif$_MIAPE
                    #if$_MIMIX MIMix (The Minimum Information required for reporting a Molecular Interaction Experiment), #endif$_MIMIX
                    #endif$_PROTEOMIC  
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE (Minimum Information about Plant Phenotyping Experiment) #endif$_MIAPPE  #endif$_PHENOTYPIC .
                    </span>
            </span>
            <span class="c1">#if$_METABOLOMIC #if$_METABOLIGHTS The Metabolights submission compliant standards are used for metabolomic data.  #issuewarning Some metabolomics partners considers Metabolights
                not an accepted standard. #endissuewarning #endif$_METABOLIGHTS #endif$_METABOLOMIC These specific standard unlike cross-domain minimal sets such as the Dublin core, which
                mostly define the submitter and the general type of data, allow reusability by other researchers by
                defining properties of the plant (see the preceding section). However, $_PROJECTNAME also implement minimal cross-domain annotations #if$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 such as #endif$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 <span class="list-to-remove-comma"> #if$_DUBLINCORE Dublin Core, #endif$_DUBLINCORE #if$_DARWINCORE Darwin Core, #endif$_DARWINCORE #if$_SCHEMAORG Schema.org, #endif$_SCHEMAORG #if$_BIOSCHEMAS BioSchemas, #endif$_BIOSCHEMAS #if$_MARC21 MARC 21 #endif$_MARC21 . </span> #if$_DATAPLANT The core integration with DataPLANT will also allow
                individual releases to be tagged with a Digital Object Identifier (DOI).
                #endif$_DATAPLANT #if$_OTHERSTANDARDS Other standards such as $_OTHERSTANDARDINPUT are also adhered
                to. #endif$_OTHERSTANDARDS
            </span>
        </p>

        <p class="c0">
            <span class="c1">
                Open ontologies will be used where they are mature. As stated above, some ontologies and controlled
                vocabularies might need to be extended. #if$_DATAPLANT Here, $_PROJECTNAME will build on the
                advanced ontologies integrated in DataPLANT. #endif$_DATAPLANT
                Keywords about the experiment and the general consortium will be included, as well as an abstract
                about the data, where useful. In addition, certain keywords can be auto-generated from dense
                metadata and
                its underlying ontologies. #if$_DATAPLANT Here, DataPLANT strives to complement these with
                standardized DataPLANT ontologies that are supplemented where the ontology does not yet include the
                variables.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                In fact, open biomedical ontologies will be used where they are mature. As stated in the previous
                question, sometimes ontologies and controlled vocabularies might have to be extended. #if$_DATAPLANT
                Here, the
                $_PROJECTNAME will build on the advanced ontologies integrated in DataPLANT. #endif$_DATAPLANT
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">2.2&nbsp;&nbsp;&nbsp;&nbsp;What measures are being adopted to ensure high data
                quality?</span>
        </h3>

        <p class="c0">
            <span class="c1">
                $_PROJECTNAME aims at the following aim: $_PROJECTAIM. Therefore, data
                collection#if!$_VVISUALIZATION and integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION ,
                integration and visualization #endif$_VVISUALIZATION #if$_DATAPLANT using the DataPLANT ARC
                structure are absolutely necessary, #endif$_DATAPLANT #if!$_DATAPLANT through a standardized data
                management process is absolutely necessary, #endif!$_DATAPLANT because the data are used not only to
                understand principles, but also be informed about the provenance of data analysis information.
                Stakeholders must also be informed about the provenance of data. It is therefore necessary to ensure
                that the data are well generated and also well annotated with metadata using open standards.

                Data variables will be allocated standard names. For example, genes, proteins and metabolites will
                be named according to approved nomenclature and conventions. These will also be linked to functional
                ontologies where possible. Datasets will also be named I a meaningful way to ensure readability by
                humans. Plant names will include traditional names, binomials, and all
                strain/cultivar/subspecies/variety identifiers.


            </span>
        </p>

        <p class="c0">
            <span class="c1">
                To maintain data integrity and to be able to re-analyze data, data sets will get version numbers
                where this is useful (e.g. raw data must not be changed and will not get a version number and is
                considered
                immutable). #if$_DATAPLANT this is automatically supported by the ARC Git DataPLANT infrastructure.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                As mentioned above, <span class="c1 list-to-remove-comma">we will use Investigation, Study, Assay (ISA) specification for metadata creation.
                    The following metadata standards will also be used:
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE, #endif$_MIAPPE #endif$_PHENOTYPIC
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS, #endif$_MIXS
                    #if$_MIGSEU MigsEu, #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg, #endif$_MIGSORG
                    #if$_MIMS MIMS, #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen, #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey, #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG, #endif$_MISAG
                    #if$_MIMAG MIMAG, #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_GENOMIC|$_TRANSCRIPTOMIC #if$_MINSEQE MINSEQE, #endif$_MINSEQE #endif$_GENOMIC|$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME, #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI, #endif$_REMBI
                    #endif$_IMAGE
                    #if$_METABOLOMIC
                    #if$_MMIAMET MIAMET, #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE, #endif$_MIAPE
                    #if$_MIMIX MIMix, #endif$_MIMIX
                    #endif$_PROTEOMIC.
                    
                </span> The metadata standards will
                thus allow the integration of data across projects and safeguards that reuse established and tested
                protocols.
                Additionally, we will use ontology terms to enrich the data sets relying on free and open
                ontologies. In addition, additional ontology terms might be created and be canonized during the
                $_PROJECTNAME.
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">2.3&nbsp;&nbsp;&nbsp;&nbsp;Are quality controls in place and if so, how do they
                operate? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                The data will be checked and curated through the project period. #if$_DATAPLANT Furthermore, data
                will be analyzed for quality control (QC) problems using automatic procedures as well as by manual
                curation.
                #endif$_DATAPLANT Phd students and lab professionals will be responsible for the first-hand quality
                control. Afterwards, the data will be checked and annotated by $_DATAOFFICER. #if$_RNASEQ|$_GENOMIC
                FastQC will be conducted on the base-calling. #endif$_RNASEQ|$_GENOMIC Before publication, the data
                will be controlled again.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">2.4&nbsp;&nbsp;&nbsp;&nbsp;Which digital methods and tools (e.g. software) are required
                to use the data?</span>
        </h3>
            <p class="c0"><span class="c1 list-to-remove-comma">Yes, $_PROJECTNAME will use common Research Data Management (RDM) infrastructures #if$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC developed by #endif$_DATAPLANT|$_NFDI|$_FRENCH|$_EOSC #if$_DATAPLANT|$_NFDI the NFDI of Germany, #endif$_DATAPLANT|$_NFDI #if$_FRENCH INRAe from France, #endif$_FRENCH #if$_EOSC EOSC (European Open Science Cloud), #endif$_EOSC .
                    </span></p>
        <p class="c0">
            <span class="c1">#if$_PROPRIETARY $_PROJECTNAME relies on the tool(s) $_PROPRIETARY.
                #endif$_PROPRIETARY</span>
        </p>
        <p class="c0">
            <span class="c1">
                #if!$_PROPRIETARY No specialized software will be needed to access the data, usually just a modern
                browser. Access will be possible through web interfaces. For data processing after obtaining raw
                data,
                typical open-source software can be used. As no proprietary software is needed, no documentation
                needs to be provided. #endif!$_PROPRIETARY
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT However, DataPLANT resources are well described, and their setup is documented on
                their github project pages.
                #endif$_DATAPLANT
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT DataPLANT offers tools such as the open-source SWATE plugin for Excel, the ARC
                commander, and the DMP tool which will not necessarily make the interaction with data more
                convenient.
                #endif$_DATAPLANT
            </span>
        </p>

        <span class="c1">As stated above, here we use publicly available open-source and well-documented certified
            software, #if$_PROPRIETARY except for $_PROPRIETARY #endif$_PROPRIETARY.</span>
        <p class="c0"></p>

        <h2 class="c0">
            <span class="c2">3.&nbsp;&nbsp;&nbsp;&nbsp;Storage and technical archiving the project </span>
        </h2>

        <h3 class="c0">
            <span class="c3">3.1&nbsp;&nbsp;&nbsp;&nbsp;How is the data to be stored and archived throughout the
                project duration?</span>
        </h3>

          <p class="c0">
                        <span class="c1 list-to-remove-comma"> #if$_GENETIC|$_GENOMIC For genetic or genomic data: #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_GENETIC #if$_SRA NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                            #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #endif$_GENETIC #if$_GENETIC|$_GENOMIC . #endif$_GENETIC|$_GENOMIC </span>
                    </p>

        <p class="c0">
            <span class="c1  list-to-remove-comma">#if$_TRANSCRIPTOMIC For Transcriptomic data: #if$_SRA
                NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #if$_ARRAYEXPRESS
                EBI-ArrayExpress, #endif$_ARRAYEXPRESS . #endif$_TRANSCRIPTOMIC</span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_IMAGE For image data: #if$_BIOIMAGE EBI-BioImage
                Archive, #endif$_BIOIMAGE #if$_IDR IDR (Image Data Resource), #endif$_IDR .
                #endif$_IMAGE </span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_METABOLOMIC For metabolomic data: #if$_METABOLIGHTS
                EBI-Metabolights, #endif$_METABOLIGHTS #if$_METAWORKBENCH Metabolomics
                Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct (Molecular
                interactions), #endif$_INTACT . #endif$_METABOLOMIC </span>
        </p>
        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_PROTEOMIC For proteomics data: #if$_PRIDE
                EBI-PRIDE (PRoteomics IDEntifications Database), #endif$_PRIDE #if$_PDB PDB (Protein Data Bank), #endif$_PDB
                #if$_CHEBI Chebi (Chemical Entities of Biological Interest), #endif$_CHEBI .
                #endif$_PROTEOMIC </span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">#if$_PHENOTYPIC For phenotypic data: #if$_EDAL e!DAL-PGP (Plant
                Genomics & Phenomics Research Data Repository), #endif$_EDAL . #endif$_PHENOTYPIC
                #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                well. #endif$_OTHEREP
            </span>
        </p>
    
        <p class="c0">
            <span class="c1">Data will be made available for many years#if$_DATAPLANT and potentially indefinitely
                after the end of the project #endif$_DATAPLANT.</span>
        </p>
        <p class="c0">
            <span class="c1">In any case data submitted to international
                discipline related repositories which use specialized technologies (as detailed above) e.g. ENA
                /Pride would be subject to local data storage regulation.</span>
        </p>
        <h3 class="c0">
            <span class="c3">3.2&nbsp;&nbsp;&nbsp;&nbsp;What is in place to secure sensitive data throughout the
                project duration (access and usage rights)? </span>
        </h3>
        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT In DataPLANT, data management relies on the Annotated Research Context (ARC). It is
                password protected, so before any data can be obtained or samples generated, an authentication needs
                to
                take place. #endif$_DATAPLANT
            </span>
        </p>
        <p class="c20">
            <span class="c1">
                In case data is only shared within the consortium, if the data is not yet finished or under IP
                checks, the data is hosted internally, and the username and the password will be required (see also
                our GDPR rules).
                In the case data is made public under final EU or US repositories, completely anonymous access is
                normally allowed. this is the case for ENA as well and both are in line with GDPR requirements.
            </span>
        </p>
        <p class="c0">
            <span class="c1">There will be no restrictions once the data is made public.</span>
        </p>

        <h2 class="c0">
            <span class="c2">4.&nbsp;&nbsp;&nbsp;&nbsp;Legal obligations and conditions </span>
        </h2>

        <h3 class="c0">
            <span class="c3">4.1&nbsp;&nbsp;&nbsp;&nbsp;What are the legal specifics associated with the handling of
                research data in your project?</span>
        </h3>

        <p class="c0">
            <span class="c1">
                At the moment, we do not anticipate ethical or legal issues with data sharing. In terms of ethics,
                since this is plant data, there is no need for an ethics committee, however, diligence for plant
                resource
                benefit sharing is considered. #issuewarning Please ensure that you complete any necessary due diligence. Currently, we are awaiting clarification on whether the Nagoya Protocol ( see Nagoya Protocol) will encompass sequence information. Regardless, if you use material from a country other than your own (or that of your partner), and you conduct physical or biochemical characterization (e.g., metabolites, proteome, RNASeq, etc.), this may constitute an action relevant under the Nagoya Protocol. Exceptions might include materials from countries such as the U.S. (non-partner), Ireland (has not signedstill contact them), etc., though other laws could apply. #endissuewarning
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The only personal data that will potentially be stored is the submitter name and affiliation in the
                metadata for data. In addition, personal data will be collected for dissemination and communication
                activities using specific methods and procedures developed by $_PROJECTNAME partners to adhere to
                data protection. #issuewarning You need to inform and better get WRITTEN consent that you store
                emails and
                names or even pseudonyms such as twitter handles, we are very sorry about these issues we didnt
                invent them #endissuewarning
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">4.2&nbsp;&nbsp;&nbsp;&nbsp;Do you anticipate any implications or restrictions regarding
                subsequent publication or accessibility? </span>
        </h3>
        <p class="c20">
            <span class="c1">
                Once data is transferred to the $_PROJECTNAME platform #if$_DATAPLANT and ARCs have been generated in
                DataPLANT #endif$_DATAPLANT , data security will be imposed. This comprises secure storage, and the
                use of
                passwords and usernames is generally transferred via separate safe media.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">4.3&nbsp;&nbsp;&nbsp;&nbsp;What is in place to consider aspects of use and copyright
                law as well as ownership issues? </span>
        </h3>
        <p class="c0">
            <span class="c1">Open licenses, such as Creative Commons (CC), will be used whenever possible.</span>
        </p>
        <h3 class="c0">
            <span class="c3">4.4&nbsp;&nbsp;&nbsp;&nbsp;Are there any significant research codes or professional
                standards to be taken into account? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                Whenever possible, data will be stored in common and openly defined formats including all the
                necessary metadata to interpret and analyze data in a biological context. By default, no proprietary
                formats will
                be used; however, Microsoft Excel files (according to ISO/IEC 29500-1:2016) might be used as
                intermediates by the consortium#if$_DATAPLANT and by some ARC components in form #endif$_DATAPLANT.
                In addition,
                text files might be edited in text processor files, but will be shared as pdf.
            </span>
        </p>

        <h2 class="c0">
            <span class="c2">5.&nbsp;&nbsp;&nbsp;&nbsp;Data exchange and long-term data accessibility </span>
        </h2>

        <h3 class="c0">
            <span class="c3">5.1&nbsp;&nbsp;&nbsp;&nbsp;Which data sets are especially suitable for use in other
                contexts? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                The data will be useful for $_PROJECTNAME partners, the scientific community working on
                $_STUDYOBJECT or the general public interested in $_STUDYOBJECT. Hence, $_PROJECTNAME also strives
                to collect the data
                that has been disseminated and potentially advertise it #if$_DATAPLANT e.g. through the DataPLANT
                platform or other means #endif$_DATAPLANT , if it is not included in a publication anyway, which is
                the most
                likely form of dissemination.
            </span>
        </p>

        <h3 class="c0">
            <span class="c3">5.2&nbsp;&nbsp;&nbsp;&nbsp;Which criteria are used to select research data to make it
                available for subsequent use by others? </span>
        </h3>

        <p class="c0">
            <span class="c1">
                By default, all data sets from $_PROJECTNAME will be shared with the community and made openly
                available. This is, however, after partners have had the ability to check for IP protection
                (according to
                agreements and background rights). #if$_INDUSTRY This applies in particular to data pertaining to
                the industry. #endif$_INDUSTRY However, all partners also strive for IP protection of data sets
                which will
                be tested and due diligence will be given.
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                Note that in multi-beneficiary projects it is also possible for specific beneficiaries to keep their
                data closed if relevant provisions are made in the consortium agreement and are in line with the
                reasons for opting out.
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">5.3&nbsp;&nbsp;&nbsp;&nbsp;Are you planning to archive your data in a suitable
                infrastructure? </span>
        </h3>
        <p class="c0">
            #if$_DATAPLANT As $_PROJECTNAME is closely aligned with DataPLANT, the ARC converter and DataHUB will be
            used to find the end-point repositories and upload to the repositories automatically. #endif$_DATAPLANT
            <span class="c1"> </span>
        </p>

        <p class="c0">
            <span class="c1 list-to-remove-comma">
                As mentioned above, data will be made archived in
                international discipline related repositories which use specialized technologies: #if$_GENETIC|$_GENOMIC #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_TRANSCRIPTOMIC|$_GENETIC 
                #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_ARRAYEXPRESS
                EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC|$_GENOMIC #if$_IMAGE #if$_BIOIMAGE
                EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-Metabolights, #endif$_METABOLIGHTS
                #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct, #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                #endif$_EDAL #endif$_PHENOTYPIC.
                #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be processed there as
                well. #endif$_OTHEREP
            </span>
        </p>
        <p class="c0">
            <span class="c1">
                The submission is for free, and it is the goal (at least of ENA) to obtain as much data as possible.
                Therefore, arrangements are neither necessary nor useful. Catch-all repositories are not required.
                #if$_DATAPLANT For DataPLANT, this has been agreed upon. #endif$_DATAPLANT #issuewarning If no data
                management platform such as DataPLANT is used, then you need to find appropriate repository to store
                or archive your data after publication. #endissuewarning
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">5.4&nbsp;&nbsp;&nbsp;&nbsp;If so, how and where? Are there any retention periods?
            </span>
        </h3>
        <p class="c0">
            <span class="c1">There are no restrictions, beyond the aforementioned IP checks, which are in line with
                e.g. European open data policies.</span>
        </p>
        <p class="c0">
            <span class="c1">
                The $_PARTNERS decides on preservation of data not submitted to end-point subject area repositories
                #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT after project end. This will be in line with EU
                institute policies and data sharing based on EU and international standards.
            </span>
        </p>
        <h3 class="c0">
            <span class="c3">5.5&nbsp;&nbsp;&nbsp;&nbsp;When is the research data available for use by third
                parties? </span>
        </h3>
        <p class="c0">
            <span class="c1">
                #if$_EARLY Some raw data is made public as soon as it is collected and processed. #endif$_EARLY
                #if$_BEFOREPUBLICATION Relevant processed datasets are made public when the research findings are
                published. #endif$_BEFOREPUBLICATION #if$_ENDOFPROJECT At the end of the project, all data without
                embargo period will be published. #endif$_ENDOFPROJECT #if$_EMBARGO Data, which is subject to an
                embargo period, is not publicly accessible until the end of embargo period. #endif$_EMBARGO
                #if$_REQUEST Data is made available upon request, allowing controlled sharing while ensuring
                responsible use. #endif$_REQUEST #if$_IPISSUE IP issues will be checked before publication.
                #endif$_IPISSUE All consortium partners will be
                encouraged to make data available before publication, openly and/or under pre-publication agreements
                #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto International
                Data
                Release Workshop #endif$_GENOMIC . This will be implemented as soon as IP-related checks are
                complete.
            </span>
        </p>

        <h2 class="c0">
            <span class="c2">6.&nbsp;&nbsp;&nbsp;&nbsp;Responsibilities and resources </span>
        </h2>

        <h3 class="c0">
            <span class="c3">6.1&nbsp;&nbsp;&nbsp;&nbsp;Who is responsible for adequate handling of the research
                data (description of roles and responsibilities within the project)? </span>
        </h3>
        <p class="c0">
            <span class="c1">The responsible will be $_DATAOFFICER as data Officer.</span>
            <span class="c1">The data responsible(s) (data officer#if$_PARTNERS or $_PARTNERS #endif$_PARTNERS)
                decides on the preservation of data not submitted to end-point subject area repositories
                #if$_DATAPLANT or ARCs in DataPLANT #endif$_DATAPLANT after the
                project end. This will be in line with EU institute policies, and data sharing based on EU and
                international standards.</span>
        </p>
        <h3 class="c0">
            <span class="c3">6.2&nbsp;&nbsp;&nbsp;&nbsp;Which resources (costs; time or other) are required to
                implement adequate handling of research data within the project? </span>
        </h3>
        <p class="c0">
            <span class="c1">The costs comprise data curation, #if$_DATAPLANT ARC consistency checks,
                #endif$_DATAPLANT and maintenance on $_PROJECTNAMEs side.</span>
        </p>
        <p class="c0">
            <span class="c1">
                Additionally, last-level costs for storage are incurred by end-point repositories (e.g. ENA) but not
                charged against $_PROJECTNAME or its members but by the operation budget of these repositories.
            </span>
        </p>
        <p class="c0">
            <span class="c1">A large part of the cost is covered by $_PROJECTNAME #if$_DATAPLANT and the structures,
                tools and knowledge laid down in the DataPLANT consortium. #endif$_DATAPLANT</span>
        </p>
        <h3 class="c0">
            <span class="c3">6.3&nbsp;&nbsp;&nbsp;&nbsp;Who is responsible for curating the data once the project
                has ended? </span>
        </h3>
        <p class="c0">
            <span class="c1">As applicable, $_DATAOFFICER, who is responsible for ongoing data maintenance will also
                take care of it after the finish of $_PROJECTNAME. #if$_DATAPLANT DataPLANT as external data
                archives may provide such services in some cases. #endif$_DATAPLANT</span>
        </p>

        <p class="c0">
            <span class="c1"></span>
        </p>
        <p class="c6">
            <span class="c1"></span>
        </p>
        <h1 class="c13" ><span class="c2">7&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c15">
        </span><span class="c12 c2">Annexes</span></h1>
    <p class="c0"><span class="c1"></span></p>
    <h2 class="c9" ><span class="c11 c18">7.1&nbsp;&nbsp;&nbsp;&nbsp;</span><span
            class="c15"> </span><span class="c4">Abbreviations</span></h2>
    <p class="c0"><span class="c1"></span></p>
    #if$_DATAPLANT  <span class="c11">  ARC </span> <span class="c1"> Annotated Research Context</span> <br>  #endif$_DATAPLANT 

    <span class="c11"> CC</span><span class="c1"> Creative Commons</span> <br>
    <span class="c11"> CC CEL</span><span class="c1"> Creative Commons Rights Expressio Language</span> <br>
    #if$_PROTEOMIC #if$_CHEBI <span class="c11"> ChEBI</span><span class="c1"> Chemical Entities of Biological Interest </span> <br>  #endif$_CHEBI #endif$_PROTEOMIC      
    <span class="c11"> DDBJ</span><span class="c1"> DNA Data Bank of Japan</span> <br>
    <span class="c11"> DMP</span><span class="c1"> Data Management Plan</span> <br>
    <span class="c11"> DoA</span><span class="c1"> Description of Action</span> <br>
    <span class="c11"> DOI</span><span class="c1"> Digital Object Identifier</span> <br>
    <span class="c11"> EBI</span><span class="c1"> European Bioinformatics Institute</span> <br>
    #if$_PHENOTYPIC #if$_EDAL  <span class="c11"> e!DAL-PGP</span><span class="c1"> Plant Genomics & Phenomics Research Data Repository</span> <br>  #endif$_EDAL   #endif$_PHENOTYPIC  
    #if$_GENETIC  #if$_ENA   <span class="c11">  ENA </span> <span class="c1"> European Nucleotide Archive </span> <br>  #endif$_ENA        #endif$_GENETIC   
    <span class="c11"> EU </span><span class="c1">European Union</span> <br>
    <span class="c11"> FAIR </span><span class="c1">Findable Accessible Interoperable Reproducible</span> <br>
    <span class="c11"> GDPR</span><span class="c1"> General data protection regulation (of the EU)</span> <br>
    #if$_TRANSCRIPTOMIC|$_GENETIC #if$_GEO  <span class="c11"> GEO</span><span class="c1"> Gene Expression Omnibus </span>  <br>  #endif$_GEO   #endif$_TRANSCRIPTOMIC|$_GENETIC   
    #if$_IMAGE #if$_IDR  <span class="c11"> IDR</span><span class="c1"> Image Data Resource </span>  <br>  #endif$_IDR  #endif$_IMAGE 
    <span class="c11"> IP</span><span class="c1"> Intellectual Property</span> <br>
    <span class="c11"> ISO</span><span class="c1"> International Organization for Standardization</span>  <br>
    #if$_TRANSCRIPTOMIC #if$_MIAME <span class="c11"> MIAME</span><span class="c1"> Minimum Information About a Microarray Experiment</span> <br>  #endif$_MIAME  #endif$_TRANSCRIPTOMIC  
    #if$_METABOLOMIC #if$_MMIAMET <span class="c11"> MIAMET</span><span class="c1"> Minimum Information About a METabolomics experiment</span> <br>  #endif$_MMIAMET   #endif$_METABOLOMIC  
    #if$_PROTEOMIC #if$_MIAPE <span class="c11"> MIAPE</span><span class="c1"> Minimum Information About a Proteomics Experiment</span> <br>  #endif$_MIAPE  #endif$_PROTEOMIC  
    #if$_PHENOTYPIC #if$_MIAPPE <span class="c11"> MIAPPE</span><span class="c1"> Minimum Information about Plant Phenotyping Experiment</span> <br>  #endif$_MIAPPE  #endif$_PHENOTYPIC 
    #if$_GENOMIC|$_GENETIC #if$_MIGSEU <span class="c11"> MigsEu </span><span class="c1"> Minimum Information about a Genome Sequence: Eucaryote </span> <br>  #endif$_MIGSEU  
    #if$_MIGSORG <span class="c11"> MigsOrg </span><span class="c1"> Minimum Information about a Genome Sequence: Organelle </span> <br>  #endif$_MIGSORG  
    #if$_MIMAG <span class="c11"> MIMAG </span><span class="c1"> Minimum Information about Metagenome-Assembled Genome </span> <br>  #endif$_MIMAG  
    #if$_MIMARKSSPECIMEN <span class="c11"> MIMARKSSpecimen</span><span class="c1"> Minimal Information about a Marker Specimen: Specimen </span> <br>  #endif$_MIMARKSSPECIMEN 
    #if$_MIMARKSSURVEY  <span class="c11">  MIMARKSSurvey </span><span class="c1"> Minimal Information about a Marker Specimen: Survey </span> <br>  #endif$_MIMARKSSURVEY    #endif$_GENOMIC|$_GENETIC  
    #if$_PROTEOMIC #if$_MIMIX <span class="c11"> MIMIX</span><span class="c1"> The Minimum Information required for reporting a Molecular Interaction Experiment </span> <br>  #endif$_MIMIX   #endif$_PROTEOMIC 
    #if$_GENOMIC|$_GENETIC #if$_MIMS  <span class="c11">  MIMS</span><span class="c1"> Molecular Interactions </span> <br>  #endif$_MIMS    #endif$_GENOMIC|$_GENETIC 
    #if$_TRANSCRIPTOMIC|$_GENOMIC #if$_MINSEQE <span class="c11"> MinSEQe</span><span class="c1"> Minimum Information about a high-throughput Sequencing Experiment</span> <br>  #endif$_MINSEQE #endif$_TRANSCRIPTOMIC|$_GENOMIC 
    #if$_GENOMIC|$_GENETIC #if$_MISAG <span class="c11"> MISAG</span><span class="c1"> Minimum Information about a Single Amplified Genome </span> <br>  #endif$_MISAG
    #if$_MIXS <span class="c11"> MIxS</span><span class="c1"> Minimum Information about any (X) Sequence </span> <br>  #endif$_MIXS #endif$_GENOMIC|$_GENETIC 
    <span class="c11"> NCBI</span><span class="c1"> National Center for Biotechnology Information</span> <br>
    <span class="c11"> NFDI</span><span class="c1"> National Research Data Infrastructure (of Germany)</span> <br>
    <span class="c11"> NGS</span><span class="c1"> Next Generation Sequencing</span> <br>
    #if$_PROTEOMIC #if$_PRIDE  <span class="c11"> PRIDE</span><span class="c1"> PRoteomics IDEntifications Database</span> <br>  #endif$_PRIDE
    #if$_PDB  <span class="c11"> PDB</span><span class="c1"> Protein Data Bank </span> <br>  #endif$_PDB #endif$_PROTEOMIC
    <span class="c11"> RDM</span><span class="c1"> Research Data Management</span> <br>
    #if$_IMAGE  #if$_REMBI  <span class="c11"> REMBI</span><span class="c1"> Recommended Metadata for Biological Images </span> <br>  #endif$_REMBI #endif$_IMAGE 

    #if$_RNASEQ  <span class="c11"> RNASeq</span><span class="c1"> RNA Sequencing</span> <br>  #endif$_RNASEQ 
    <span class="c11"> SOP</span><span class="c1"> Standard Operating Procedures</span>  <br>
    #if$_GENETIC #if$_SRA  <span class="c11"> SRA</span><span class="c1"> Sequence Read Archive</span> <br>  #endif$_SRA #endif$_GENETIC
    #if$_DATAPLANT  <span class="c11">  SWATE</span><span class="c1"> Swate Workflow Annotation Tool for Excel</span> <br>  #endif$_DATAPLANT 
    <span class="c11"> ONP</span><span class="c1"> Oxford Nanopore</span>  <br>
    <span class="c11"> qRT</span> <span class="c11"> PCR</span><span class="c1"> quantitative real time polymerase chain reaction</span>  <br>
    <span class="c11"> WP</span><span class="c1"> Work Package</span>  <br>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c1"></span></p>
        <p class="c22"><span class="c1"></span></p>
    </div>
</div>`,

"practical-guide" : `<div id="practical-guide" class="">
    <div id="practical-guide1" class="content-page">
        <h1 id="text_start" dir="ltr"
            style="line-height: 1.38; text-align: center; margin-top: 20pt; margin-bottom: 6pt;">
            <span style="
                                    font-size: 20pt;
                                    font-family: Arial;
                                    color: #000000;
                                    background-color: transparent;
                                    font-weight: 400;
                                    font-style: normal;
                                    font-variant: normal;
                                    text-decoration: none;
                                    vertical-align: baseline;
                                    
                                    
                                ">
                Practical Data Management Guide of $_PROJECTNAME
            </span>
        </h1>
        <h1 class="c13">
            <b style="font-weight: normal;" id="docs-internal-guid-dc2b5b8c-7fff-c042-d6fa-27fdc29616b1">
                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        This practical guide of data management in $_PROJECTNAME should be considered as a minimum
                        description, leaving flexibility to include additional actions of specific domain or to
                        national or local
                        legislation.#if$_EU $_PROJECTNAME will follow EU FAIR principles.&nbsp; #endif$_EU&nbsp;
                    </span>
                </p>
                <br />
                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        The practical guide of data management in $_PROJECTNAME aims at providing a complete
                        walkthrough for the researcher. The contents are customized based on the user input in the
                        Data Management Plant
                        Generator (DMPG). The practices in this guide are customized to fit related legal, ethical,
                        standardization and funding body requirements. The suitable practices will cover all steps
                        of a data
                        management life-cycle:
                    </span>
                </p>
                <br />
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;">
                    <li dir="ltr" style="
                                            list-style-type: decimal;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                        " aria-level="1">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: transparent;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Data acquisition:
                            </span>
                        </p>
                    </li>
                    <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;">
                        <li dir="ltr" style="
                                                list-style-type: lower-alpha;
                                                font-size: 11pt;
                                                font-family: Arial;
                                                color: #000000;
                                                background-color: transparent;
                                                font-weight: 400;
                                                font-style: normal;
                                                font-variant: normal;
                                                text-decoration: none;
                                                vertical-align: baseline;
                                                
                                            " aria-level="2">
                            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                                role="presentation">
                                <span style="
                                                        font-size: 11pt;
                                                        font-family: Arial;
                                                        color: #000000;
                                                        background-color: transparent;
                                                        font-weight: 400;
                                                        font-style: normal;
                                                        font-variant: normal;
                                                        text-decoration: none;
                                                        vertical-align: baseline;
                                                        
                                                        
                                                    ">
                                    Data generation
                                </span>
                            </p>
                        </li>
                    </ol>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        Data should be generated by devices that are compatible with the open-source format. The
                        $_STUDYOBJECT should be compliant to biodiversity protocols. The protocols used to collect
                       #if$_GENOMIC genomic data, #endif$_GENOMIC 
                    #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA 
                    #if$_TRANSCRIPTOMIC transcriptomic data, #endif$_TRANSCRIPTOMIC  
                    #if$_RNASEQ RNAseq data, #endif$_RNASEQ 
                    #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC  
                    #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC 
                    #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC  
                    #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED  
                    #if$_IMAGE image datasets, #endif$_IMAGE  
                    #if$_MODELS modelling data, #endif$_MODELS 
                    #if$_CODE computational code, #endif$_CODE  
                    #if$_EXCEL excel files, #endif$_EXCEL about $_STUDYOBJECT will be
                        stored#if$_DATAPLANT in the assays folder of ARC
                        repositories. #endif$_DATAPLANT#if!$_DATAPLANT in a FAIR data
                        storage. #endif!$_DATAPLANT&nbsp;
                    </span>
                </p>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="2">
                    <li dir="ltr" style="
                                            list-style-type: lower-alpha;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            margin-left: 36pt;
                                        " aria-level="2">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Data collection
                            </span>
                        </p>
                    </li>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        The data collection process is conducted by experimental scientists and stewarded by
                        $_DATAOFFICER.#if$_DATAPLANT An electronic lab notebook will be used to ensure enough
                        metadata is recorded and
                        guarantees that the data can be further reused. #endif$_DATAPLANT&nbsp;
                    </span>
                </p>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="3">
                    <li dir="ltr" style="
                                            list-style-type: lower-alpha;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            margin-left: 36pt;
                                        " aria-level="2">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Data Organization
                            </span>
                        </p>
                    </li>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        The data organization process is conducted by $_DATAOFFICER. The detailed organization
                        method and procedure are reported to the PIs. #if$_DATAPLANT The data organization will
                        profit from the
                        knowledge-base and data-base of DataPLANT, elastic search will be used to find better ways
                        to organize the data. #endif$_DATAPLANT&nbsp;
                    </span>
                </p>
                <br />
                <br />
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="2">
                    <li dir="ltr" style="
                                            list-style-type: decimal;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                        " aria-level="1">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Annotation
                            </span>
                        </p>
                    </li>
                    <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;">
                        <li dir="ltr" style="
                                                list-style-type: lower-alpha;
                                                font-size: 11pt;
                                                font-family: Arial;
                                                color: #000000;
                                                background-color: transparent;
                                                font-weight: 400;
                                                font-style: normal;
                                                font-variant: normal;
                                                text-decoration: none;
                                                vertical-align: baseline;
                                                
                                            " aria-level="2">
                            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                                role="presentation">
                                <span style="
                                                        font-size: 11pt;
                                                        font-family: Arial;
                                                        color: #000000;
                                                        background-color: #ffffff;
                                                        font-weight: 400;
                                                        font-style: normal;
                                                        font-variant: normal;
                                                        text-decoration: none;
                                                        vertical-align: baseline;
                                                        
                                                        
                                                    ">
                                    Workflow documentation
                                </span>
                            </p>
                        </li>
                    </ol>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        Because the data collection process is conducted by experimental scientists and stewarded by
                        $_DATAOFFICER.#if$_DATAPLANT An electronic lab notebook was used to ensure enough metadata
                        is recorded
                        and guarantees that the data can be further reused. The workflow can be retrieved from the
                        electronic workbook by using the toolkits provided from the DataPLANT such as SWATE and
                        arccommander.
                        #endif$_DATAPLANT&nbsp;
                    </span>
                </p>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="2">
                    <li dir="ltr" style="
                                            list-style-type: lower-alpha;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            margin-left: 36pt;
                                        " aria-level="2">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Metadata completion
                            </span>
                        </p>
                    </li>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        In case some of the metadata is still missing from the documentation from the experimental
                        scientists and data officer. #if$_DATAPLANT Raw data identifier and parsers provided by
                        DataPLANT will be
                        used to get meta data directly from the raw data file. The metadata collected from the raw
                        data file can also be used to validate the metadata previously collected in case there are
                        any mistakes.
                        #endif$_DATAPLANT We foresee using #if$_RNASEQ|$_GENOMIC e.g.#if$_MINSEQE MinSEQe for
                        sequencing data and #endif$_MINSEQE #endif$_RNASEQ|$_GENOMIC Metabolights compatible forms
                        for metabolites as
                        well as MIAPPE for phenotyping like data. The latter will thus allow the integration of data
                        across projects and safeguards that reuse established and tested protocols. Additionally, we
                        will use ontology
                        terms to enrich the data sets relying on free and open ontologies. In addition, additional
                        ontology terms might be created and be canonized during $_PROJECTNAME.
                    </span>
                </p>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="3">
                    <br />
                    <li dir="ltr" style="
                                            list-style-type: decimal;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                        " aria-level="1">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Maintenance:&nbsp;
                            </span>
                        </p>
                    </li>
                </ol>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;">
                    <li dir="ltr" style="
                                            list-style-type: lower-alpha;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            margin-left: 36pt;
                                        " aria-level="2">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Data storage
                            </span>
                        </p>
                    </li>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        Raw data collected in previous steps are stored immediately by using#if$_DATAPLANT the
                        infrastructure of DataPLANT #endif$_DATAPLANT #if!$_DATAPLANT in a secure infrastructure. <a
                            href="https://nfdi4plants.org/content/learn-more/annotated-research-context.html">ARC</a>
                        (Annotated
                        Research Context) is used as a container to store the raw data as well as metadata and
                        workflow. #endif!$_DATAPLANT
                    </span>
                </p>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="2">
                    <li dir="ltr" style="
                                            list-style-type: lower-alpha;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            margin-left: 36pt;
                                        " aria-level="2">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Data curation
                            </span>
                        </p>
                    </li>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        #if$_DATAPLANT Data stored in Annotated Research Context (ARC) is curated regularly as long as there are needs for update
                        or revision. #endif$_DATAPLANT #if!$_DATAPLANT Data is curated regularly as long as there are
                        needs for
                        update or revision. #endif!$_DATAPLANT
                    </span>
                </p>
                <br />
                <br />
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="4">
                    <li dir="ltr" style="
                                            list-style-type: decimal;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                        " aria-level="1">
                        <p dir="ltr"
                            style="line-height: 1.38; background-color: #ffffff; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Publication and sharing
                            </span>
                        </p>
                    </li>
                    <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;">
                        <li dir="ltr" style="
                                                list-style-type: lower-alpha;
                                                font-size: 11pt;
                                                font-family: Arial;
                                                color: #000000;
                                                background-color: transparent;
                                                font-weight: 400;
                                                font-style: normal;
                                                font-variant: normal;
                                                text-decoration: none;
                                                vertical-align: baseline;
                                                
                                            " aria-level="2">
                            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                                role="presentation">
                                <span style="
                                                        font-size: 11pt;
                                                        font-family: Arial;
                                                        color: #000000;
                                                        background-color: #ffffff;
                                                        font-weight: 400;
                                                        font-style: normal;
                                                        font-variant: normal;
                                                        text-decoration: none;
                                                        vertical-align: baseline;
                                                        
                                                        
                                                    ">
                                    Data publishing
                                </span>
                            </p>
                        </li>
                    </ol>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        <p class="c0">
                            <span class="c1">
                                Data will be made available via the $_PROJECTNAME platform using a user-friendly front
                                end that allows data visualization. Besides this it will be ensured that data which
                                can be stored in international discipline related repositories which use specialized technologies: #if$_GENETIC #if$_GENBANK NCBI-GenBank, #endif$_GENBANK
                                #if$_ENA EBI-ENA, #endif$_ENA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                                #endif$_GENETIC #if$_TRANSCRIPTOMIC|$_GENETIC #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO
                                NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC #if$_ARRAYEXPRESS
                                EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC #if$_IMAGE #if$_BIOIMAGE
                                EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                                #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-Metabolights, #endif$_METABOLIGHTS
                                #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct
                                (Molecular interactions) #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                                EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                                Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                                #endif$_EDAL #endif$_PHENOTYPIC

                                #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be
                                processed there as well. #endif$_OTHEREP
                            </span>
                        </p>
                    </span>
                </p>
                <ol style="margin-top: 0; margin-bottom: 0; padding-inline-start: 48px;" start="2">
                    <li dir="ltr" style="
                                            list-style-type: lower-alpha;
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: transparent;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            margin-left: 36pt;
                                        " aria-level="2">
                        <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;"
                            role="presentation">
                            <span style="
                                                    font-size: 11pt;
                                                    font-family: Arial;
                                                    color: #000000;
                                                    background-color: #ffffff;
                                                    font-weight: 400;
                                                    font-style: normal;
                                                    font-variant: normal;
                                                    text-decoration: none;
                                                    vertical-align: baseline;
                                                    
                                                ">
                                Data sharing
                            </span>
                        </p>
                    </li>
                </ol>
                <p dir="ltr" style="line-height: 1.38; margin-left: 72pt; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        In case data is only shared within the consortium, if the data is not yet finished or under
                        IP checks, the data is hosted internally, and the username and the password will be required
                        (see also our GDPR
                        rules). In the case data is made public under final EU or US repositories, completely
                        anonymous access is normally allowed. This is the case for ENA as well and both are in line
                        with GDPR
                        requirements.
                    </span>
                </p>
                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        <span class="Apple-tab-span"> </span>
                    </span>
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        <span style="width: 100px; display: inline-block; position: relative;"></span>
                    </span>
                </p>
            </b>
        </h1>
        <h3 dir="ltr" style="line-height: 1.38; margin-top: 16pt; margin-bottom: 4pt;">
            <span style="
                                    font-size: 13.999999999999998pt;
                                    font-family: Arial;
                                    color: #434343;
                                    background-color: transparent;
                                    font-weight: 400;
                                    font-style: normal;
                                    font-variant: normal;
                                    text-decoration: none;
                                    vertical-align: baseline;
                                    
                                    
                                ">
                Metadata focus timeline
            </span>
        </h3>
        <h1 class="c13">
            <b style="font-weight: normal;">
                <br />
                <br />
                <div dir="ltr" style="margin-left: 0pt;" align="left">
                    <table style="border: none; border-collapse: collapse;">
                        <colgroup>
                            <col width="104" />
                            <col width="520" />
                            <col width="132" />
                            <col width="104" />
                            <col width="94" />
                            <col width="102" />
                            <col width="88" />
                        </colgroup>
                        <thead>
                            <tr style="height: 0pt;">
                                <th style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    " scope="col">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Stages
                                        </span>
                                    </p>
                                </th>
                                <th style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    " scope="col">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Actions
                                        </span>
                                    </p>
                                </th>
                            </tr>
                            <tr style="height: 0pt;">
                                <th style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    " scope="col">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Study
                                        </span>
                                    </p>
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            initialization
                                        </span>
                                    </p>
                                </th>
                                <th style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    " scope="col">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            The metadata of study is created at the beginning of the project and
                                            updated continuously afterwards #if$_DATAPLANT , the input of the DMP
                                            generator created during the proposal
                                            stage can be reused. #endif$_DATAPLANT &nbsp;
                                        </span>
                                    </p>
                                </th>
                            </tr>
                            <tr style="height: 0pt;">
                                <th style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    " scope="col">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Sample
                                        </span>
                                    </p>
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Collection
                                        </span>
                                    </p>
                                </th>
                                <th style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    " scope="col">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            The information used to identify exact samples are initiated before
                                            experiments and updated at assay creation stages.
                                        </span>
                                    </p>
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            #if$_DATAPLANT The sample SWATE template will be used to document the
                                            sample metadata. A part of sample metadata which can be retrieved from
                                            the raw data will be updated
                                            afterwards using the <a
                                                href="https://nfdi4plants.org/content/learn-more/annotated-research-context.html">ARC</a>
                                            parsers #endif$_DATAPLANT&nbsp;
                                        </span>
                                    </p>
                                </th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr style="height: 0pt;">
                                <td style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    ">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Assay
                                        </span>
                                    </p>
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Creation
                                        </span>
                                    </p>
                                </td>
                                <td style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    ">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Assay metadata must be collected as a daily routine during the
                                            experimental phrase. #if$_DATAPLANT A electronic lab notebooks will be
                                            used to guarantee the applicability and
                                            correctness of the notebook content #endif$_DATAPLANT&nbsp;
                                        </span>
                                    </p>
                                </td>
                            </tr>
                            <tr style="height: 0pt;">
                                <td style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    ">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Computational Analysis
                                        </span>
                                    </p>
                                    <br />
                                </td>
                                <td style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    ">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Workflow annotation will be conducted during the computational analysis
                                            phrase. #if$_DATAPLANT The workflow metadata will be stored in the assay
                                            folder of the
                                            <a
                                                href="https://nfdi4plants.org/content/learn-more/annotated-research-context.html">ARC</a>. #endif$_DATAPLANT&nbsp;
                                        </span>
                                    </p>
                                </td>
                            </tr>
                            <tr style="height: 0pt;">
                                <td style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    ">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            Results Sharing
                                        </span>
                                    </p>
                                </td>
                                <td style="
                                                        border-left: solid #000000 1pt;
                                                        border-right: solid #000000 1pt;
                                                        border-bottom: solid #000000 1pt;
                                                        border-top: solid #000000 1pt;
                                                        vertical-align: top;
                                                        padding: 5pt 5pt 5pt 5pt;
                                                        overflow: hidden;
                                                        overflow-wrap: break-word;
                                                    ">
                                    <p dir="ltr" style="line-height: 1.2; margin-top: 0pt; margin-bottom: 0pt;">
                                        <span style="
                                                                font-size: 11pt;
                                                                font-family: Arial;
                                                                color: #000000;
                                                                background-color: #ffffff;
                                                                font-weight: 400;
                                                                font-style: normal;
                                                                font-variant: normal;
                                                                text-decoration: none;
                                                                vertical-align: baseline;
                                                                
                                                                
                                                            ">
                                            The metadata of results are collected after all modifications and should
                                            not be changed after publication. #if$_DATAPLANT Collection of result
                                            metadata before publication and
                                            the conversion from <a
                                                href="https://nfdi4plants.org/content/learn-more/annotated-research-context.html">ARC</a>
                                            to the repositories will be taken care of by the ARC2REPO converter and
                                            done with minimal efforts. #endif$_DATAPLANT&nbsp;
                                        </span>
                                    </p>
                                </td>
                            </tr>
                        </tbody>
                    </table>
                </div>
                <br />
                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                    <span style="
                                            font-size: 11pt;
                                            font-family: Arial;
                                            color: #000000;
                                            background-color: #ffffff;
                                            font-weight: 400;
                                            font-style: normal;
                                            font-variant: normal;
                                            text-decoration: none;
                                            vertical-align: baseline;
                                            
                                            
                                        ">
                        <span style="width: 100px; display: inline-block; position: relative;"></span>
                    </span>
                </p>
            </b>
        </h1>
        <h3 dir="ltr" style="line-height: 1.38; margin-top: 16pt; margin-bottom: 4pt;">
            <span style="
                                    font-size: 13.999999999999998pt;
                                    font-family: Arial;
                                    color: #434343;
                                    background-color: transparent;
                                    font-weight: 400;
                                    font-style: normal;
                                    font-variant: normal;
                                    text-decoration: none;
                                    vertical-align: baseline;
                                    
                                    
                                ">
                Preferred formats for raw data
            </span>
        </h3>
        <h1 class="c13">
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">#if$_GENOMIC<span>&nbsp;&nbsp;</span></span>
            </p>
            <div dir="ltr" style="margin-left: 0pt;" align="left">
                <table style="border: none;">
                    <colgroup>
                        <col width="100" />
                        <col width="190" />
                    </colgroup>
                    <tbody>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        extension_ident
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Format Name
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.h5</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Hierarchical Data Format
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .bam
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        compressed binary version of a SAM file
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 51.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .cram
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        compressed columnar file format for storing biological sequences aligned to
                                        a reference sequence
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.fa</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .faa
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fas
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fasta
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fastq
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fastq
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .ffn
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fna
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.fq</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fastq
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .frn
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .sff
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        sff-trim
                                    </span>
                                </p>
                            </td>
                        </tr>
                    </tbody>
                </table>
            </div>
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                    #endif$_GENOMIC</span>
            </p>
            <br />
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">#if$_RNASEQ<span>&nbsp;&nbsp;</span></span>
            </p>
            <div dir="ltr" style="margin-left: 0pt;" align="left">
                <table style="border: none;">
                    <colgroup>
                        <col width="100" />
                        <col width="190" />
                    </colgroup>
                    <tbody>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .bam
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        compressed binary version of a SAM file
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 51.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .cram
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        compressed columnar file format for storing biological sequences aligned to
                                        a reference sequence
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.fa</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .faa
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fas
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fast5
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        HDF5
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fasta
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fastq
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fastq
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .ffn
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fna
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.fq</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fastq
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .frn
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        fasta
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .sff
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        sff-trim
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        bas.h5
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        HDF5
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.h5</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Hierarchical Data Format
                                    </span>
                                </p>
                            </td>
                        </tr>
                    </tbody>
                </table>
            </div>
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                    #endif$_RNASEQ</span>
            </p>
            <br />
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">&nbsp;#if$_METABOLOMIC<span>&nbsp;&nbsp;</span></span>
            </p>
            <div dir="ltr" style="margin-left: 0pt;" align="left">
                <table style="border: none;">
                    <colgroup>
                        <col width="100" />
                        <col width="190" />
                    </colgroup>
                    <tbody>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .cdf
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        netCDF (AIA/ANDI) interchange data format
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .cmp
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        netCDF compare file
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .abf
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Axon Binary File
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.d</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Agilent
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .dat
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Chromtech, Finnigan, VG
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .idb
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span style="
                                                            font-size: 10pt;
                                                            color: rgb(28, 29, 30);
                                                            background-color: transparent;
                                                            font-variant-numeric: normal;
                                                            font-variant-east-asian: normal;
                                                            vertical-align: baseline;
                                                            
                                                        ">
                                        MASSLAB binary file
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 39.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .jpf
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Mass Center Main Mass Spectrometry Data (JEOL USA, Inc.)
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .lcd
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Shimadzu LC Solution / Labsolutions Data File
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .mgf
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Mascot Generic File
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .raw
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Thermo Xcalibur, Micromass (Waters), PerkinElmer, Waters
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .scan
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        a spectrum or a Total Ion Chromatogram (TIC)
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .wiff
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        ABI/Sciex
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .xps
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Thermo Fisher Scientific K-Alpha+ spectrometer file
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        cdf.cmp
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        netCDF compare file
                                    </span>
                                </p>
                            </td>
                        </tr>
                    </tbody>
                </table>
            </div>
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                    #endif$_METABOLOMIC</span>
            </p>
            <br />
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">&nbsp;#if$_PROTEOMIC<span>&nbsp;&nbsp;</span></span>
            </p>
            <div dir="ltr" style="margin-left: 0pt;" align="left">
                <table style="border: none;">
                    <colgroup>
                        <col width="100" />
                        <col width="190" />
                    </colgroup>
                    <tbody>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .baf
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Bruker
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.d</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Agilent
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .dat
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Chromtech, Finnigan, VG
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .fid
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Bruker
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .ita
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        ION-TOF
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .itm
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        ION-TOF
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .mgf&nbsp;
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Mascot Generic File
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">.ms</span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Finnigan (Thermo)
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .ms2
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Sequest MS/MS peak list
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .pkl
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Micromass peak list
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .qgd
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Shimadzu
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .qgd
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Shimadzu
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .raw
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Thermo Xcalibur, Micromass (Waters), PerkinElmer, Waters
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .raw
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Physical Electronics/ULVAC-PHI
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .sms
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Bruker/Varian
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .spc
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Shimadzu
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .splib&nbsp;
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        spectral library file
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .t2d
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        ABI/Sciex
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 27.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .tdc
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Physical Electronics/ULVAC-PHI
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .wiff
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        ABI/Sciex
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .xms
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Bruker/Varian
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .yep
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Bruker
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .dta
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        Sequest MS/MS peak list
                                    </span>
                                </p>
                            </td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .msp
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <br /></td>
                        </tr>
                        <tr style="height: 15.75pt;">
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                                    <span
                                        style="font-size: 10pt; background-color: transparent; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                                        .nist
                                    </span>
                                </p>
                            </td>
                            <td
                                style="border-width: 0.75pt; border-color: rgb(204, 204, 204); vertical-align: bottom; padding: 2pt; overflow: hidden; overflow-wrap: break-word;">
                                <br /></td>
                        </tr>
                    </tbody>
                </table>
            </div>
            <p dir="ltr" style="line-height: 1.38; margin-top: 0pt; margin-bottom: 0pt;">
                <span
                    style="font-size: 11pt; font-variant-numeric: normal; font-variant-east-asian: normal; vertical-align: baseline;  ">
                    #endif$_PROTEOMIC</span>
            </p>
            <br />
            <br />
        </h1>
    </div>
</div>`,

"bmbf-dmp" : `<div id="bmbf-dmp" class="">
    <div id="bmbf-dmp1" class="content-page" id="text_start">
        <h1 class="text-center">
            <p class="text-center h1">
                <center>Datenmanagementplan fr BMBF (Beta test)</center>
            </p>
        </h1>
        <p class="c0"><span class="c3"><b>Projektname: $_PROJECTNAME</b></span></p>
        <p class="c0"><span class="c3"><b>Forschungsf&ouml;rderer</b>: Bundesministerium f&uuml;r Bildung und
                Forschung</span></p>
        <p class="c0"><span class="c3"><b>F&ouml;rderprogramm:</b> $_FUNDINGPROGRAMME </span></p>
        <p class="c0"><span class="c3"><b>FKZ: $_DMPVERSION</b></span></p>

        <p class="c0"><span class="c3"><b>Projektkoordinator</b>: $_USERNAME</span></p>
        <p class="c0"><span class="c3"><b>Kontaktperson Datenmanagement</b>: $_DATAOFFICER</span></p>

        <p class="c0"><span class="c3"><b>Kontakt</b>: $_EMAIL</span></p>
        <p class="c0"><span class="c3"><b>Projektbeschreibung</b>:


                <p class="c0"> <span class="c1">
                        Das $_PROJECTNAME hat folgendes Ziel: $_PROJECTAIM. Daher sind Datenerhebung #if!$_VVISUALIZATION und Integration #endif!$_VVISUALIZATION #if$_VVISUALIZATION , Integration und Visualisierung
                        #endif$_VVISUALIZATION #if$_DATAPLANT unter Verwendung der DataPLANT ARC-Struktur absolut
                        notwendig #endif$_DATAPLANT #if!$_DATAPLANT durch einen standardisierten
                        Datenmanagementprozess absolut notwendig #endif!$_DATAPLANT da die Daten nicht nur zum
                        Verstndnis von Prinzipien verwendet werden, sondern auch ber die Herkunft der analysierten
                        Daten informiert werden muss. Stakeholder mssen ebenfalls ber die Herkunft der Daten
                        informiert werden. Es ist daher notwendig sicherzustellen, dass die Daten gut generiert und
                        auch gut mit Metadaten unter Verwendung offener Standards annotiert werden, wie im nchsten
                        Abschnitt dargelegt.

                    </span></p>

                <p class="c0">
                    <span class="c1">
                        Das $_PROJECTNAME wird die folgenden Arten von Rohdaten sammeln und/oder generieren:
                        #if$_GENETIC genetische Daten, #endif$_GENETIC  
                #if$_GENOMIC genomische Daten, #endif$_GENOMIC 
                #if$_CLONED-DNA geklonte DNA Daten, #endif$_CLONED-DNA 
                #if$_TRANSCRIPTOMIC transkriptomische Daten, #endif$_TRANSCRIPTOMIC  
                #if$_RNASEQ RNA-Seq-Daten, #endif$_RNASEQ 
                #if$_METABOLOMIC metabolomische Daten, #endif$_METABOLOMIC  
                #if$_PROTEOMIC proteomische Daten, #endif$_PROTEOMIC 
                #if$_PHENOTYPIC Pflanzenphnotypische Daten, #endif$_PHENOTYPIC  
                #if$_TARGETED gezielte Tests (z. B. Glukose- und Fruktosegehalt), #endif$_TARGETED  
                #if$_IMAGE Bilddatenstze, #endif$_IMAGE  
                #if$_MODELS Modellausgnge, #endif$_MODELS 
                #if$_CODE Rechencode, #endif$_CODE  
                #if$_EXCEL Excel-Daten, #endif$_EXCEL Daten, die sich auf $_STUDYOBJECT
                        beziehen. Zustzlich werden die Rohdaten auch durch analytische Pipelines verarbeitet und
                        modifiziert, was zu unterschiedlichen Ergebnissen fhren kann oder ad-hoc-Datenanalyse-Teile
                        umfassen kann. #if$_DATAPLANT Diese Pipelines werden im DataPLANT ARC
                        verfolgt. #endif$_DATAPLANT Daher wird darauf geachtet, diese Ressourcen (einschlielich der
                        analytischen Pipelines) zu dokumentieren und zu archivieren#if$_DATAPLANT unter Rckgriff
                        auf die Expertise im DataPLANT-Konsortium #endif$_DATAPLANT.
                    </span>
                </p>
            </span></p>
        <p class="c0 c2"><span class="c1"></span></p>
        <p class="c0"><span class="c3"><b>Erstellungsdatum:</b> $_CREATIONDATE </span></p>
        <p class="c0"><span class="c3"><b>&Auml;nderungsdatum:</b> $_MODIFICATIONDATE </span></p>
        <p class="c0"><span class="c3"><b>Zu beachtende Vorgaben:</b> </span></p>

        <p class="c0"><span class="c1">#if$_EU Das $_PROJECTNAME ist Teil der Open Data Initiative (ODI) der EU.
                #endif$_EU Um optimal von offenen Daten zu profitieren, ist es notwendig, die Daten nicht nur zu
                speichern, sondern sie auch auffindbar, zugnglich, interoperabel und wiederverwendbar (FAIR) zu
                machen. #if$_PROTECT Wir untersttzen offene und FAIR-Daten, bercksichtigen jedoch auch die
                Notwendigkeit, einzelne Datenstze zu schtzen. #endif$_PROTECT
            </span>
        </p>
        <p class="c0"><span class="c1">#if$_DATAPLANT Durch die Implementierung von DataPLANT knnen Forscher
                sicherstellen, dass alle relevanten Richtlinien und Anforderungen im Zusammenhang mit dem
                Datenmanagement eingehalten werden, was zu einer hheren Qualitt und Zuverlssigkeit der
                Forschungsdaten fhrt. #endif$_DATAPLANT</span>
        </p>



        <p class="c0 c2"><span class="c3"><b>Datenerhebung</b></span></p>

        <p class="c0 c2"><span class="c1">
                <p class="c0"><span class="c1">ffentliche Daten werden wie im vorherigen Absatz beschrieben
                        extrahiert. Fr das $_PROJECTNAME werden spezifische Datenstze von den Konsortialpartnern
                        generiert.</span></p>


                <p class="c0">
                    <span class="c1"> Daten unterschiedlicher Typen oder aus verschiedenen Bereichen werden mit
                        einzigartigen Anstzen generiert. Zum Beispiel:</span>
                </p>
                <ul style="list-style-type:disc;">

                    #if$_GENETIC&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Genetische Daten werden durch Kreuzungen und Zuchtexperimente
                                generiert und umfassen Rekombinationsfrequenzen und Crossover-Ereignisse, die
                                genetische Marker und quantitative Merkmalsloci positionieren knnen, die mit
                                physischen genomischen Markern/Varianten assoziiert werden knnen. </span>
                        </p>
                    </li>
                    #endif$_GENETIC

                    #if$_GENOMIC&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Genomische Daten werden aus Sequenzdaten erstellt, die verarbeitet
                                werden, um Gene, regulatorische Elemente, transponierbare Elemente und physikalische
                                Marker wie SNPs, Mikrosatelliten und strukturelle Varianten zu identifizieren.
                            </span>
                        </p>
                    </li>
                    #endif$_GENOMIC
                    #if$_CLONED-DNA&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Der Ursprung und die Zusammenstellung der klonierten DNA umfassen (a)
                                die Quelle der ursprnglichen Vektorsequenz mit Add-Gene-Referenz, sofern verfgbar,
                                und die Quelle der Insert-DNA (z.B. Amplifikation durch PCR aus einer bestimmten
                                Probe oder aus einer vorhandenen Bibliothek), (b) die Klonierungsstrategie (z.B.
                                Restriktionsendonuklease-Verdau/Ligation, PCR, TOPO-Klonierung, Gibson-Assembly,
                                LR-Rekombination), und (c) die verifizierte DNA-Sequenz des finalen rekombinanten
                                Vektors.</span>
                        </p>
                    </li>
                    #endif$_CLONED-DNA

                    #if$_TRANSCRIPTOMIC&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Methoden zur Erfassung von Transkriptomik-Daten werden aus
                                Mikroarrays, quantitativer PCR, Northern Blotting, RNA-Immunprzipitation und
                                Fluoreszenz-in-situ-Hybridisierung ausgewhlt. RNA-Seq-Daten werden mit separaten
                                Methoden gesammelt.</span>
                        </p>
                    </li>
                    #endif$_TRANSCRIPTOMIC
                    #if$_RNASEQ&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> RNA-Sequenzierung wird unter Verwendung von Short-Read- oder
                                Long-Read-Plattformen entweder intern oder an akademische Einrichtungen oder
                                kommerzielle Dienste ausgelagert und die Rohdaten werden mit etablierten
                                bioinformatischen Pipelines verarbeitet. </span>
                        </p>
                    </li>
                    #endif$_RNASEQ
                    #if$_METABOLOMIC&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Metabolomische Daten werden durch gekoppelte Chromatographie und
                                Massenspektrometrie unter Verwendung gezielter oder ungezielter Anstze
                                generiert.</span>
                        </p>
                    </li>
                    #endif$_METABOLOMIC #if$_PROTEOMIC&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Proteomische Daten werden durch gekoppelte Chromatographie und
                                Massenspektrometrie zur Analyse der Proteinmenge und -identifikation sowie durch
                                zustzliche Techniken zur Strukturanalyse, zur Identifizierung posttranslationaler
                                Modifikationen und zur Charakterisierung von Proteininteraktionen generiert.</span>
                        </p>
                    </li>
                    #endif$_PROTEOMIC

                    #if$_PHENOTYPIC&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Phnotypische Daten werden mit Hilfe von Phnotypisierungsplattformen
                                und entsprechenden Ontologien generiert, einschlielich Anzahl/Gre von Organen wie
                                Bltter, Blumen, Knospen usw., Gre der gesamten Pflanze,
                                Stngel/Wurzel-Architektur (Anzahl der seitlichen Zweige/Wurzeln usw.),
                                Organstrukturen/Morphologien, quantitativen Metriken wie Farbe, Turgor,
                                Gesundheits-/Nhrstoffindikatoren und anderen. </span>
                        </p>
                    </li>
                    #endif$_PHENOTYPIC



                    #if$_TARGETED&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Gezielte Assays-Daten (z. B. Glukose- und Fruktosekonzentrationen oder
                                Produktions-/Nutzungsraten) werden mit spezifischen Gerten und Methoden generiert,
                                die im Laborbuch vollstndig dokumentiert sind. </span>
                        </p>
                    </li>
                    #endif$_TARGETED

                    #if$_IMAGE&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Bilddaten werden durch Gerte wie Kameras, Scanner und Mikroskope in
                                Kombination mit Software generiert. Originalbilder, die Metadaten wie
                                EXIF-Fotoinformationen enthalten, werden archiviert.</span>
                        </p>
                    </li>
                    #endif$_IMAGE

                    #if$_MODELS&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Modelldaten werden durch Softwaresimulationen generiert. Der
                                vollstndige Workflow, einschlielich der Umgebung, Laufzeit, Parameter und
                                Ergebnisse, wird dokumentiert und archiviert. </span>
                        </p>
                    </li>
                    #endif$_MODELS

                    #if$_CODE&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Computercode wird von Programmierern erstellt. </span>
                        </p>
                    </li>
                    #endif$_CODE

                    #if$_EXCEL&nbsp;
                    <li>
                        <p class="c0">
                            <span class="c1"> Excel-Tabellen werden durch Ausfllen spezifischer Dateien erstellt,
                                die Feldbeobachtungen oder andere digitale Erhebungen enthalten. </span>
                        </p>
                    </li>
                    #endif$_EXCEL

                    

                    
                </ul>

                #if$_PREVIOUSPROJECTS&nbsp;
                <p class="c0"><span class="c1"> Daten aus frheren Projekten wie $_PREVIOUSPROJECTS werden
                        bercksichtigt. </span></p>
                #endif$_PREVIOUSPROJECTS

                <p class="c0"><span class="c1">Wir erwarten die Erzeugung von $_RAWDATA GB Rohdaten und bis zu
                        $_DERIVEDDATA GB verarbeiteten Daten.</span></p>
                <p class="c0"><span class="c3"></span></p>

            </span></p>


        <p class="c0"><span class="c3"><b>Datenspeicherung:</b> </span></p>

        <p class="c0">
            <span class="c1">
                #if$_DATAPLANT In DataPLANT, die Datenspeicherung basiert auf dem Annotated Research Context (ARC).
                Dieser ist passwortgeschtzt, daher muss vor dem Erhalt von Daten oder der Generierung von Proben
                eine Authentifizierung erfolgen. #endif$_DATAPLANT
            </span>
        </p>


        <p class="c20">
            <span class="c1">
                Online-Plattformen werden durch Schwachstellen-Scans, Zwei-Faktor-Authentifizierung und tgliche
                automatische Backups geschtzt, die eine sofortige Wiederherstellung ermglichen. Alle Partner, die
                vertrauliche Projektdaten halten, nutzen sichere Plattformen mit automatischen Backups und sicheren
                externen Kopien.
                #if$_DATAPLANT DataHUB und ARCs wurden in DataPLANT generiert, Datensicherheit wird durchgesetzt.
                Dies umfasst sichere Speicherung, und die Verwendung von Passwrtern und Benutzernamen wird generell
                ber separate sichere Medien bertragen. #endif$_DATAPLANT
            </span>
        </p>

        <p class="c0 "><span class="c1">
                <p class="c0"><span class="c1">Das $_PROJECTNAME trgt die Kosten fr die Datenkuratierung,
                        #if$_DATAPLANT ARC-Konsistenzprfungen, #endif$_DATAPLANT und die Datenwartung/-sicherheit
                        vor der bertragung an ffentliche Repositorien. Nachfolgende Kosten werden dann von den
                        Betreibern dieser Repositorien getragen.</span></p>

                <p class="c0">
                    <span class="c1">
                        Zustzlich werden Kosten fr die Speicherung nach der Verffentlichung von den
                        Endpunkt-Repositorien (z.B. ENA) getragen, jedoch nicht vom $_PROJECTNAME oder seinen
                        Mitgliedern, sondern durch das Betriebsbudget dieser Repositorien.
                    </span>
                </p>
                <span class="c1">
                    Es wird sichergestellt, dass Daten, die in internationalen, disziplinspezifischen Repositories
                    gespeichert werden knnen, die spezialisierte Technologien nutzen:

                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_GENETIC Fr genetische Daten: #if$_GENBANK
                            NCBI-GenBank, #endif$_GENBANK #if$_SRA NCBI-SRA, #endif$_SRA #if$_ENA
                            EBI-ENA, #endif$_ENA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                            #if$_GEO NCBI-GEO, #endif$_GEO . #endif$_GENETIC </span>
                    </p>

                    <p class="c0">
                        <span class="c1  list-to-remove-comma">#if$_TRANSCRIPTOMIC Fr Transkriptomdaten: #if$_SRA
                            NCBI-SRA, #endif$_SRA #if$_GEO NCBI-GEO, #endif$_GEO #if$_ARRAYEXPRESS
                            EBI-ArrayExpress, #endif$_ARRAYEXPRESS . #endif$_TRANSCRIPTOMIC</span>
                    </p>

                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_IMAGE Fr Bilddaten: #if$_BIOIMAGE EBI-BioImage
                            Archive #endif$_BIOIMAGE #if$_IDR IDR (Image Data Resource), #endif$_IDR .
                            #endif$_IMAGE </span>
                    </p>

                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_METABOLOMIC Fr Metabolomdaten: #if$_METABOLIGHTS
                            EBI-Metabolights, #endif$_METABOLIGHTS #if$_METAWORKBENCH Metabolomics
                            Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct (Molecular
                            interactions), #endif$_INTACT . #endif$_METABOLOMIC </span>
                    </p>
                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_PROTEOMIC Fr Proteomikdaten: #if$_PRIDE
                            EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB
                            #if$_CHEBI Chebi, #endif$_CHEBI .
                            #endif$_PROTEOMIC </span>
                    </p>

                    <p class="c0">
                        <span class="c1 list-to-remove-comma">#if$_PHENOTYPIC Fr phnotypische Daten: #if$_EDAL e!DAL-PGP (Plant
                            Genomics & Phenomics Research Data Repository), #endif$_EDAL . #endif$_PHENOTYPIC
                        </span>
                    </p>

                
                    #if$_OTHEREP und $_OTHEREP werden auch verwendet, um Daten zu speichern und die Daten werden
                    dort ebenfalls verarbeitet. #endif$_OTHEREP

                </span>


            </span></p>

        <p class="c0"><span class="c3"><b>Die Dateibenennung erfolgt nach folgendem Standard</b>:</span></p>
        <p class="c0">
            <span class="c1">
                Datenvariablen werden mit Standardnamen versehen. Zum Beispiel werden Gene, Proteine und Metaboliten
                gem anerkannter Nomenklatur und Konventionen benannt. Diese werden nach Mglichkeit auch mit
                funktionalen Ontologien verknpft. Datenstze werden ebenfalls sinnvoll benannt, um die Lesbarkeit
                durch Menschen zu gewhrleisten. Pflanzennamen umfassen traditionelle Namen, Binomialnamen und alle
                Stamm-/Kultivar-/Unterart-/Sortenbezeichner.
            </span>
        </p>

        <p class="c0"><span class="c1"></span></p>
        <p class="c0"><span class="c3"><b>Datendokumentation</b></span></p>
        <p class="c0">
            <span class="c1">Wir verwenden die Investigation, Study, Assay (ISA) Spezifikation zur
                Metadaten-Erstellung. #if$_RNASEQ|$_GENOMIC Fr spezifische Daten (z.B. RNASeq oder genomische
                Daten) verwenden wir Metadatentemplates der Endpunkt-Repositorien. </span><span
                class="c1">#if$_MINSEQE The Minimum Information About a Next-generation Sequencing Experiment
                (MinSEQe) wird ebenfalls verwendet. #endif$_MINSEQE #endif$_RNASEQ|$_GENOMIC</span>
            <span class="c1">
                Die folgenden Metadaten-/Mindestinformationsstandards werden zur Sammlung von Metadaten verwendet:
                    <span class="list-to-remove-comma">
                    
                        #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS (Minimum Information about any (X) Sequence), #endif$_MIXS
                        #if$_MIGSEU MigsEu (Minimum Information about a Genome Sequence: Eucaryote), #endif$_MIGSEU
                        #if$_MIGSORG MigsOrg (Minimum Information about a Genome Sequence: Organelle), #endif$_MIGSORG
                        #if$_MIMS MIMS (Minimum Information about Metagenome or Environmental), #endif$_MIMS
                        #if$_MIMARKSSPECIMEN MIMARKSSpecimen (Minimal Information about a Marker Specimen:
                        Specimen), #endif$_MIMARKSSPECIMEN
                        #if$_MIMARKSSURVEY MIMARKSSurvey (Minimal Information about a Marker Specimen:
                        Survey), #endif$_MIMARKSSURVEY
                        #if$_MISAG MISAG (Minimum Information about a Single Amplified Genome), #endif$_MISAG
                        #if$_MIMAG MIMAG (Minimum Information about Metagenome-Assembled Genome), #endif$_MIMAG
                        #endif$_GENOMIC|$_GENETIC
                        #if$_TRANSCRIPTOMIC
                        #if$_MINSEQE MINSEQE (Minimum Information about a high-throughput SEQuencing
                        Experiment), #endif$_MINSEQE #endif$_TRANSCRIPTOMIC
                        #if$_TRANSCRIPTOMIC #if$_MIAME MIAME (Minimum Information About a Microarray
                        Experiment), #endif$_MIAME #endif$_TRANSCRIPTOMIC
                        #if$_IMAGE
                        #if$_REMBI REMBI (Recommended Metadata for Biological Images), #endif$_REMBI
                        #endif$_IMAGE
                        #if$_RNASEQ|$_GENOMIC 
                        #if$_MINSEQE
                            MinSEQe (Minimum Information about a high-throughput Sequencing Experiment),
                        #endif$_MINSEQE 
                        #endif$_RNASEQ|$_GENOMIC
                        #if$_METABOLOMIC
                        #if$_MMIAMET
                        MIAMET (Minimum Information About a METabolomics experiment),
                        #endif$_MMIAMET
                        #endif$_METABOLOMIC
                        #if$_PROTEOMIC
                        #if$_MIAPE MIAPE (Minimum Information About a Proteomics Experiment), #endif$_MIAPE
                        #if$_MIMIX MIMix (The Minimum Information required for reporting a Molecular Interaction Experiment), #endif$_MIMIX
                        #endif$_PROTEOMIC .
                        </span>

                #if$_METABOLOMIC #if$_METABOLIGHTS Metabolights-Einreichungskonforme Standards werden fr
                metabolomische Daten verwendet, wo dies von den Konsortialpartnern akzeptiert wird.#issuewarning
                Einige Metabolomik-Partner betrachten Metabolights nicht als akzeptierten Standard.#endissuewarning
                #endif$_METABOLIGHTS #endif$_METABOLOMIC Als Teil der Pflanzenforschungsgemeinschaft verwenden wir
                #if$_MIAPPE MIAPPE fr Phnotypisierungsdaten im weitesten Sinne, werden aber auch auf
                #endif$_MIAPPE spezifische SOPs fr zustzliche Annotationen #if$_DATAPLANT zurckgreifen, die
                fortgeschrittene DataPLANT-Annotationen und Ontologien bercksichtigen. #endif$_DATAPLANT


            </span>
        </p>
        <p class="c0"><span class="c1">
                In dem Fall, dass einige Metadaten noch fehlen, werden diese von den experimentellen
                Wissenschaftlern und dem Datenbeauftragten dokumentiert. #if$_DATAPLANT Rohdaten-Identifier und
                Parser, die von DataPLANT bereitgestellt werden, um
                Metadaten direkt aus der Rohdatei zu extrahieren. Die aus der Rohdatei gesammelten Metadaten knnen
                auch verwendet werden, um die zuvor gesammelten Metadaten zu validieren, falls Fehler auftreten.
                #endif$_DATAPLANT Wir sehen vor, #if$_RNASEQ|$_GENOMIC z.B.#if$_MINSEQE MinSEQe fr
                Sequenzierungsdaten zu verwenden und #endif$_MINSEQE #endif$_RNASEQ|$_GENOMIC Metabolights-kompatible
                Formulare fr Metaboliten sowie MIAPPE fr phnotypische Daten.
                Letzteres ermglicht die Integration von Daten ber Projekte hinweg und stellt sicher, dass
                etablierte und getestete Protokolle wiederverwendet werden. Darber hinaus werden wir
                Ontologiebegriffe verwenden, um die Datenstze mit freien und offenen Ontologien anzureichern.
                Zustzlich knnten zustzliche Ontologiebegriffe erstellt und whrend des $_PROJECTNAME kanonisiert
                werden.</span></p>


        <p class="c0"><span class="c3"><b>Legitimit&auml;t</b></span></p>
        <p class="c0"><span class="c1">
                <p class="c0">
                    <span class="c1">
                        Im Moment erwarten wir keine ethischen oder rechtlichen Probleme beim Datenaustausch. In
                        Bezug auf Ethik, da es sich um Pflanzendaten handelt, ist kein Ethikkomitee erforderlich,
                        jedoch wird Sorgfalt bei der Aufteilung der Vorteile von Pflanzenressourcen bercksichtigt.
                        #issuewarning Sie mssen sicherstellen, dass Sie die erforderliche Sorgfaltspflicht erfllen. Derzeit warten wir auf eine Klrung, ob das Nagoya-Protokoll ( siehe Nagoya-Protokoll) auch Sequenzinformationen umfasst. Unabhngig davon kann die Nutzung von Material aus einem anderen Land als Ihrem eigenen (oder dem Ihres Partners) und dessen physikalische oder biochemische Charakterisierung (z.B. Metaboliten, Proteom, RNASeq, etc.) eine relevante Handlung im Sinne des Nagoya-Protokolls darstellen. Ausnahmen knnten Materialien aus Lndern wie den USA (Nicht-Partner), Irland (hat nicht unterzeichnettrotzdem kontaktieren), usw. sein, wobei jedoch andere Gesetze Anwendung finden knnten. #endissuewarning
                    </span>
                </p>
                <p class="c0">
                    <span class="c1">
                        Die einzigen personenbezogenen Daten, die mglicherweise gespeichert werden, sind der Name
                        und die Zugehrigkeit des Einreichers in den Metadaten der Daten. Darber hinaus werden
                        personenbezogene Daten fr Verbreitungs- und Kommunikationsaktivitten gesammelt, wobei
                        spezifische Methoden und Verfahren verwendet werden, die von den $_PROJECTNAME-Partnern
                        entwickelt wurden, um den Datenschutz einzuhalten. #issuewarning Sie mssen informieren und
                        besser eine SCHRIFTLICHE Zustimmung einholen, dass Sie E-Mails und Namen oder sogar
                        Pseudonyme wie Twitter-Handles speichern, wir entschuldigen uns sehr fr diese Probleme, die
                        wir nicht erfunden haben. #endissuewarning
                    </span>
                </p>


            </span></p>

        <p class="c0"><span class="c3"><b>Data Sharing</b></span></p>
        <p class="c0 c2"><span class="c1">
                <p class="c20">
                    <span class="c1">
                        Falls Daten nur innerhalb des Konsortiums geteilt werden, wenn die Daten noch nicht fertig
                        sind oder sich in der IP-Prfung befinden, werden die Daten intern gehostet und der
                        Benutzername und das Passwort werden bentigt (siehe auch unsere GDPR-Regeln).
                        Wenn Daten unter finalen EU- oder US-Repositorys ffentlich gemacht werden, ist
                        normalerweise ein vollstndig anonymer Zugang erlaubt. Dies ist auch bei ENA der Fall und
                        beide entsprechen den GDPR-Anforderungen.

                    </span>
                </p>
                <p class="c0">
                    <span class="c1">Es wird keine Einschrnkungen geben, sobald die Daten ffentlich gemacht
                        werden.</span>
                    <span class="c1">
                        #if$_EARLY Einige Rohdaten werden sofort nach ihrer Erfassung und Verarbeitung ffentlich
                        gemacht. #endif$_EARLY #if$_BEFOREPUBLICATION Relevante verarbeitete Datenstze werden
                        ffentlich gemacht, wenn die Forschungsergebnisse verffentlicht
                        werden. #endif$_BEFOREPUBLICATION #if$_ENDOFPROJECT Am Ende des Projekts werden alle Daten
                        ohne Sperrfrist verffentlicht. #endif$_ENDOFPROJECT #if$_EMBARGO Daten, die einer Sperrfrist
                        unterliegen, sind bis zum Ende der Sperrfrist nicht ffentlich zugnglich. #endif$_EMBARGO
                        #if$_REQUEST Daten werden auf Anfrage verfgbar gemacht, was eine kontrollierte Weitergabe
                        ermglicht und gleichzeitig eine verantwortungsvolle Nutzung sicherstellt. #endif$_REQUEST
                        #if$_IPISSUE IP-Probleme werden vor der Verffentlichung berprft. #endif$_IPISSUE Alle
                        Konsortialpartner werden ermutigt,
                        Daten vor der Verffentlichung zugnglich zu machen, offen und/oder unter
                        Vorverffentlichungsvereinbarungen #if$_GENOMIC wie die in Fort Lauderdale gestarteten und
                        durch den Toronto International Data Release Workshop festgelegten Vereinbarungen
                        #endif$_GENOMIC . Dies wird umgesetzt, sobald die IP-bezogenen berprfungen abgeschlossen
                        sind.
                    </span>
                </p>


                <p class="c0">
                    <span class="c1">
                        Die Daten werden zunchst den $_PROJECTNAME Partnern zugutekommen, aber auch ausgewhlten
                        Stakeholdern, die eng in das Projekt eingebunden sind, und dann der wissenschaftlichen
                        Gemeinschaft, die an $_STUDYOBJECT arbeitet. $_DATAUTILITY Darber hinaus knnen auch die
                        allgemeine ffentlichkeit, die an $_STUDYOBJECT interessiert ist, die Daten nach der
                        Verffentlichung nutzen. Die Daten werden gem dem Verbreitungs- und Kommunikationsplan des
                        $_PROJECTNAME verbreitet, #if$_DATAPLANT der sich mit der DataPLANT-Plattform oder anderen
                        Mitteln abstimmt #endif$_DATAPLANT.

                    </span>
                </p>
            </span></p>
        <p class="c0"><span class="c3"><b>Datenerhalt</b></span></p>
        <p class="c0 c2"><span class="c1">

                <p class="c0">
                    <span class="c1">Wir erwarten, dass wir Rohdaten im Bereich von $_RAWDATA GB an Daten
                        generieren. Die Gre der abgeleiteten Daten wird etwa $_DERIVEDDATA GB betragen.
                    </span>
                </p>


                <p class="c0">
                    #if$_DATAPLANT Da das $_PROJECTNAME eng mit DataPLANT abgestimmt ist, werden der ARC-Konverter und
                    DataHUB verwendet, um die Endpunkt-Repositories zu finden und die Daten automatisch in die
                    Repositories hochzuladen. #endif$_DATAPLANT

                    <span class="c1"> </span>
                </p>
                <p class="c0">
                    <span class="c1">
                        Die Daten werden ber die $_PROJECTNAME-Plattform mit einer benutzerfreundlichen Oberflche
                        verfgbar gemacht, die eine Datenvisualisierung ermglicht. Die Endpunkt-Repositories sind:
                        #if$_GENETIC #if$_GENBANK NCBI-GenBank, #endif$_GENBANK
                        #if$_ENA EBI-ENA, #endif$_ENA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                        #endif$_GENETIC #if$_TRANSCRIPTOMIC|$_GENETIC #if$_SRA NCBI-SRA, #endif$_SRA #if$_GEO
                        NCBI-GEO, #endif$_GEO #endif$_TRANSCRIPTOMIC|$_GENETIC #if$_TRANSCRIPTOMIC #if$_ARRAYEXPRESS
                        EBI-ArrayExpress, #endif$_ARRAYEXPRESS #endif$_TRANSCRIPTOMIC #if$_IMAGE #if$_BIOIMAGE
                        EBI-BioImage Archive, #endif$_BIOIMAGE #if$_IDR IDR, #endif$_IDR
                        #endif$_IMAGE #if$_METABOLOMIC #if$_METABOLIGHTS EBI-MetaboLights, #endif$_METABOLIGHTS
                        #if$_METAWORKBENCH Metabolomics Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct
                        (Molecular interactions) #endif$_INTACT #endif$_METABOLOMIC #if$_PROTEOMIC #if$_PRIDE
                        EBI-PRIDE, #endif$_PRIDE #if$_PDB PDB, #endif$_PDB #if$_CHEBI
                        Chebi, #endif$_CHEBI #endif$_PROTEOMIC #if$_PHENOTYPIC #if$_EDAL e!DAL-PGP,
                        #endif$_EDAL #endif$_PHENOTYPIC.

                        #if$_OTHEREP und $_OTHEREP werden auch verwendet, um Daten zu speichern und die Daten werden
                        dort ebenfalls verarbeitet. #endif$_OTHEREP

                    </span>
                </p>
                <p class="c0">
                    <span class="c1">
                        Die Einreichung ist kostenlos, und es ist das Ziel (zumindest von ENA), so viele Daten wie
                        mglich zu erhalten. Daher sind Absprachen weder notwendig noch sinnvoll.
                        Catch-all-Repositories sind nicht erforderlich.
                        #if$_DATAPLANT Fr DataPLANT wurde dies vereinbart. #endif$_DATAPLANT #issuewarning Wenn
                        keine Datenmanagementplattform wie DataPLANT verwendet wird, mssen Sie ein geeignetes
                        Repository finden, um Ihre Daten nach der Verffentlichung zu speichern oder zu archivieren.
                        #endissuewarning

                    </span>
                </p>

            </span></p>
    </div>
</div>
</div>`,


"BBSRC-dmp": `<div id="BBSRC-dmp" class="">
    <div id="BBSRC-dmp1" class="content-page" id="text_start">
        <span class="c2">Data management plan of $_PROJECTNAME for BBSRC</span>
        <ul class="c0 ">

            <li class="c0 li-bullet-0"><span class="c0">Data Areas and Data Types &ndash; $_PROJECTNAME will collect
                    and/or generate the following types of raw data: #if$_GENETIC genetic data, #endif$_GENETIC  
                    #if$_GENOMIC genomic data, #endif$_GENOMIC 
                    #if$_CLONED-DNA cloned DNA data, #endif$_CLONED-DNA 
                    #if$_TRANSCRIPTOMIC transcriptomic data, #endif$_TRANSCRIPTOMIC  
                    #if$_RNASEQ RNAseq data, #endif$_RNASEQ 
                    #if$_METABOLOMIC Metabolomic data, #endif$_METABOLOMIC  
                    #if$_PROTEOMIC proteomic data, #endif$_PROTEOMIC 
                    #if$_PHENOTYPIC phenotypic data, #endif$_PHENOTYPIC  
                    #if$_TARGETED targeted assays (e.g. glucose and fructose content), #endif$_TARGETED  
                    #if$_IMAGE image datasets, #endif$_IMAGE  
                    #if$_MODELS modelling data, #endif$_MODELS 
                    #if$_CODE computational code, #endif$_CODE  
                    #if$_EXCEL excel files, #endif$_EXCEL
                    which are related to $_STUDYOBJECT. In addition, the raw data will also be processed and
                    modified using analytical pipelines, which may yield different results or include ad hoc data
                    analysis parts. #if$_DATAPLANT These pipelines will be tracked in the DataPLANT
                    ARC. #endif$_DATAPLANT Therefore, care will be taken to document and archive these resources
                    (including the analytical pipelines) as well#if$_DATAPLANT relying on the expertise in the
                    DataPLANT consortium #endif$_DATAPLANT. </span></li>

            <span class="c0">We expect to generate raw data in the range of $_RAWDATA GB of data. The size of the
                derived data will be about $_DERIVEDDATA GB.</span>
            <p class="c0"><span class="c0"></span></p>
            <li class="c0 li-bullet-0"><span class="c0">Standards and Metadata &ndash;&nbsp;                <span class="c1">
                All datasets will be associated with unique identifiers and will be annotated with metadata. We will
                use Investigation, Study, Assay (ISA) specification for metadata creation. $_PROJECTNAME will rely
                on community standards plus additional recommendations applicable in the plant science, such as the
                <span class="list-to-remove-comma">
                    The following metadata/ minimum informatin standards will be used to collect metadata:
                    #if$_GENOMIC|$_GENETIC #if$_MIXS MIxS (Minimum Information about any (X) Sequence), #endif$_MIXS
                    #if$_MIGSEU MigsEu (Minimum Information about a Genome Sequence: Eucaryote), #endif$_MIGSEU
                    #if$_MIGSORG MigsOrg (Minimum Information about a Genome Sequence: Organelle), #endif$_MIGSORG
                    #if$_MIMS MIMS (Minimum Information about Metagenome or Environmental), #endif$_MIMS
                    #if$_MIMARKSSPECIMEN MIMARKSSpecimen (Minimal Information about a Marker Specimen:
                    Specimen), #endif$_MIMARKSSPECIMEN
                    #if$_MIMARKSSURVEY MIMARKSSurvey (Minimal Information about a Marker Specimen:
                    Survey), #endif$_MIMARKSSURVEY
                    #if$_MISAG MISAG (Minimum Information about a Single Amplified Genome), #endif$_MISAG
                    #if$_MIMAG MIMAG (Minimum Information about Metagenome-Assembled Genome), #endif$_MIMAG
                    #endif$_GENOMIC|$_GENETIC
                    #if$_TRANSCRIPTOMIC
                    #if$_MINSEQE MINSEQE (Minimum Information about a high-throughput SEQuencing
                    Experiment), #endif$_MINSEQE #endif$_TRANSCRIPTOMIC
                    #if$_TRANSCRIPTOMIC #if$_MIAME MIAME (Minimum Information About a Microarray
                    Experiment), #endif$_MIAME #endif$_TRANSCRIPTOMIC
                    #if$_IMAGE
                    #if$_REMBI REMBI (Recommended Metadata for Biological Images), #endif$_REMBI
                    #endif$_IMAGE
                    #if$_RNASEQ|$_GENOMIC 
                    #if$_MINSEQE
                        MinSEQe (Minimum Information about a high-throughput Sequencing Experiment),
                    #endif$_MINSEQE 
                    #endif$_RNASEQ|$_GENOMIC
                    #if$_METABOLOMIC
                    #if$_MMIAMET
                    MIAMET (Minimum Information About a METabolomics experiment),
                    #endif$_MMIAMET
                    #endif$_METABOLOMIC
                    #if$_PROTEOMIC
                    #if$_MIAPE MIAPE (Minimum Information About a Proteomics Experiment), #endif$_MIAPE
                    #if$_MIMIX MIMix (The Minimum Information required for reporting a Molecular Interaction Experiment), #endif$_MIMIX
                    #endif$_PROTEOMIC 
                    #if$_PHENOTYPIC #if$_MIAPPE MIAPPE (Minimum Information about Plant Phenotyping Experiment) #endif$_MIAPPE  #endif$_PHENOTYPIC .
                    </span>
            </span>
            <span class="c1">#if$_METABOLOMIC #if$_METABOLIGHTS The Metabolights submission compliant standards are used for metabolomic data. #issuewarning Some metabolomics partners considers Metabolights
                not an accepted standard. #endissuewarning #endif$_METABOLIGHTS #endif$_METABOLOMIC These specific standard unlike cross-domain minimal sets such as the Dublin core, which
                mostly define the submitter and the general type of data, allow reusability by other researchers by
                defining properties of the plant (see the preceding section). However, $_PROJECTNAME also implement minimal cross-domain annotations #if$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 such as #endif$_DUBLINCORE|$_DARWINCORE|$_BIOSCHEMAS|$_SCHEMAORG|$_MARC21 <span class="list-to-remove-comma"> #if$_DUBLINCORE Dublin Core, #endif$_DUBLINCORE #if$_DARWINCORE Darwin Core, #endif$_DARWINCORE #if$_SCHEMAORG Schema.org, #endif$_SCHEMAORG #if$_BIOSCHEMAS BioSchemas, #endif$_BIOSCHEMAS #if$_MARC21 MARC 21 #endif$_MARC21 . </span> #if$_DATAPLANT The core integration with DataPLANT will also allow
                individual releases to be tagged with a Digital Object Identifier (DOI).
                #endif$_DATAPLANT #if$_OTHERSTANDARDS Other standards such as $_OTHERSTANDARDINPUT are also adhered
                to. #endif$_OTHERSTANDARDS
            </span>

                </span>
            </li>
            <li class="c0 li-bullet-0"><span class="c0">Reuse of published data &ndash; The project builds on
                    existing data sets and relies on them. #if$_RNASEQ For example, without a proper genomic
                    reference it is very difficult to analyze next-generation sequencing (NGS) data
                    sets. #endif$_RNASEQ It is also important to include existing data-sets on the expression and
                    metabolic behavior of the $_STUDYOBJECT, and on existing background knowledge. #if$_PARTNERS of
                    the partners: $_PARTNERS #endif$_PARTNERS Genomic references can be gathered from reference databases for
                    genomes/ and sequences, like the NCBI (US National Center for Biotechnology Information),
                    EBI (European Bioinformatics Institute) and DDBJ (DNA Data Bank of Japan). Furthermore, prior
                    &#39;unstructured&#39; data in the form of publications and data contained therein will be used
                    for decision making.</span></li>
        </ul>
        <p class="c0 c0"><span class="c0"></span></p>
        <ul class="c0 ">
            <li class="c0 li-bullet-0"><span class="c0">Secondary Use &ndash; The data will initially benefit the
                    $_PROJECTNAME partners, but will also be made available to selected stakeholders closely involved in
                    the project, and then the scientific community working on $_STUDYOBJECT. $_DATAUTILITY In
                    addition, the general public interested in $_STUDYOBJECT can also use the data after
                    publication. The data will be disseminated according to $_PROJECTNAME&#39;s dissemination and
                    communication plan, #if$_DATAPLANT which aligns with DataPLANT platform or other means #endif$_DATAPLANT.</span></li>

            <li class="c0 li-bullet-0"><span class="c0">Methods for Data Sharing &ndash; <p class="c0">
                        <span class="c1">
                            Data will be made available via the $_PROJECTNAME platform using a user-friendly front end
                            that allows data visualization. Besides this it will be ensured that data which can be
                            stored in
                            international discipline related repositories which use specialized technologies:
                        


                            <p class="c0">
                                <span class="c1 list-to-remove-comma"> #if$_GENETIC|$_GENOMIC For genetic or genomic data: #if$_GENBANK NCBI-GenBank, #endif$_GENBANK #if$_ENA EBI-ENA, #endif$_ENA #endif$_GENETIC|$_GENOMIC #if$_GENETIC #if$_SRA NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_ARRAYEXPRESS EBI-ArrayExpress, #endif$_ARRAYEXPRESS
                                    #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #endif$_GENETIC #if$_GENETIC|$_GENOMIC . #endif$_GENETIC|$_GENOMIC </span>
                            </p>

                            <p class="c0">
                                <span class="c1  list-to-remove-comma">#if$_TRANSCRIPTOMIC For Transcriptomic data: #if$_SRA
                                    NCBI-SRA (Sequence Read Archive), #endif$_SRA #if$_GEO NCBI-GEO (Gene Expression Omnibus), #endif$_GEO #if$_ARRAYEXPRESS
                                    EBI-ArrayExpress, #endif$_ARRAYEXPRESS . #endif$_TRANSCRIPTOMIC</span>
                            </p>

                            <p class="c0">
                                <span class="c1 list-to-remove-comma">#if$_IMAGE For image data: #if$_BIOIMAGE EBI-BioImage
                                    Archive, #endif$_BIOIMAGE #if$_IDR IDR (Image Data Resource), #endif$_IDR .
                                    #endif$_IMAGE </span>
                            </p>

                            <p class="c0">
                                <span class="c1 list-to-remove-comma">#if$_METABOLOMIC For metabolomic data: #if$_METABOLIGHTS
                                    EBI-Metabolights, #endif$_METABOLIGHTS #if$_METAWORKBENCH Metabolomics
                                    Workbench, #endif$_METAWORKBENCH #if$_INTACT IntAct (Molecular
                                    interactions), #endif$_INTACT . #endif$_METABOLOMIC </span>
                            </p>
                            <p class="c0">
                                <span class="c1 list-to-remove-comma">#if$_PROTEOMIC For proteomics data: #if$_PRIDE
                                    EBI-PRIDE (PRoteomics IDEntifications Database), #endif$_PRIDE #if$_PDB PDB (Protein Data Bank), #endif$_PDB
                                    #if$_CHEBI Chebi (Chemical Entities of Biological Interest), #endif$_CHEBI .
                                    #endif$_PROTEOMIC </span>
                            </p>

                            <p class="c0">
                                <span class="c1 list-to-remove-comma">#if$_PHENOTYPIC For phenotypic data: #if$_EDAL e!DAL-PGP (Plant
                                    Genomics & Phenomics Research Data Repository), #endif$_EDAL . #endif$_PHENOTYPIC
                                </span>
                            </p>
                            #if$_OTHEREP $_OTHEREP will also be used to store data and the data will be
                            processed there as well. #endif$_OTHEREP
                        </span>
                    </p></span></li li class="c0 li-bullet-0"><span class="c0">Proprietary Data &ndash; Open public
                data will be used whenever possible. </span></li>
            <li class="c0 li-bullet-0"><span class="c0">Timeframes
                    #if$_EARLY Some raw data is made public as soon as it is collected and processed. #endif$_EARLY
                    #if$_BEFOREPUBLICATION Relevant processed datasets are made public when the research findings
                    are published. #endif$_BEFOREPUBLICATION #if$_ENDOFPROJECT At the end of the project, all data
                    without embargo period will be published. #endif$_ENDOFPROJECT #if$_EMBARGO Data, which is
                    subject to an embargo period, is not publicly accessible until the end of embargo
                    period. #endif$_EMBARGO #if$_REQUEST Data is made available upon request, allowing controlled
                    sharing while ensuring responsible use. #endif$_REQUEST #if$_IPISSUE IP issues will be checked
                    before publication. #endif$_IPISSUE All consortium partners will be
                    encouraged to make data available before publication, openly and/or under pre-publication
                    agreements #if$_GENOMIC such as those started in Fort Lauderdale and set forth by the Toronto
                    International Data
                    Release Workshop #endif$_GENOMIC . This will be implemented as soon as IP-related checks are
                    complete.
                </span></li>
            <li class="c0 li-bullet-0"><span class="c0">Format of the Final Dataset &ndash; Whenever possible, data
                    will be stored in common and openly defined formats including all the necessary metadata. By
                    default, no proprietary formats will be used. However Microsoft Excel files (according to
                    ISO/IEC 29500-1:2016) might be used as intermediates by the consortium#if$_DATAPLANT and by some
                    Annotated Research Context (ARC) components#endif$_DATAPLANT. In addition, text files might be edited in text processor
                    files, but will be shared as pdf.</span>
    </div>
</div>`


}
